<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Methods</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="methods_files/libs/clipboard/clipboard.min.js"></script>
<script src="methods_files/libs/quarto-html/quarto.js"></script>
<script src="methods_files/libs/quarto-html/popper.min.js"></script>
<script src="methods_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="methods_files/libs/quarto-html/anchor.min.js"></script>
<link href="methods_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="methods_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="methods_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="methods_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="methods_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#defining-biologging" id="toc-defining-biologging" class="nav-link active" data-scroll-target="#defining-biologging">Defining biologging</a></li>
  <li><a href="#reproducible-literature-search" id="toc-reproducible-literature-search" class="nav-link" data-scroll-target="#reproducible-literature-search">Reproducible literature search</a></li>
  <li><a href="#assessing-open-data-practices" id="toc-assessing-open-data-practices" class="nav-link" data-scroll-target="#assessing-open-data-practices">Assessing open data practices</a></li>
  <li><a href="#statistical-analysis" id="toc-statistical-analysis" class="nav-link" data-scroll-target="#statistical-analysis">Statistical analysis</a></li>
  </ul>
</nav>
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Methods</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="defining-biologging" class="level2">
<h2 class="anchored" data-anchor-id="defining-biologging">Defining biologging</h2>
<p>The scientific literature has yet to arrive on consensus definitions for biologging and related terms, such as biotelemetry, tagging, and tracking. Rutz and Hays <span class="citation" data-cites="rutz2009">(<a href="#ref-rutz2009" role="doc-biblioref">Rutz and Hays 2009</a>)</span> defined biologging as “the use of miniaturized animal-attached tags for logging and/or relaying of data about an animal’s movements, behaviour, physiology and/or environment” and Cooke et al. <span class="citation" data-cites="cooke2004">(<a href="#ref-cooke2004" role="doc-biblioref">Cooke et al. 2004</a>)</span> defined biotelemetry as “remote measurement of physiological, behavioural, or energetic data”. These and other definitions share the following two characteristics:</p>
<ul>
<li><p>An electronic device is attached to an animal, externally or internally.</p></li>
<li><p>The device uses sensors and memory to record observations (e.g.&nbsp;GPS and accelerometers) <em>or</em> produces a signal that may be recorded by other sensors (e.g.&nbsp;acoustic telemetry and Argos satellites).</p>
<ul>
<li>Hybrid devices may do both, either separately (e.g.&nbsp;a tag that transmits to Argos but also records depth from a pressure sensor, which may be downloaded after recovering the device) or simultaneously (e.g.&nbsp;a tag that records location using a GPS sensor, then transmits that location to Argos).</li>
</ul></li>
</ul>
<p>Further complicating matters, biologging may be used in many research contexts. Much of the biologger literature involves deployments on unrestrained, undomesticated animals for the purposes of addressing questions in ecology and conservation. But agricultural research also uses biologging devices to assess animal welfare and resource use (domesticated animals) and biomedical research uses animal-borne heart rate sensors in lab experiments (restrained).</p>
<p>The focus of this review is:</p>
<ol type="1">
<li>Biologging devices that use sensors (as opposed to acoustic or radio transmitters)</li>
<li>Deployed on unrestrained, undomesticated animals</li>
</ol>
<p>The choice to exclude transmitters was based on the properties of the data collected. Data from sensor-based biologgers can reasonably be considered self-contained, whereas data from transmitters depend equally on the receiver network. This external dependency complicates the assessment of open data sharing, the primary goal of this review. We make an exception for one type of transmitter, Argos tags, because the Argos system provides data to researchers with properties more similar to sensor-based biologgers.</p>
</section>
<section id="reproducible-literature-search" class="level2">
<h2 class="anchored" data-anchor-id="reproducible-literature-search">Reproducible literature search</h2>
<p>On 2023-08-21, we queried Web of Science Core Collection for papers related to biologging with the following query:</p>
<p><code>animal AND (biologg* OR bio-logg* OR biotelemetry OR electronic tag* OR satellite track* OR GPS telemetry OR satellite telemetry OR satellite transmit* OR GPS collar* OR depth recorder* OR accelerometer OR archival tag*)</code></p>
<p>This yielded 6654 papers. We limited results to Web of Science categories with 100+ papers we deemed relevant, leaving 4799 papers. The categories we included were:</p>
<ul>
<li><p>Ecology</p></li>
<li><p>Zoology</p></li>
<li><p>Marine Freshwater Biology</p></li>
<li><p>Biodiversity Conservation</p></li>
<li><p>Multidisciplinary Sciences</p></li>
<li><p>Environmental Sciences</p></li>
<li><p>Oceanography</p></li>
<li><p>Biology</p></li>
<li><p>Behavioral Sciences</p></li>
<li><p>Ornithology</p></li>
<li><p>Evolutionary Biology</p></li>
<li><p>Fisheries</p></li>
</ul>
<p>We excluded the following categories with 100+ papers:</p>
<ul>
<li><p>Agriculture Dairy Animal Science</p></li>
<li><p>Veterinary Sciences</p></li>
<li><p>Engineering Electrical Electronic</p></li>
<li><p>Physiology</p></li>
<li><p>Food Science Technology</p></li>
<li><p>Engineering Biomedical</p></li>
<li><p>Agriculture Multidisciplinary</p></li>
<li><p>Computer Science Interdisciplinary Applications</p></li>
<li><p>Instruments Instrumentation</p></li>
</ul>
</section>
<section id="assessing-open-data-practices" class="level2">
<h2 class="anchored" data-anchor-id="assessing-open-data-practices">Assessing open data practices</h2>
<p>At least two reviewers examined each paper. We recorded the following for each paper:</p>
<ul>
<li><p>Type of study (research, review, method, perspective, data, or unrelated)</p></li>
<li><p>Did the paper present novel (i.e., previously unpublished) biologging data?</p></li>
<li><p>What was the animal’s context? (wild, captive, or domestic)</p></li>
<li><p>Did the study incorporate data collected elsewhere? (e.g., satellite remote sensing or phylogenetic tree)</p>
<ul>
<li>This can be subjective. For example, if a paper involved 3d path reconstruction from dead reckoning and they used a movement speed reported in another paper, does that count as data collected elsewhere? As a heuristic, we decided that “data” for these purposes would be limited to products that don’t fit into papers themselves. E.g., numbers in a table don’t count, but a raster of land cover does.</li>
</ul></li>
<li><p>What types of sensors were used? Following <span class="citation" data-cites="williams2019">(<a href="#ref-williams2019" role="doc-biblioref">Williams et al. 2019</a>)</span>, we divided these into three categories:</p>
<ul>
<li><p>Location: GPS, Argos, geolocation (using light and/or temperature), depth, altitude, other</p></li>
<li><p>Intrinsic: accelerometer, magnetometer, gyroscope, heart rate, Hall sensor, stomach temperature, neurological sensors, Pitot tubes, speed sensors (e.g., paddle wheels), internal body temperature (other than stomach), other</p></li>
<li><p>Environment: ambient temperature, video, audio, salinity, fluorescence, proximity sensors (to other tagged animals), light, other</p></li>
</ul></li>
<li><p>Which taxa were tracked? We standardized scientific names using the Integrated Taxonomic Information System.</p></li>
<li><p>Were data shared according to <a href="https://www.go-fair.org/fair-principles/">FAIR principles</a>? For each data source used in a study (biologging or otherwise), we recorded:</p>
<ul>
<li><p>Is it biologging data?</p></li>
<li><p>Is there a data availability statement?</p>
<ul>
<li><p>Positive examples include a DOI, a URL, supplemental material, citation of a published work, the name of a widely-used data source (e.g., Landsat Thematic Mapper), or “available on request”</p></li>
<li><p>Negative examples include not-widely-used data sources (e.g., a local municipality), insufficient descriptions (e.g., bathymetry is included in the analysis, but no source or description specified), and future intentions (e.g., “data will be archived in Dryad”)</p></li>
</ul></li>
<li><p>If the data deposited were deposited in a repository, which one?</p></li>
<li><p>FAIR principle F1: do the data have a DOI or other <em>unique</em> and <em>persistent</em> identifier?</p>
<ul>
<li><p>Positive examples include DOIs</p></li>
<li><p>Negative examples include URLs, Movebank project IDs (not to be confused with the Movebank Data Repository, which assigns DOIs), and insufficient descriptions to identify data source</p></li>
</ul></li>
<li><p>FAIR principle F4: are the data indexed in a searchable resource?</p>
<ul>
<li>Positive examples include repositories that meet most of the TRUST Principles for digital repositories <span class="citation" data-cites="lin2020">(<a href="#ref-lin2020" role="doc-biblioref">Lin et al. 2020</a>)</span>, such as the Movebank Data Repository, Dryad, or the National Snow and Ice Data Center.</li>
<li>Negative examples include other websites and supplemental materials to articles.</li>
</ul></li>
<li><p>FAIR principle A1.1: are the data accessible by an open, free protocol?</p>
<ul>
<li><p>Positive examples include <em>open</em> repositories, such as the Movebank Data Repository or Dryad (note: this criterion passes even if incomplete data are placed in an open repository, such as summary tables on Dryad)</p></li>
<li><p>Negative examples include <em>closed</em> repositories, such as EUROMAMMAL or the Seabird Tracking Database, and “available upon request”</p></li>
</ul></li>
<li><p>FAIR principle I1: do the data use a formal, accessible, shared, and broadly applicable language for knowledge representation?</p>
<ul>
<li><p>Positive examples include the Movebank data model and widely-used GIS formats (e.g., GeoTIFFs)</p></li>
<li><p>Negative examples include CSV files that don’t follow a <em>formal</em>, <em>shared</em> data standard</p></li>
</ul></li>
<li><p>FAIR principle R1.1: are the data released with a clear and accessible usage license?</p>
<ul>
<li><p>Positive examples include open data licenses (e.g., CC0) or data distributed by a public entity (e.g., National Centers for Environmental Information)</p></li>
<li><p>Negative examples include unspecified licenses or “rights retained”</p></li>
</ul></li>
<li><p>Are the data complete for <em>reproducibility</em> and <em>reuse</em>?</p>
<ul>
<li><p>Positive examples include raw and processed biologging data</p></li>
<li><p>Negative examples include summary tables</p></li>
</ul></li>
</ul></li>
</ul>
</section>
<section id="statistical-analysis" class="level2">
<h2 class="anchored" data-anchor-id="statistical-analysis">Statistical analysis</h2>
<p>Crunch numbers om nom nom</p>

</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="list">
<div id="ref-cooke2004" class="csl-entry" role="listitem">
Cooke, Steven J., Scott G. Hinch, Martin Wikelski, Russel D. Andrews, Louise J. Kuchel, Thomas G. Wolcott, and Patrick J. Butler. 2004. <span>“Biotelemetry: A Mechanistic Approach to Ecology.”</span> <em>Trends in Ecology &amp; Evolution</em> 19 (6): 334–43. <a href="https://doi.org/10.1016/j.tree.2004.04.003">https://doi.org/10.1016/j.tree.2004.04.003</a>.
</div>
<div id="ref-lin2020" class="csl-entry" role="listitem">
Lin, Dawei, Jonathan Crabtree, Ingrid Dillo, Robert R. Downs, Rorie Edmunds, David Giaretta, Marisa De Giusti, et al. 2020. <span>“The TRUST Principles for Digital Repositories.”</span> <em>Scientific Data</em> 7 (1). <a href="https://doi.org/10.1038/s41597-020-0486-7">https://doi.org/10.1038/s41597-020-0486-7</a>.
</div>
<div id="ref-rutz2009" class="csl-entry" role="listitem">
Rutz, Christian, and Graeme C. Hays. 2009. <span>“New Frontiers in Biologging Science.”</span> <em>Biology Letters</em> 5 (3): 289–92. <a href="https://doi.org/10.1098/rsbl.2009.0089">https://doi.org/10.1098/rsbl.2009.0089</a>.
</div>
<div id="ref-williams2019" class="csl-entry" role="listitem">
Williams, Hannah J., Lucy A. Taylor, Simon Benhamou, Allert I. Bijleveld, Thomas A. Clay, Sophie Grissac, Urška Demšar, et al. 2019. <span>“Optimizing the Use of Biologgers for Movement Ecology Research.”</span> Edited by Jean-Michel Gaillard. <em>Journal of Animal Ecology</em> 89 (1): 186–206. <a href="https://doi.org/10.1111/1365-2656.13094">https://doi.org/10.1111/1365-2656.13094</a>.
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>