<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Trends, challenges, and opportunities in open biologging data</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="registeredreport_files/libs/clipboard/clipboard.min.js"></script>
<script src="registeredreport_files/libs/quarto-html/quarto.js"></script>
<script src="registeredreport_files/libs/quarto-html/popper.min.js"></script>
<script src="registeredreport_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="registeredreport_files/libs/quarto-html/anchor.min.js"></script>
<link href="registeredreport_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="registeredreport_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="registeredreport_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="registeredreport_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="registeredreport_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction">Introduction</a></li>
  <li><a href="#methods" id="toc-methods" class="nav-link" data-scroll-target="#methods">Methods</a>
  <ul class="collapse">
  <li><a href="#literature-review" id="toc-literature-review" class="nav-link" data-scroll-target="#literature-review">Literature review</a></li>
  <li><a href="#initial-classification" id="toc-initial-classification" class="nav-link" data-scroll-target="#initial-classification">Initial classification</a></li>
  <li><a href="#assessing-open-data" id="toc-assessing-open-data" class="nav-link" data-scroll-target="#assessing-open-data">Assessing open data</a></li>
  <li><a href="#analysis" id="toc-analysis" class="nav-link" data-scroll-target="#analysis">Analysis</a></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results">Results</a>
  <ul class="collapse">
  <li><a href="#simulation" id="toc-simulation" class="nav-link" data-scroll-target="#simulation">Simulation</a></li>
  <li><a href="#preliminary-results" id="toc-preliminary-results" class="nav-link" data-scroll-target="#preliminary-results">Preliminary results</a></li>
  </ul></li>
  </ul>
</nav>
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Trends, challenges, and opportunities in open biologging data</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="introduction" class="level2">
<h2 class="anchored" data-anchor-id="introduction">Introduction</h2>
<p>(450/500 words)</p>
<p>Biologging is the practice of using animal-borne sensors to remotely observe animals and their environments<span class="citation" data-cites="rutz2009"><sup><a href="#ref-rutz2009" role="doc-biblioref">1</a></sup></span>. In the six decades since a biologist first attached a modified kitchen timer to a seal<span class="citation" data-cites="kooyman1966"><sup><a href="#ref-kooyman1966" role="doc-biblioref">2</a></sup></span>, biologging has become an invaluable, multidisciplinary tool to address questions in ecology<span class="citation" data-cites="wilmers2015"><sup><a href="#ref-wilmers2015" role="doc-biblioref">3</a></sup></span>, atmospheric science<span class="citation" data-cites="weinzierl2016 yonehara2016"><sup><a href="#ref-weinzierl2016" role="doc-biblioref">4</a>,<a href="#ref-yonehara2016" role="doc-biblioref">5</a></sup></span>, and oceanography<span class="citation" data-cites="treasure2017"><sup><a href="#ref-treasure2017" role="doc-biblioref">6</a></sup></span>. As biologging data continue to grow in size and complexity (e.g., millions of new records daily in Movebank<span class="citation" data-cites="kays2021"><sup><a href="#ref-kays2021" role="doc-biblioref">7</a></sup></span> and novel sensors like infrasound recorders<span class="citation" data-cites="denouden2021"><sup><a href="#ref-denouden2021" role="doc-biblioref">8</a></sup></span>), “big data” methods have been highlighted as a key direction for future developments<span class="citation" data-cites="nathan2022 grémillet2022"><sup><a href="#ref-nathan2022" role="doc-biblioref">9</a>,<a href="#ref-grémillet2022" role="doc-biblioref">10</a></sup></span>. Simultaneously, biologging data are increasingly used to inform conservation efforts across taxonomic groups<span class="citation" data-cites="beal2021 queiroz2019"><sup><a href="#ref-beal2021" role="doc-biblioref">11</a>,<a href="#ref-queiroz2019" role="doc-biblioref">12</a></sup></span> and regions<span class="citation" data-cites="hindell2020"><sup><a href="#ref-hindell2020" role="doc-biblioref">13</a></sup></span>. However, the envisioned impacts of aggregated biologging data on research (expanding modes and scope of inquiry) and conservation (providing robust evidence to guide policy and management) are limited by inconsistent standardization and discoverability, which has become a growing matter of practical and ethical concern<span class="citation" data-cites="rutz2022"><sup><a href="#ref-rutz2022" role="doc-biblioref">14</a></sup></span>. A systematic review of the instruments, species, and data availability of biologging studies is thus essential to support ongoing and future efforts to promote collaborative data sharing and curation efforts.</p>
<p>Open science practices, such as publishing data and code alongside manuscripts, improve research transparency and efficiency<span class="citation" data-cites="lowndes2017 hampton2017"><sup><a href="#ref-lowndes2017" role="doc-biblioref">15</a>,<a href="#ref-hampton2017" role="doc-biblioref">16</a></sup></span>. However, data “openness” is determined by multiple factors, and poor research data management and sharing practices are a major source of biodiversity data loss<span class="citation" data-cites="peterson2018"><sup><a href="#ref-peterson2018" role="doc-biblioref">17</a></sup></span>. Fortunately, the advent of the FAIR principles<span class="citation" data-cites="wilkinson2016"><sup><a href="#ref-wilkinson2016" role="doc-biblioref">18</a></sup></span> (<a href="#tbl-fair">Table&nbsp;1</a>) has provided a useful framework to facilitate open data sharing, with FAIR data enabling groundbreaking advances in synthesis, biodiversity, and conservation science<span class="citation" data-cites="halpern2020 fredston2023 roche2021 buxton2021"><sup><a href="#ref-halpern2020" role="doc-biblioref">19</a>–<a href="#ref-buxton2021" role="doc-biblioref">22</a></sup></span> .</p>
<p>There have been numerous efforts within the biologging research community to promote open data sharing practices. Essential cyberinfrastructure include domain-specific data repositories (Movebank<span class="citation" data-cites="kays2021"><sup><a href="#ref-kays2021" role="doc-biblioref">7</a></sup></span>, Seabird Tracking Database <a href="https://www.seabirdtracking.org">seabirdtracking.org</a>, Euromammal<span class="citation" data-cites="urbano2021"><sup><a href="#ref-urbano2021" role="doc-biblioref">23</a></sup></span>) and data standards<span class="citation" data-cites="sequeira2021 dettki2023 kranstauber2011"><sup><a href="#ref-sequeira2021" role="doc-biblioref">24</a>–<a href="#ref-kranstauber2011" role="doc-biblioref">26</a></sup></span> that facilitate adoption of the FAIR principles (<a href="#tbl-fair">Table&nbsp;1</a>).</p>
<div id="tbl-fair" class="anchored">
<table class="table">
<caption>Table&nbsp;1: The FAIR principles<span class="citation" data-cites="wilkinson2016"><sup><a href="#ref-wilkinson2016" role="doc-biblioref">18</a></sup></span>, as implemented in biologging cyberinfrastructure, support data sharing and improve research transparency and efficiency.</caption>
<colgroup>
<col style="width: 23%">
<col style="width: 23%">
<col style="width: 30%">
<col style="width: 23%">
</colgroup>
<thead>
<tr class="header">
<th>FAIR principle</th>
<th>Definition</th>
<th>Application to biologging</th>
<th>Example</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Findability</td>
<td>Data and metadata have a globally unique and persistent identifier (e.g., a digital object identifier, DOI) and are indexed in a searchable resource</td>
<td>Data repositories, like the Movebank Data Repository, improve data discoverability and may assign DOIs to data (avoiding issues with broken hyperlinks, for example)</td>
<td>A tracking dataset deposited in the Movebank Data Repository is findable by its permanent DOI or by searching the repository</td>
</tr>
<tr class="even">
<td>Accessibility</td>
<td>Data and metadata are retrievable by open and universal protocols, such as HTTP</td>
<td>Data repositories allow scientists to retrieve biologging data via a web browser or other open source tools</td>
<td>Publicly available data on Movebank may be downloaded via the website, API, or with the <code>move2</code> R package<span class="citation" data-cites="kranstauber2023"><sup><a href="#ref-kranstauber2023" role="doc-biblioref">27</a></sup></span></td>
</tr>
<tr class="odd">
<td>Interoperability</td>
<td>Data use formal and shared formats and vocabularies</td>
<td>Shared protocols like the Darwin Core standard<span class="citation" data-cites="wieczorek2012"><sup><a href="#ref-wieczorek2012" role="doc-biblioref">28</a></sup></span>, Movebank data model<span class="citation" data-cites="MPIAB2023"><sup><a href="#ref-MPIAB2023" role="doc-biblioref">29</a></sup></span>, and proposed bio-logging standardization framework<span class="citation" data-cites="sequeira2021"><sup><a href="#ref-sequeira2021" role="doc-biblioref">24</a></sup></span> reduce barriers to combining datasets and increase uptake within and across scientific disciplines</td>
<td>Location and environmental data from seal-borne biologgers harmonized to a standard netCDF format facilitated their reuse by oceanographers to study polar regions<span class="citation" data-cites="treasure2017"><sup><a href="#ref-treasure2017" role="doc-biblioref">6</a></sup></span></td>
</tr>
<tr class="even">
<td>Reusability</td>
<td>Data and metadata are richly described and reuse permissions are clearly defined</td>
<td>Data repositories and standards together capture essential context for biologging data and provide licensing options for data reuse</td>
<td>The Movebank Data Repository releases datasets under the CC0 license and the Movebank data model<span class="citation" data-cites="MPIAB2023"><sup><a href="#ref-MPIAB2023" role="doc-biblioref">29</a></sup></span> includes fields important for data reuse (e.g., whether the animal was relocated before release)</td>
</tr>
</tbody>
</table>
</div>
<p>Despite these biologging cyberinfrastructure developments, the general state of open data within the community remains unknown, and there are indications that a majority of tracking data remain inaccessible<span class="citation" data-cites="campbell2015 scarpignato2023"><sup><a href="#ref-campbell2015" role="doc-biblioref">30</a>,<a href="#ref-scarpignato2023" role="doc-biblioref">31</a></sup></span>. We will conduct a systematic review to quantify the diversity of instruments, species, and open data practices in biologging studies published 2007-2023. Our review will test the following three hypotheses:</p>
<p><strong>H1:</strong> Open biologging data practices are increasing over time.</p>
<p><strong>H2:</strong> Open biologging data practices vary by ecosystem (marine or terrestrial).</p>
<p><strong>H3:</strong> Open biologging data practices are greater for spatial biologging data than aspatial (e.g., accelerometer) data.</p>
<p>In addition to hypothesis testing, our review will provide the first biologging bibliometric database, itself a valuable form of cyberinfrastrucure for future systematic reviews and meta-analyses. Based on the results of our review, we will also provide a minimum information standard (<em>sensu</em> MIAPE<span class="citation" data-cites="taylor2007"><sup><a href="#ref-taylor2007" role="doc-biblioref">32</a></sup></span> for proteomics experiments and MIReAD<span class="citation" data-cites="rund2019"><sup><a href="#ref-rund2019" role="doc-biblioref">33</a></sup></span> for arthropod abundances) as a resource for authors, journal editors, and funding bodies to facilitate FAIR sharing of biologging data.</p>
</section>
<section id="methods" class="level2">
<h2 class="anchored" data-anchor-id="methods">Methods</h2>
<p>Biologging terminology is highly varied in the literature (e.g., tags, tracking devices, data loggers) and the distinction between biologging (i.e., instruments with sensors and memory) and biotelemetry (i.e., instruments that generate a signal for an external receiver) is inconsistent<span class="citation" data-cites="rutz2009 cooke2004"><sup><a href="#ref-rutz2009" role="doc-biblioref">1</a>,<a href="#ref-cooke2004" role="doc-biblioref">34</a></sup></span>. Furthermore, biologgers may be used in a range of contexts outside of ecology and evolutionary biology, including agricultural and medical research. This review will only consider biologging studies in ecology, evolutionary biology, or related fields (see Initial Classification).</p>
<p>We decided to exclude biotelemetry (e.g., VHF transmitters and acoustic pingers) based on the properties of the data collected. Biologging data can reasonably be considered self-contained, whereas biotelemetry data depends on the receiver network. This external dependency complicates the assessment of open data sharing, the primary goal of this review. We make an exception for satellite biotelemetry using the Argos network, because the Argos system provides data with properties more similar to biologgers.</p>
<section id="literature-review" class="level3">
<h3 class="anchored" data-anchor-id="literature-review">Literature review</h3>
<p>On 2023-08-21, we queried Web of Science Core Collection for papers published since 2007 related to biologging with the following query:</p>
<p><code>animal AND (biologg* OR bio-logg* OR biotelemetry OR electronic tag* OR satellite track* OR GPS telemetry OR satellite telemetry OR satellite transmit* OR GPS collar* OR depth recorder* OR accelerometer OR archival tag*)</code></p>
<p>This yielded 6654 papers. We limited results to Web of Science categories with 100+ papers we deemed relevant, leaving 4799 papers. The categories we included were: Ecology, Zoology, Marine Freshwater Biology, Biodiversity Conservation, Multidisciplinary Sciences, Environmental Sciences, Oceanography, Biology, Behavioral Sciences, Ornithology, Evolutionary Biology, and Fisheries. We excluded the following categories with 100+ papers: Agriculture Dairy Animal Science, Veterinary Sciences, Engineering Electrical Electronic, Physiology, Food Science Technology, Engineering Biomedical, Agriculture Multidisciplinary, Computer Science Interdisciplinary Applications, Instruments Instrumentation.</p>
</section>
<section id="initial-classification" class="level3">
<h3 class="anchored" data-anchor-id="initial-classification">Initial classification</h3>
<p>Every paper will be reviewed by two individuals to assess its relevance to this review, record the presence or absence of a data availability statement, and to categorize the study by sensor type, taxonomy, and ecosystem. Papers will be considered relevant if they satisfy the following two characteristics:</p>
<ol type="1">
<li>They involved the collection of previously unpublished biologging data. Our definition of biologging excludes telemetry devices (e.g., acoustic pingers and radiotransmitters) with the exception of devices that communicate with Argos satellites.</li>
<li>The biologgers were deployed on non-domesticated animals in unrestrained habitat. Studies involving domesticated animals (e.g., cattle), animals in managed care (e.g., a zoo or aquarium), or biomedical laboratories will be excluded.</li>
</ol>
<p>We will consider a paper to have a data availability statement (DAS) if it states how to access the data or explicitly states why the data are not available. Any of the following examples would be considered a DAS: “Data were deposited in a repository” with an accompanying DOI, “Data are available at a website” with an accompanying URL, “Data are available upon request”, or “Data withheld for the safety of the tracked animals”.</p>
<p>We will assign sensor types to one or more categories following Williams et al.<span class="citation" data-cites="williams2019"><sup><a href="#ref-williams2019" role="doc-biblioref">35</a></sup></span>: location (e.g., GPS, depth), intrinsic (e.g., accelerometer, internal temperature), or environment (e.g., ambient temperature, camera). Biologgers can combine sensors from multiple categories (e.g., a GPS-accelerometer collar).</p>
<p>We will resolve taxonomy to the species level using the Integrated Taxonomic Information System (www.itis.gov). Each study’s ecosystem will be classified as one or more of aquatic, marine, and terrestrial according to the IUCN Red List (https://www.iucnredlist.org).</p>
</section>
<section id="assessing-open-data" class="level3">
<h3 class="anchored" data-anchor-id="assessing-open-data">Assessing open data</h3>
<p>Following our initial classification, we will randomly sample 600 studies to assess the openness of biologging data. We will limit our sample to studies with a DAS, and we will stratify our sample by year, ecosystem, and sensor type. We will attempt to sample fifty studies in each combination of the following categories:</p>
<ul>
<li><p>Time period (early 2007-2015, middle 2016-2019, recent 2020-2023)</p></li>
<li><p>Ecosystem (marine, terrestrial)</p></li>
<li><p>Sensor type (spatial, aspatial)</p></li>
</ul>
<p>The sensor type of a study will be considered spatial if it includes a location sensor and aspatial if it includes an intrinsic or environmental sensor. Our preliminary results indicate spatial sensors are deployed in the vast majority of studies (93%), whereas aspatial sensors are deployed less frequently (35% of studies). Therefore, at the time of sampling, we will assign each paper to either the aspatial or spatial category and limit our assessment to the corresponding sensor data. For example, if we sample a study for the aspatial category that deployed GPS-accelerometer biologgers, we would exclude the GPS data and only assess the accelerometer data.</p>
<p>For each sampled paper, we will attempt to retrieve the biologging dataset underlying the study. If data are available upon request, we will email the corresponding author to request the data and, if we do not receive a response within two weeks, we will send one follow-up email. We will record whether each dataset satisfies the following properties:</p>
<ol type="1">
<li><p>( <span class="math inline">\(f\)</span> ) Do the data have a permanent identifier, such as a DOI? I.e., FAIR principle F1<span class="citation" data-cites="wilkinson2016"><sup><a href="#ref-wilkinson2016" role="doc-biblioref">18</a></sup></span>.</p></li>
<li><p>( <span class="math inline">\(a\)</span> ) Are the data retrievable via a standard communications protocol, such as HTTP? I.e., FAIR principle A1<span class="citation" data-cites="wilkinson2016"><sup><a href="#ref-wilkinson2016" role="doc-biblioref">18</a></sup></span>.</p></li>
<li><p>( <span class="math inline">\(b\)</span> ) Do the data contain biologging data (raw or processed), as opposed to tabular summaries?</p></li>
<li><p>( <span class="math inline">\(c\)</span> ) Are the data sufficiently complete to reproduce the analyses in the study? I.e., are all necessary data and metadata included within the dataset.</p></li>
</ol>
<p>If we cannot retrieve the dataset for a study, we will consider the study to not satisfy any of the four properties listed above.</p>
<p>To clarify our methods for spatial versus aspatial data, consider a hypothetical GPS-accelerometer biologging study sampled in the aspatial category. The authors deposited the GPS data in the Movebank Data Repository (which issues DOIs), but the accelerometer data are available upon request. We contact the author and they provide us with the raw accelerometer data, but they no longer have the files necessary to calibrate them. Then this study would be scored as <span class="math inline">\(f=0,a=0,b=1,c=0\)</span> because (1) the aspatial data do not have a DOI, (2) retrieving the aspatial data required contacting the author, (3) the accelerometer data were made available, and (4) without the calibration files we could not reproduce the analysis. Had this same study been sampled in the spatial category, we would have scored it as <span class="math inline">\(f=1,a=1,b=1,c=1\)</span> because (1) the spatial data have a DOI, (2) the data were retrievable via HTTP, (3) the GPS sensor data were available, and (4) all the (meta)data necessary for the analysis were present.</p>
</section>
<section id="analysis" class="level3">
<h3 class="anchored" data-anchor-id="analysis">Analysis</h3>
<p>Our independent variables will be: year of publication (<span class="math inline">\(t\)</span>), ecosystem (<span class="math inline">\(e\)</span>), and sensor type (<span class="math inline">\(s\)</span>) [see “Initial classification”]. Our dependent variables will be: the presence of a permanent identifier (<span class="math inline">\(f\)</span>), whether data are retrievable by a standard protocol (<span class="math inline">\(a\)</span>), whether data include biologging data (<span class="math inline">\(b\)</span>), and whether data are complete (<span class="math inline">\(c\)</span>) [see “Assessing open data”]. We will fit four Bayesian generalized linear models of the form:</p>
<p><span class="math display">\[
\begin{align}
DV_{i} &amp;\sim Bernoulli(p) \\
logit(p_i) &amp;= \alpha + \beta_e e_i + \beta_s s_i + \beta_t mo(t, \delta) \\
\alpha &amp;\sim Normal(-1, 2) \\
\beta_e, \beta_s &amp;\sim Normal(0, 1) \\
\beta_t &amp;\sim Normal(1, 2) \\
\delta &amp;\sim Dirichlet(1)
\end{align}
\]</span></p>
<p>Where <span class="math inline">\(DV\)</span> is the dependent variable (<span class="math inline">\(f\)</span>, <span class="math inline">\(a\)</span>, <span class="math inline">\(r\)</span>, or <span class="math inline">\(c\)</span>). <span class="math inline">\(e\)</span> and <span class="math inline">\(s\)</span> encode ecosystem (marine, terrestrial) and sensor type (aspatial, spatial), respectively. Year of publication (<span class="math inline">\(t\)</span>) is treated as an ordinal categorical predictor<span class="citation" data-cites="bürkner2020"><sup><a href="#ref-bürkner2020" role="doc-biblioref">36</a></sup></span>. Modeling <span class="math inline">\(t\)</span> this way, instead of as a metric continuous variable, allows the rate of change of the dependent variables to be nonlinear in time. <span class="math inline">\(\beta_t\)</span> is the coefficient for the effect of time and <span class="math inline">\(mo(t,\delta)\)</span> is a monotonic transformation of <span class="math inline">\(t\)</span> from 0 to 1. Thus <span class="math inline">\(\beta_t mo(t, \delta)\)</span> is 0 at the beginning of the time period (2007) and increases to <span class="math inline">\(\beta_t\)</span> at the end (2023). <span class="math inline">\(\delta\)</span> is a simplex that controls the incremental steps for each year in the time period.</p>
<p>We will set priors according to available information in the literature. To our knowledge, there is no quantitative information about differences in open data practices within biologging. Therefore, the priors for <span class="math inline">\(\beta_e\)</span>, <span class="math inline">\(\beta_s\)</span>, and <span class="math inline">\(\delta\)</span> will be uninformative (standard normal and symmetric Dirichlet with concentration 1). More generally in ecology, open data rates remain relatively low (~50%), but are increasing in recent years<span class="citation" data-cites="roche2022"><sup><a href="#ref-roche2022" role="doc-biblioref">37</a></sup></span>. Therefore we chose a prior for <span class="math inline">\(\alpha\)</span> with a negative expected value (<span class="math inline">\(Normal(-1, 2)\)</span>) and a prior for <span class="math inline">\(\beta_t\)</span> with a positive expected value (<span class="math inline">\(Normal(1, 2\)</span>).</p>
<p>We will use Stan<span class="citation" data-cites="Stan"><sup><a href="#ref-Stan" role="doc-biblioref">38</a></sup></span> and brms<span class="citation" data-cites="bürkner2018 bürkner2021"><sup><a href="#ref-bürkner2018" role="doc-biblioref">39</a>,<a href="#ref-bürkner2021" role="doc-biblioref">40</a></sup></span> for model fitting.</p>
<section id="h1-open-biologging-data-practices-are-increasing-over-time" class="level4">
<h4 class="anchored" data-anchor-id="h1-open-biologging-data-practices-are-increasing-over-time">H1: Open biologging data practices are increasing over time</h4>
<p>We will test hypothesis <strong>H1</strong> by comparing posterior predictions for 2023 and 2016. We will use 2016 as the baseline instead of the beginning of our study period (2007) to undertake a more stringent test of our hypothesis. Widespread adoption of open data policies by ecology and evolutionary biology journals began circa 2012-2014<span class="citation" data-cites="roche2015"><sup><a href="#ref-roche2015" role="doc-biblioref">41</a></sup></span> and the FAIR data principles were published in 2016<span class="citation" data-cites="wilkinson2016"><sup><a href="#ref-wilkinson2016" role="doc-biblioref">18</a></sup></span>. Therefore we consider 2016 a more meaningful point of comparison.</p>
</section>
<section id="h2-open-biologging-data-practices-vary-by-ecosystem" class="level4">
<h4 class="anchored" data-anchor-id="h2-open-biologging-data-practices-vary-by-ecosystem">H2: Open biologging data practices vary by ecosystem</h4>
<p>We will test hypothesis <strong>H2</strong> by contrasting posterior predictions for marine and terrestrial ecosystems for the year 2023.</p>
</section>
<section id="h3-open-biologging-data-practices-vary-by-sensor-class" class="level4">
<h4 class="anchored" data-anchor-id="h3-open-biologging-data-practices-vary-by-sensor-class">H3: Open biologging data practices vary by sensor class</h4>
<p>We will test hypothesis <strong>H3</strong> by contrasting posterior predictions for aspatial and spatial ecosystems for the year 2023.</p>
</section>
</section>
</section>
<section id="results" class="level2">
<h2 class="anchored" data-anchor-id="results">Results</h2>
<p>We conducted a simulation to demonstrate how we will conduct our analysis. We also present preliminary data from the “initial classification” step, which was necessary to design our protocol for the “assessing open data” step.</p>
<section id="simulation" class="level3">
<h3 class="anchored" data-anchor-id="simulation">Simulation</h3>
<p>We simulated resonable values for <span class="math inline">\(t\)</span>, <span class="math inline">\(e\)</span>, <span class="math inline">\(s\)</span> (independent variables), and <span class="math inline">\(f\)</span> (dependent variable) to demonstrate how we will perform our analysis. The full analysis will also include dependent variables <span class="math inline">\(a\)</span>, <span class="math inline">\(b\)</span>, and <span class="math inline">\(c\)</span>.</p>
<p>We set up our analysis by loading necessary packages and setting the seed.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(brms)</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(cowplot)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">1234</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>The following code simulates our independent and dependent variables. We simulate independent variables according to our stratified sampling plan (<code>biolog_grid</code>). Then we choose values for our parameters <span class="math inline">\(\delta\)</span> (<code>delta</code>), <span class="math inline">\(\beta_E\)</span> (<code>beta_E</code>), <span class="math inline">\(\beta_S\)</span> (<code>beta_S</code>), <span class="math inline">\(\beta_T\)</span> (<code>beta_T</code>), and <span class="math inline">\(\alpha\)</span> (<code>alpha</code>), which we use to simulate our dependent variable, <span class="math inline">\(f\)</span> (<code>F</code>). Generally, the probability of <span class="math inline">\(f\)</span> (<span class="math inline">\(p(f)\)</span>) increases over time, and <span class="math inline">\(p(f)\)</span> is greater for spatial sensors and terrestrial ecosystems than aspatial sensors and marine ecosystems (Fig. <a href="#fig-sim-data">Figure&nbsp;1</a>).</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Grid of independent variables</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>biolog_grid <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">era =</span> <span class="fu">c</span>(<span class="st">"early"</span>, <span class="st">"middle"</span>, <span class="st">"recent"</span>),</span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="fu">c</span>(<span class="st">"marine"</span>, <span class="st">"terrestrial"</span>),</span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="fu">c</span>(<span class="st">"aspatial"</span>, <span class="st">"spatial"</span>)</span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">slice</span>(<span class="fu">rep</span>(<span class="fu">seq</span>(<span class="fu">nrow</span>(.)), <span class="at">each =</span> <span class="dv">100</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">T =</span> <span class="fu">case_when</span>(</span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a>    era <span class="sc">==</span> <span class="st">"early"</span>  <span class="sc">~</span> <span class="fu">sample</span>(<span class="dv">2007</span><span class="sc">:</span><span class="dv">2015</span>, <span class="fu">nrow</span>(.), <span class="at">replace =</span> <span class="cn">TRUE</span>),</span>
<span id="cb2-10"><a href="#cb2-10" aria-hidden="true" tabindex="-1"></a>    era <span class="sc">==</span> <span class="st">"middle"</span> <span class="sc">~</span> <span class="fu">sample</span>(<span class="dv">2016</span><span class="sc">:</span><span class="dv">2019</span>, <span class="fu">nrow</span>(.), <span class="at">replace =</span> <span class="cn">TRUE</span>),</span>
<span id="cb2-11"><a href="#cb2-11" aria-hidden="true" tabindex="-1"></a>    era <span class="sc">==</span> <span class="st">"recent"</span> <span class="sc">~</span> <span class="fu">sample</span>(<span class="dv">2020</span><span class="sc">:</span><span class="dv">2023</span>, <span class="fu">nrow</span>(.), <span class="at">replace =</span> <span class="cn">TRUE</span>)</span>
<span id="cb2-12"><a href="#cb2-12" aria-hidden="true" tabindex="-1"></a>  ))</span>
<span id="cb2-13"><a href="#cb2-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-14"><a href="#cb2-14" aria-hidden="true" tabindex="-1"></a><span class="co"># Parameters for simulation</span></span>
<span id="cb2-15"><a href="#cb2-15" aria-hidden="true" tabindex="-1"></a><span class="co"># delta</span></span>
<span id="cb2-16"><a href="#cb2-16" aria-hidden="true" tabindex="-1"></a>delta <span class="ot">&lt;-</span> <span class="fu">c</span>(</span>
<span id="cb2-17"><a href="#cb2-17" aria-hidden="true" tabindex="-1"></a>  <span class="co"># dummy for 2007</span></span>
<span id="cb2-18"><a href="#cb2-18" aria-hidden="true" tabindex="-1"></a>  <span class="dv">0</span>,</span>
<span id="cb2-19"><a href="#cb2-19" aria-hidden="true" tabindex="-1"></a>  <span class="co"># slow increases in early period</span></span>
<span id="cb2-20"><a href="#cb2-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rep</span>(<span class="dv">1</span>, <span class="dv">8</span>),</span>
<span id="cb2-21"><a href="#cb2-21" aria-hidden="true" tabindex="-1"></a>  <span class="co"># fastest increases in middle period</span></span>
<span id="cb2-22"><a href="#cb2-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rep</span>(<span class="dv">3</span>, <span class="dv">4</span>),</span>
<span id="cb2-23"><a href="#cb2-23" aria-hidden="true" tabindex="-1"></a>  <span class="co"># moderate increases in recent period</span></span>
<span id="cb2-24"><a href="#cb2-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rep</span>(<span class="dv">2</span>, <span class="dv">4</span>)</span>
<span id="cb2-25"><a href="#cb2-25" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb2-26"><a href="#cb2-26" aria-hidden="true" tabindex="-1"></a>delta <span class="ot">&lt;-</span> delta <span class="sc">/</span> <span class="fu">sum</span>(delta)</span>
<span id="cb2-27"><a href="#cb2-27" aria-hidden="true" tabindex="-1"></a><span class="co"># betas (E, S, T)</span></span>
<span id="cb2-28"><a href="#cb2-28" aria-hidden="true" tabindex="-1"></a>beta_E <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="at">marine =</span> <span class="dv">0</span>, <span class="at">terrestrial =</span> <span class="fl">0.5</span>)</span>
<span id="cb2-29"><a href="#cb2-29" aria-hidden="true" tabindex="-1"></a>beta_S <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="at">aspatial =</span> <span class="dv">0</span>, <span class="at">spatial =</span> <span class="dv">1</span>)</span>
<span id="cb2-30"><a href="#cb2-30" aria-hidden="true" tabindex="-1"></a>beta_T <span class="ot">&lt;-</span> <span class="fl">2.5</span></span>
<span id="cb2-31"><a href="#cb2-31" aria-hidden="true" tabindex="-1"></a>alpha <span class="ot">&lt;-</span> <span class="sc">-</span><span class="dv">4</span></span>
<span id="cb2-32"><a href="#cb2-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-33"><a href="#cb2-33" aria-hidden="true" tabindex="-1"></a><span class="co"># Run simulation</span></span>
<span id="cb2-34"><a href="#cb2-34" aria-hidden="true" tabindex="-1"></a>inv_logit <span class="ot">&lt;-</span> \(x) <span class="fu">exp</span>(x) <span class="sc">/</span> (<span class="dv">1</span> <span class="sc">+</span> <span class="fu">exp</span>(x))</span>
<span id="cb2-35"><a href="#cb2-35" aria-hidden="true" tabindex="-1"></a>biolog <span class="ot">&lt;-</span> biolog_grid <span class="sc">%&gt;%</span></span>
<span id="cb2-36"><a href="#cb2-36" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">cumsum_delta =</span> <span class="fu">map_dbl</span>(T, \(t) <span class="fu">sum</span>(delta[<span class="dv">1</span><span class="sc">:</span>(t <span class="sc">-</span> <span class="dv">2007</span> <span class="sc">+</span> <span class="dv">1</span>)])),</span>
<span id="cb2-37"><a href="#cb2-37" aria-hidden="true" tabindex="-1"></a>         <span class="at">logit_p =</span> alpha <span class="sc">+</span> beta_E[E] <span class="sc">+</span> beta_S[S] <span class="sc">+</span> beta_T <span class="sc">*</span> cumsum_delta,</span>
<span id="cb2-38"><a href="#cb2-38" aria-hidden="true" tabindex="-1"></a>         <span class="at">p =</span> <span class="fu">inv_logit</span>(logit_p),</span>
<span id="cb2-39"><a href="#cb2-39" aria-hidden="true" tabindex="-1"></a>         <span class="at">F =</span> <span class="fu">rbinom</span>(<span class="fu">nrow</span>(.), <span class="at">size =</span> <span class="dv">1</span>, <span class="at">prob =</span> p))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Visualize data</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>e_palette <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="at">marine =</span> <span class="st">"navy"</span>, <span class="at">terrestrial =</span> <span class="st">"goldenrod"</span>)</span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a>biolog_text <span class="ot">&lt;-</span> <span class="fu">filter</span>(biolog, T <span class="sc">==</span> <span class="dv">2023</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">distinct</span>(E, S, p) <span class="sc">%&gt;%</span> </span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">label =</span> <span class="fu">str_to_title</span>(<span class="fu">paste</span>(E, S, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\n</span><span class="st">"</span>)))</span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(biolog, <span class="fu">aes</span>(T, p, <span class="at">color =</span> E)) <span class="sc">+</span></span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_jitter</span>(<span class="fu">aes</span>(<span class="at">y =</span> F, <span class="at">shape =</span> S), </span>
<span id="cb3-10"><a href="#cb3-10" aria-hidden="true" tabindex="-1"></a>              <span class="at">width =</span> <span class="fl">0.25</span>, <span class="at">height =</span> <span class="fl">0.05</span>,</span>
<span id="cb3-11"><a href="#cb3-11" aria-hidden="true" tabindex="-1"></a>              <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb3-12"><a href="#cb3-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="fu">aes</span>(<span class="at">linetype =</span> S)) <span class="sc">+</span></span>
<span id="cb3-13"><a href="#cb3-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text</span>(<span class="fu">aes</span>(<span class="at">label =</span> label), biolog_text,</span>
<span id="cb3-14"><a href="#cb3-14" aria-hidden="true" tabindex="-1"></a>            <span class="at">x =</span> <span class="fl">2023.1</span>, <span class="at">hjust =</span> <span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb3-15"><a href="#cb3-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> e_palette) <span class="sc">+</span></span>
<span id="cb3-16"><a href="#cb3-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="cn">NA</span>, <span class="dv">2025</span>), </span>
<span id="cb3-17"><a href="#cb3-17" aria-hidden="true" tabindex="-1"></a>                     <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">2010</span>, <span class="dv">2020</span>, <span class="at">by =</span> <span class="dv">5</span>)) <span class="sc">+</span></span>
<span id="cb3-18"><a href="#cb3-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">y =</span> <span class="fu">expression</span>(<span class="fu">p</span>(f))) <span class="sc">+</span></span>
<span id="cb3-19"><a href="#cb3-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">expand_limits</span>(<span class="at">x =</span> <span class="dv">2025</span>) <span class="sc">+</span></span>
<span id="cb3-20"><a href="#cb3-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb3-21"><a href="#cb3-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>,</span>
<span id="cb3-22"><a href="#cb3-22" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.title.x =</span> <span class="fu">element_blank</span>())</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="fig-sim-data" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="registeredreport_files/figure-html/fig-sim-data-1.png" style="width:120mm;height:120mm" class="figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;1: In our simulation, probability of biologging data having a DOI (<span class="math inline">\(p(f)\)</span>) increases over time. Lines represent <span class="math inline">\(p(f)\)</span>, points are simulated studies. <span class="math inline">\(p(f)\)</span> is greater for terrestrial (gold) than marine (blue) data, and greater for spatial (dashed lines, triangles) than aspatial (solid lines, circles) data.</figcaption>
</figure>
</div>
</div>
</div>
<p>Here we specify our model, including priors, and fit it to our simulated data. The simulation parameters are all within the 95% confidence intervals of the fitted model (Fig. <a href="#fig-param">Figure&nbsp;2</a>).</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Fit model</span></span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>biolog_prior <span class="ot">&lt;-</span> <span class="fu">c</span>(</span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_prior</span>(<span class="at">prior =</span> <span class="st">"normal(-1, 2)"</span>, <span class="at">class =</span> <span class="st">"Intercept"</span>),</span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_prior</span>(<span class="at">prior =</span> <span class="st">"normal(0, 1)"</span>, <span class="at">coef =</span> <span class="st">"Eterrestrial"</span>),</span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_prior</span>(<span class="at">prior =</span> <span class="st">"normal(0, 1)"</span>, <span class="at">coef =</span> <span class="st">"Sspatial"</span>),</span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_prior</span>(<span class="at">prior =</span> <span class="st">"normal(1, 2)"</span>, <span class="at">coef =</span> <span class="st">"moT"</span>),</span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_prior</span>(<span class="at">prior =</span> <span class="st">"dirichlet(1)"</span>, <span class="at">class =</span> <span class="st">"simo"</span>, <span class="at">coef =</span> <span class="st">"moT1"</span>)</span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a>biolog_mod <span class="ot">&lt;-</span> <span class="fu">brm</span>(F <span class="sc">~</span> E <span class="sc">+</span> S <span class="sc">+</span> <span class="fu">mo</span>(T),</span>
<span id="cb4-10"><a href="#cb4-10" aria-hidden="true" tabindex="-1"></a>                  <span class="at">data =</span> biolog,</span>
<span id="cb4-11"><a href="#cb4-11" aria-hidden="true" tabindex="-1"></a>                  <span class="at">family =</span> <span class="fu">bernoulli</span>(<span class="at">link =</span> <span class="st">"logit"</span>),</span>
<span id="cb4-12"><a href="#cb4-12" aria-hidden="true" tabindex="-1"></a>                  <span class="at">prior =</span> biolog_prior,</span>
<span id="cb4-13"><a href="#cb4-13" aria-hidden="true" tabindex="-1"></a>                  <span class="at">chains =</span> <span class="dv">4</span>,</span>
<span id="cb4-14"><a href="#cb4-14" aria-hidden="true" tabindex="-1"></a>                  <span class="at">iter =</span> <span class="dv">5000</span>,</span>
<span id="cb4-15"><a href="#cb4-15" aria-hidden="true" tabindex="-1"></a>                  <span class="at">seed =</span> <span class="dv">6789</span>,</span>
<span id="cb4-16"><a href="#cb4-16" aria-hidden="true" tabindex="-1"></a>                  <span class="at">refresh =</span> <span class="dv">0</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a>D <span class="ot">&lt;-</span> <span class="fu">length</span>(<span class="dv">2007</span><span class="sc">:</span><span class="dv">2023</span>) <span class="sc">-</span> <span class="dv">1</span></span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a>draws <span class="ot">&lt;-</span> <span class="fu">as_draws_df</span>(biolog_mod) <span class="sc">%&gt;%</span> </span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">b_T =</span> bsp_moT <span class="sc">*</span> D)</span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-5"><a href="#cb5-5" aria-hidden="true" tabindex="-1"></a>params <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"b_Intercept"</span>, <span class="st">"b_Eterrestrial"</span>, <span class="st">"b_Sspatial"</span>, <span class="st">"b_T"</span>)</span>
<span id="cb5-6"><a href="#cb5-6" aria-hidden="true" tabindex="-1"></a>dens_95 <span class="ot">&lt;-</span> <span class="fu">map</span>(params, \(param) {</span>
<span id="cb5-7"><a href="#cb5-7" aria-hidden="true" tabindex="-1"></a>  d <span class="ot">&lt;-</span> <span class="fu">density</span>(draws[[param]])</span>
<span id="cb5-8"><a href="#cb5-8" aria-hidden="true" tabindex="-1"></a>  cdf <span class="ot">&lt;-</span> <span class="fu">cumsum</span>(d<span class="sc">$</span>y <span class="sc">*</span> <span class="fu">mean</span>(<span class="fu">diff</span>(d<span class="sc">$</span>x)))</span>
<span id="cb5-9"><a href="#cb5-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tibble</span>(<span class="at">x =</span> d<span class="sc">$</span>x, <span class="at">y =</span> d<span class="sc">$</span>y, <span class="at">is_95 =</span> <span class="fu">between</span>(cdf, <span class="fl">0.025</span>, <span class="fl">0.975</span>))</span>
<span id="cb5-10"><a href="#cb5-10" aria-hidden="true" tabindex="-1"></a>}) <span class="sc">%&gt;%</span> </span>
<span id="cb5-11"><a href="#cb5-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_names</span>(params)</span>
<span id="cb5-12"><a href="#cb5-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-13"><a href="#cb5-13" aria-hidden="true" tabindex="-1"></a><span class="fu">plot_grid</span>(</span>
<span id="cb5-14"><a href="#cb5-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">plot_grid</span>(</span>
<span id="cb5-15"><a href="#cb5-15" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>(draws, <span class="fu">aes</span>(b_Intercept)) <span class="sc">+</span></span>
<span id="cb5-16"><a href="#cb5-16" aria-hidden="true" tabindex="-1"></a>      <span class="fu">geom_line</span>(<span class="fu">aes</span>(x, y), </span>
<span id="cb5-17"><a href="#cb5-17" aria-hidden="true" tabindex="-1"></a>                   <span class="at">data =</span> dens_95<span class="sc">$</span>b_Intercept, </span>
<span id="cb5-18"><a href="#cb5-18" aria-hidden="true" tabindex="-1"></a>                   <span class="at">color =</span> <span class="st">"darkblue"</span>, <span class="at">linewidth =</span> <span class="fl">1.5</span>) <span class="sc">+</span></span>
<span id="cb5-19"><a href="#cb5-19" aria-hidden="true" tabindex="-1"></a>      <span class="fu">geom_area</span>(<span class="fu">aes</span>(x, y),</span>
<span id="cb5-20"><a href="#cb5-20" aria-hidden="true" tabindex="-1"></a>                <span class="at">data =</span> <span class="fu">filter</span>(dens_95<span class="sc">$</span>b_Intercept, is_95),</span>
<span id="cb5-21"><a href="#cb5-21" aria-hidden="true" tabindex="-1"></a>                <span class="at">fill =</span> <span class="st">"darkblue"</span>,</span>
<span id="cb5-22"><a href="#cb5-22" aria-hidden="true" tabindex="-1"></a>                <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb5-23"><a href="#cb5-23" aria-hidden="true" tabindex="-1"></a>      <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> alpha, <span class="at">linewidth =</span> <span class="dv">2</span>, <span class="at">color =</span> <span class="st">"firebrick"</span>) <span class="sc">+</span></span>
<span id="cb5-24"><a href="#cb5-24" aria-hidden="true" tabindex="-1"></a>      <span class="fu">labs</span>(<span class="at">x =</span> <span class="fu">expression</span>(alpha)) <span class="sc">+</span></span>
<span id="cb5-25"><a href="#cb5-25" aria-hidden="true" tabindex="-1"></a>      <span class="fu">theme_classic</span>(),</span>
<span id="cb5-26"><a href="#cb5-26" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb5-27"><a href="#cb5-27" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>(draws, <span class="fu">aes</span>(b_Eterrestrial)) <span class="sc">+</span></span>
<span id="cb5-28"><a href="#cb5-28" aria-hidden="true" tabindex="-1"></a>      <span class="fu">geom_line</span>(<span class="fu">aes</span>(x, y), </span>
<span id="cb5-29"><a href="#cb5-29" aria-hidden="true" tabindex="-1"></a>                   <span class="at">data =</span> dens_95<span class="sc">$</span>b_Eterrestrial, </span>
<span id="cb5-30"><a href="#cb5-30" aria-hidden="true" tabindex="-1"></a>                   <span class="at">color =</span> <span class="st">"darkblue"</span>, <span class="at">linewidth =</span> <span class="fl">1.5</span>) <span class="sc">+</span></span>
<span id="cb5-31"><a href="#cb5-31" aria-hidden="true" tabindex="-1"></a>      <span class="fu">geom_area</span>(<span class="fu">aes</span>(x, y),</span>
<span id="cb5-32"><a href="#cb5-32" aria-hidden="true" tabindex="-1"></a>                <span class="at">data =</span> <span class="fu">filter</span>(dens_95<span class="sc">$</span>b_Eterrestrial, is_95),</span>
<span id="cb5-33"><a href="#cb5-33" aria-hidden="true" tabindex="-1"></a>                <span class="at">fill =</span> <span class="st">"darkblue"</span>,</span>
<span id="cb5-34"><a href="#cb5-34" aria-hidden="true" tabindex="-1"></a>                <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb5-35"><a href="#cb5-35" aria-hidden="true" tabindex="-1"></a>      <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> beta_E[<span class="dv">2</span>], <span class="at">linewidth =</span> <span class="dv">2</span>, <span class="at">color =</span> <span class="st">"firebrick"</span>) <span class="sc">+</span></span>
<span id="cb5-36"><a href="#cb5-36" aria-hidden="true" tabindex="-1"></a>      <span class="fu">labs</span>(<span class="at">x =</span> <span class="fu">expression</span>(beta[E])) <span class="sc">+</span></span>
<span id="cb5-37"><a href="#cb5-37" aria-hidden="true" tabindex="-1"></a>      <span class="fu">theme_classic</span>(),</span>
<span id="cb5-38"><a href="#cb5-38" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb5-39"><a href="#cb5-39" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>(draws, <span class="fu">aes</span>(b_Sspatial)) <span class="sc">+</span></span>
<span id="cb5-40"><a href="#cb5-40" aria-hidden="true" tabindex="-1"></a>      <span class="fu">geom_line</span>(<span class="fu">aes</span>(x, y), </span>
<span id="cb5-41"><a href="#cb5-41" aria-hidden="true" tabindex="-1"></a>                   <span class="at">data =</span> dens_95<span class="sc">$</span>b_Sspatial, </span>
<span id="cb5-42"><a href="#cb5-42" aria-hidden="true" tabindex="-1"></a>                   <span class="at">color =</span> <span class="st">"darkblue"</span>, <span class="at">linewidth =</span> <span class="fl">1.5</span>) <span class="sc">+</span></span>
<span id="cb5-43"><a href="#cb5-43" aria-hidden="true" tabindex="-1"></a>      <span class="fu">geom_area</span>(<span class="fu">aes</span>(x, y),</span>
<span id="cb5-44"><a href="#cb5-44" aria-hidden="true" tabindex="-1"></a>                <span class="at">data =</span> <span class="fu">filter</span>(dens_95<span class="sc">$</span>b_Sspatial, is_95),</span>
<span id="cb5-45"><a href="#cb5-45" aria-hidden="true" tabindex="-1"></a>                <span class="at">fill =</span> <span class="st">"darkblue"</span>,</span>
<span id="cb5-46"><a href="#cb5-46" aria-hidden="true" tabindex="-1"></a>                <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb5-47"><a href="#cb5-47" aria-hidden="true" tabindex="-1"></a>      <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> beta_S[<span class="dv">2</span>], <span class="at">linewidth =</span> <span class="dv">2</span>, <span class="at">color =</span> <span class="st">"firebrick"</span>) <span class="sc">+</span></span>
<span id="cb5-48"><a href="#cb5-48" aria-hidden="true" tabindex="-1"></a>      <span class="fu">labs</span>(<span class="at">x =</span> <span class="fu">expression</span>(beta[S])) <span class="sc">+</span></span>
<span id="cb5-49"><a href="#cb5-49" aria-hidden="true" tabindex="-1"></a>      <span class="fu">theme_classic</span>(),</span>
<span id="cb5-50"><a href="#cb5-50" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb5-51"><a href="#cb5-51" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>(draws, <span class="fu">aes</span>(b_T)) <span class="sc">+</span></span>
<span id="cb5-52"><a href="#cb5-52" aria-hidden="true" tabindex="-1"></a>      <span class="fu">geom_line</span>(<span class="fu">aes</span>(x, y), </span>
<span id="cb5-53"><a href="#cb5-53" aria-hidden="true" tabindex="-1"></a>                   <span class="at">data =</span> dens_95<span class="sc">$</span>b_T, </span>
<span id="cb5-54"><a href="#cb5-54" aria-hidden="true" tabindex="-1"></a>                   <span class="at">color =</span> <span class="st">"darkblue"</span>, <span class="at">linewidth =</span> <span class="fl">1.5</span>) <span class="sc">+</span></span>
<span id="cb5-55"><a href="#cb5-55" aria-hidden="true" tabindex="-1"></a>      <span class="fu">geom_area</span>(<span class="fu">aes</span>(x, y),</span>
<span id="cb5-56"><a href="#cb5-56" aria-hidden="true" tabindex="-1"></a>                <span class="at">data =</span> <span class="fu">filter</span>(dens_95<span class="sc">$</span>b_T, is_95),</span>
<span id="cb5-57"><a href="#cb5-57" aria-hidden="true" tabindex="-1"></a>                <span class="at">fill =</span> <span class="st">"darkblue"</span>,</span>
<span id="cb5-58"><a href="#cb5-58" aria-hidden="true" tabindex="-1"></a>                <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb5-59"><a href="#cb5-59" aria-hidden="true" tabindex="-1"></a>      <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> beta_T, <span class="at">linewidth =</span> <span class="dv">2</span>, <span class="at">color =</span> <span class="st">"firebrick"</span>) <span class="sc">+</span></span>
<span id="cb5-60"><a href="#cb5-60" aria-hidden="true" tabindex="-1"></a>      <span class="fu">labs</span>(<span class="at">x =</span> <span class="fu">expression</span>(beta[T])) <span class="sc">+</span></span>
<span id="cb5-61"><a href="#cb5-61" aria-hidden="true" tabindex="-1"></a>      <span class="fu">theme_classic</span>(),</span>
<span id="cb5-62"><a href="#cb5-62" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb5-63"><a href="#cb5-63" aria-hidden="true" tabindex="-1"></a>    <span class="at">nrow =</span> <span class="dv">2</span>, <span class="at">labels =</span> LETTERS[<span class="dv">1</span><span class="sc">:</span><span class="dv">4</span>]</span>
<span id="cb5-64"><a href="#cb5-64" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb5-65"><a href="#cb5-65" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb5-66"><a href="#cb5-66" aria-hidden="true" tabindex="-1"></a>  draws <span class="sc">%&gt;%</span> </span>
<span id="cb5-67"><a href="#cb5-67" aria-hidden="true" tabindex="-1"></a>    <span class="fu">as_tibble</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb5-68"><a href="#cb5-68" aria-hidden="true" tabindex="-1"></a>    <span class="fu">pivot_longer</span>(<span class="fu">starts_with</span>(<span class="st">"simo_moT1"</span>), </span>
<span id="cb5-69"><a href="#cb5-69" aria-hidden="true" tabindex="-1"></a>                 <span class="at">names_to =</span> <span class="st">"step"</span>, </span>
<span id="cb5-70"><a href="#cb5-70" aria-hidden="true" tabindex="-1"></a>                 <span class="at">values_to =</span> <span class="st">"delta"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb5-71"><a href="#cb5-71" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">i =</span> <span class="fu">as.numeric</span>(<span class="fu">str_extract</span>(step, <span class="st">"</span><span class="sc">\\</span><span class="st">[([0-9]+)</span><span class="sc">\\</span><span class="st">]"</span>, <span class="at">group =</span> <span class="dv">1</span>)),</span>
<span id="cb5-72"><a href="#cb5-72" aria-hidden="true" tabindex="-1"></a>           <span class="at">year =</span> <span class="fu">factor</span>((<span class="dv">2008</span><span class="sc">:</span><span class="dv">2023</span>)[i])) <span class="sc">%&gt;%</span> </span>
<span id="cb5-73"><a href="#cb5-73" aria-hidden="true" tabindex="-1"></a>    <span class="fu">group_by</span>(.draw) <span class="sc">%&gt;%</span> </span>
<span id="cb5-74"><a href="#cb5-74" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(<span class="at">delta =</span> <span class="fu">cumsum</span>(delta)) <span class="sc">%&gt;%</span> </span>
<span id="cb5-75"><a href="#cb5-75" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb5-76"><a href="#cb5-76" aria-hidden="true" tabindex="-1"></a>    <span class="fu">group_by</span>(year) <span class="sc">%&gt;%</span> </span>
<span id="cb5-77"><a href="#cb5-77" aria-hidden="true" tabindex="-1"></a>    <span class="fu">summarize</span>(<span class="at">delta_mean =</span> <span class="fu">mean</span>(delta),</span>
<span id="cb5-78"><a href="#cb5-78" aria-hidden="true" tabindex="-1"></a>              <span class="at">delta_lwr =</span> <span class="fu">quantile</span>(delta, <span class="fl">0.025</span>),</span>
<span id="cb5-79"><a href="#cb5-79" aria-hidden="true" tabindex="-1"></a>              <span class="at">delta_upr =</span> <span class="fu">quantile</span>(delta, <span class="fl">0.975</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb5-80"><a href="#cb5-80" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">x =</span> year, <span class="at">y =</span> delta_mean)) <span class="sc">+</span></span>
<span id="cb5-81"><a href="#cb5-81" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_pointrange</span>(<span class="fu">aes</span>(<span class="at">ymin =</span> delta_lwr, <span class="at">ymax =</span> delta_upr),</span>
<span id="cb5-82"><a href="#cb5-82" aria-hidden="true" tabindex="-1"></a>                    <span class="at">color =</span> <span class="st">"darkblue"</span>) <span class="sc">+</span></span>
<span id="cb5-83"><a href="#cb5-83" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_point</span>(<span class="at">data =</span> <span class="fu">tibble</span>(<span class="at">year =</span> <span class="fu">factor</span>(<span class="dv">2008</span><span class="sc">:</span><span class="dv">2023</span>), </span>
<span id="cb5-84"><a href="#cb5-84" aria-hidden="true" tabindex="-1"></a>                             <span class="at">delta_mean =</span> <span class="fu">cumsum</span>(delta[<span class="sc">-</span><span class="dv">1</span>])), </span>
<span id="cb5-85"><a href="#cb5-85" aria-hidden="true" tabindex="-1"></a>               <span class="at">color =</span> <span class="st">"firebrick"</span>, <span class="at">size =</span> <span class="dv">2</span>) <span class="sc">+</span></span>
<span id="cb5-86"><a href="#cb5-86" aria-hidden="true" tabindex="-1"></a>    <span class="fu">labs</span>(<span class="at">y =</span> <span class="fu">expression</span>(delta)) <span class="sc">+</span></span>
<span id="cb5-87"><a href="#cb5-87" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb5-88"><a href="#cb5-88" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">axis.title.x =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb5-89"><a href="#cb5-89" aria-hidden="true" tabindex="-1"></a>          <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">vjust =</span> <span class="fl">0.5</span>)),</span>
<span id="cb5-90"><a href="#cb5-90" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb5-91"><a href="#cb5-91" aria-hidden="true" tabindex="-1"></a>  <span class="at">ncol =</span> <span class="dv">2</span>, <span class="at">labels =</span> <span class="fu">c</span>(<span class="st">""</span>, <span class="st">"E"</span>)</span>
<span id="cb5-92"><a href="#cb5-92" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="fig-param" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="registeredreport_files/figure-html/fig-param-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;2: Posterior distributions of model parameters (blue) compared to simulation parameters (red). A-D: Intercept and coefficients for ecosystem, sensor type, and time. 95% interval shaded. E: Cumulative effect of year (<span class="math inline">\(mo(t,delta)\)</span>). Blue points and lines represent the mean and 95% CI of <span class="math inline">\(mo(t,delta)\)</span>.</figcaption>
</figure>
</div>
</div>
</div>
<p>We will test our hypotheses by calculating contrasts of our model’s posterior predictions (Fig. <a href="#fig-hypotheses">Figure&nbsp;3</a>). The simulation model identified support for all three hypotheses.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Hypothesis testing</span></span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a><span class="do">## H1</span></span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a><span class="do">### Calculate contrasts</span></span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a>h1_data_2016 <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="fu">c</span>(<span class="st">"marine"</span>, <span class="st">"terrestrial"</span>),</span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="fu">c</span>(<span class="st">"aspatial"</span>, <span class="st">"spatial"</span>),</span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a>  <span class="at">T =</span> <span class="dv">2016</span></span>
<span id="cb6-8"><a href="#cb6-8" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb6-9"><a href="#cb6-9" aria-hidden="true" tabindex="-1"></a>h1_pred_2016 <span class="ot">&lt;-</span> <span class="fu">posterior_predict</span>(biolog_mod,</span>
<span id="cb6-10"><a href="#cb6-10" aria-hidden="true" tabindex="-1"></a>                                  <span class="at">newdata =</span> h1_data_2016,</span>
<span id="cb6-11"><a href="#cb6-11" aria-hidden="true" tabindex="-1"></a>                                  <span class="at">ndraws =</span> <span class="dv">1000</span>)</span>
<span id="cb6-12"><a href="#cb6-12" aria-hidden="true" tabindex="-1"></a>h1_data_2023 <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb6-13"><a href="#cb6-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="fu">c</span>(<span class="st">"marine"</span>, <span class="st">"terrestrial"</span>),</span>
<span id="cb6-14"><a href="#cb6-14" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="fu">c</span>(<span class="st">"aspatial"</span>, <span class="st">"spatial"</span>),</span>
<span id="cb6-15"><a href="#cb6-15" aria-hidden="true" tabindex="-1"></a>  <span class="at">T =</span> <span class="dv">2023</span></span>
<span id="cb6-16"><a href="#cb6-16" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb6-17"><a href="#cb6-17" aria-hidden="true" tabindex="-1"></a>h1_pred_2023 <span class="ot">&lt;-</span> <span class="fu">posterior_predict</span>(biolog_mod,</span>
<span id="cb6-18"><a href="#cb6-18" aria-hidden="true" tabindex="-1"></a>                                  <span class="at">newdata =</span> h1_data_2023,</span>
<span id="cb6-19"><a href="#cb6-19" aria-hidden="true" tabindex="-1"></a>                                  <span class="at">ndraws =</span> <span class="dv">1000</span>)</span>
<span id="cb6-20"><a href="#cb6-20" aria-hidden="true" tabindex="-1"></a>h1_contrast <span class="ot">&lt;-</span> (h1_pred_2023 <span class="sc">-</span> h1_pred_2016) <span class="sc">%&gt;%</span> </span>
<span id="cb6-21"><a href="#cb6-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">t</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb6-22"><a href="#cb6-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cbind</span>(<span class="fu">select</span>(h1_data_2016, E, S), .) <span class="sc">%&gt;%</span> </span>
<span id="cb6-23"><a href="#cb6-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="sc">-</span><span class="fu">c</span>(E, S), <span class="at">names_to =</span> <span class="st">"draw"</span>, <span class="at">values_to =</span> <span class="st">"contrast"</span>)</span>
<span id="cb6-24"><a href="#cb6-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-25"><a href="#cb6-25" aria-hidden="true" tabindex="-1"></a><span class="do">## H2</span></span>
<span id="cb6-26"><a href="#cb6-26" aria-hidden="true" tabindex="-1"></a><span class="do">### Calculate contrasts</span></span>
<span id="cb6-27"><a href="#cb6-27" aria-hidden="true" tabindex="-1"></a>h2_data_marine <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb6-28"><a href="#cb6-28" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="st">"marine"</span>,</span>
<span id="cb6-29"><a href="#cb6-29" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="fu">c</span>(<span class="st">"aspatial"</span>, <span class="st">"spatial"</span>),</span>
<span id="cb6-30"><a href="#cb6-30" aria-hidden="true" tabindex="-1"></a>  <span class="at">T =</span> <span class="dv">2023</span></span>
<span id="cb6-31"><a href="#cb6-31" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb6-32"><a href="#cb6-32" aria-hidden="true" tabindex="-1"></a>h2_pred_marine <span class="ot">&lt;-</span> <span class="fu">posterior_predict</span>(biolog_mod,</span>
<span id="cb6-33"><a href="#cb6-33" aria-hidden="true" tabindex="-1"></a>                                    <span class="at">newdata =</span> h2_data_marine,</span>
<span id="cb6-34"><a href="#cb6-34" aria-hidden="true" tabindex="-1"></a>                                    <span class="at">ndraws =</span> <span class="dv">1000</span>)</span>
<span id="cb6-35"><a href="#cb6-35" aria-hidden="true" tabindex="-1"></a>h2_data_terrestrial <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb6-36"><a href="#cb6-36" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="st">"terrestrial"</span>,</span>
<span id="cb6-37"><a href="#cb6-37" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="fu">c</span>(<span class="st">"aspatial"</span>, <span class="st">"spatial"</span>),</span>
<span id="cb6-38"><a href="#cb6-38" aria-hidden="true" tabindex="-1"></a>  <span class="at">T =</span> <span class="dv">2023</span></span>
<span id="cb6-39"><a href="#cb6-39" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb6-40"><a href="#cb6-40" aria-hidden="true" tabindex="-1"></a>h2_pred_terrestrial <span class="ot">&lt;-</span> <span class="fu">posterior_predict</span>(biolog_mod,</span>
<span id="cb6-41"><a href="#cb6-41" aria-hidden="true" tabindex="-1"></a>                                         <span class="at">newdata =</span> h2_data_terrestrial,</span>
<span id="cb6-42"><a href="#cb6-42" aria-hidden="true" tabindex="-1"></a>                                         <span class="at">ndraws =</span> <span class="dv">1000</span>)</span>
<span id="cb6-43"><a href="#cb6-43" aria-hidden="true" tabindex="-1"></a>h2_contrast <span class="ot">&lt;-</span> (h2_pred_terrestrial <span class="sc">-</span> h2_pred_marine) <span class="sc">%&gt;%</span> </span>
<span id="cb6-44"><a href="#cb6-44" aria-hidden="true" tabindex="-1"></a>  <span class="fu">t</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb6-45"><a href="#cb6-45" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cbind</span>(<span class="fu">select</span>(h2_data_marine, S), .) <span class="sc">%&gt;%</span> </span>
<span id="cb6-46"><a href="#cb6-46" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="sc">-</span><span class="fu">c</span>(S), <span class="at">names_to =</span> <span class="st">"draw"</span>, <span class="at">values_to =</span> <span class="st">"contrast"</span>)</span>
<span id="cb6-47"><a href="#cb6-47" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-48"><a href="#cb6-48" aria-hidden="true" tabindex="-1"></a><span class="do">## H3</span></span>
<span id="cb6-49"><a href="#cb6-49" aria-hidden="true" tabindex="-1"></a><span class="do">### Calculate contrasts</span></span>
<span id="cb6-50"><a href="#cb6-50" aria-hidden="true" tabindex="-1"></a>h3_data_aspatial <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb6-51"><a href="#cb6-51" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="fu">c</span>(<span class="st">"marine"</span>, <span class="st">"terrestrial"</span>),</span>
<span id="cb6-52"><a href="#cb6-52" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="st">"aspatial"</span>,</span>
<span id="cb6-53"><a href="#cb6-53" aria-hidden="true" tabindex="-1"></a>  <span class="at">T =</span> <span class="dv">2023</span></span>
<span id="cb6-54"><a href="#cb6-54" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb6-55"><a href="#cb6-55" aria-hidden="true" tabindex="-1"></a>h3_pred_aspatial <span class="ot">&lt;-</span> <span class="fu">posterior_predict</span>(biolog_mod,</span>
<span id="cb6-56"><a href="#cb6-56" aria-hidden="true" tabindex="-1"></a>                                      <span class="at">newdata =</span> h3_data_aspatial,</span>
<span id="cb6-57"><a href="#cb6-57" aria-hidden="true" tabindex="-1"></a>                                      <span class="at">ndraws =</span> <span class="dv">1000</span>)</span>
<span id="cb6-58"><a href="#cb6-58" aria-hidden="true" tabindex="-1"></a>h3_data_spatial <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb6-59"><a href="#cb6-59" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="fu">c</span>(<span class="st">"marine"</span>, <span class="st">"terrestrial"</span>),</span>
<span id="cb6-60"><a href="#cb6-60" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="st">"spatial"</span>,</span>
<span id="cb6-61"><a href="#cb6-61" aria-hidden="true" tabindex="-1"></a>  <span class="at">T =</span> <span class="dv">2023</span></span>
<span id="cb6-62"><a href="#cb6-62" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb6-63"><a href="#cb6-63" aria-hidden="true" tabindex="-1"></a>h3_pred_spatial <span class="ot">&lt;-</span> <span class="fu">posterior_predict</span>(biolog_mod,</span>
<span id="cb6-64"><a href="#cb6-64" aria-hidden="true" tabindex="-1"></a>                                     <span class="at">newdata =</span> h3_data_spatial,</span>
<span id="cb6-65"><a href="#cb6-65" aria-hidden="true" tabindex="-1"></a>                                     <span class="at">ndraws =</span> <span class="dv">1000</span>)</span>
<span id="cb6-66"><a href="#cb6-66" aria-hidden="true" tabindex="-1"></a>h3_contrast <span class="ot">&lt;-</span> (h3_pred_spatial <span class="sc">-</span> h3_pred_aspatial) <span class="sc">%&gt;%</span> </span>
<span id="cb6-67"><a href="#cb6-67" aria-hidden="true" tabindex="-1"></a>  <span class="fu">t</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb6-68"><a href="#cb6-68" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cbind</span>(<span class="fu">select</span>(h3_data_aspatial, E), .) <span class="sc">%&gt;%</span> </span>
<span id="cb6-69"><a href="#cb6-69" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="sc">-</span>E, <span class="at">names_to =</span> <span class="st">"draw"</span>, <span class="at">values_to =</span> <span class="st">"contrast"</span>)</span>
<span id="cb6-70"><a href="#cb6-70" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-71"><a href="#cb6-71" aria-hidden="true" tabindex="-1"></a>mean_h1 <span class="ot">&lt;-</span> <span class="fu">mean</span>(h1_contrast<span class="sc">$</span>contrast) <span class="sc">*</span> <span class="dv">100</span></span>
<span id="cb6-72"><a href="#cb6-72" aria-hidden="true" tabindex="-1"></a>mean_h2 <span class="ot">&lt;-</span> <span class="fu">mean</span>(h2_contrast<span class="sc">$</span>contrast) <span class="sc">*</span> <span class="dv">100</span></span>
<span id="cb6-73"><a href="#cb6-73" aria-hidden="true" tabindex="-1"></a>mean_h3 <span class="ot">&lt;-</span> <span class="fu">mean</span>(h3_contrast<span class="sc">$</span>contrast) <span class="sc">*</span> <span class="dv">100</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="do">### Plot contrasts</span></span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a>h1_plot <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(h1_contrast, <span class="fu">aes</span>(contrast)) <span class="sc">+</span></span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">y =</span> <span class="fu">after_stat</span>(prop)), <span class="at">position =</span> <span class="st">"dodge"</span>) <span class="sc">+</span></span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> <span class="fu">mean</span>(h1_contrast<span class="sc">$</span>contrast), </span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a>             <span class="at">color =</span> <span class="st">"firebrick"</span>,</span>
<span id="cb7-6"><a href="#cb7-6" aria-hidden="true" tabindex="-1"></a>             <span class="at">linewidth =</span> <span class="dv">2</span>) <span class="sc">+</span></span>
<span id="cb7-7"><a href="#cb7-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">"text"</span>, </span>
<span id="cb7-8"><a href="#cb7-8" aria-hidden="true" tabindex="-1"></a>           <span class="at">x =</span> <span class="fu">mean</span>(h1_contrast<span class="sc">$</span>contrast) <span class="sc">+</span> <span class="fl">0.1</span>, <span class="at">y =</span> <span class="fl">0.75</span>,</span>
<span id="cb7-9"><a href="#cb7-9" aria-hidden="true" tabindex="-1"></a>           <span class="at">label =</span> <span class="fu">sprintf</span>(<span class="st">"Mean = %0.2f"</span>, <span class="fu">mean</span>(h1_contrast<span class="sc">$</span>contrast)),</span>
<span id="cb7-10"><a href="#cb7-10" aria-hidden="true" tabindex="-1"></a>           <span class="at">hjust =</span> <span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb7-11"><a href="#cb7-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">1</span>), <span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb7-12"><a href="#cb7-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Contrast of f (2023 vs 2016)"</span>,</span>
<span id="cb7-13"><a href="#cb7-13" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Proportion"</span>) <span class="sc">+</span></span>
<span id="cb7-14"><a href="#cb7-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span>
<span id="cb7-15"><a href="#cb7-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-16"><a href="#cb7-16" aria-hidden="true" tabindex="-1"></a>h2_plot <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(h2_contrast, <span class="fu">aes</span>(contrast)) <span class="sc">+</span></span>
<span id="cb7-17"><a href="#cb7-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">y =</span> <span class="fu">after_stat</span>(prop)), <span class="at">position =</span> <span class="st">"dodge"</span>) <span class="sc">+</span></span>
<span id="cb7-18"><a href="#cb7-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> <span class="fu">mean</span>(h2_contrast<span class="sc">$</span>contrast), </span>
<span id="cb7-19"><a href="#cb7-19" aria-hidden="true" tabindex="-1"></a>             <span class="at">color =</span> <span class="st">"firebrick"</span>,</span>
<span id="cb7-20"><a href="#cb7-20" aria-hidden="true" tabindex="-1"></a>             <span class="at">linewidth =</span> <span class="dv">2</span>) <span class="sc">+</span></span>
<span id="cb7-21"><a href="#cb7-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">"text"</span>, </span>
<span id="cb7-22"><a href="#cb7-22" aria-hidden="true" tabindex="-1"></a>           <span class="at">x =</span> <span class="fu">mean</span>(h2_contrast<span class="sc">$</span>contrast) <span class="sc">+</span> <span class="fl">0.1</span>, <span class="at">y =</span> <span class="fl">0.75</span>,</span>
<span id="cb7-23"><a href="#cb7-23" aria-hidden="true" tabindex="-1"></a>           <span class="at">label =</span> <span class="fu">sprintf</span>(<span class="st">"Mean = %0.2f"</span>, <span class="fu">mean</span>(h2_contrast<span class="sc">$</span>contrast)),</span>
<span id="cb7-24"><a href="#cb7-24" aria-hidden="true" tabindex="-1"></a>           <span class="at">hjust =</span> <span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb7-25"><a href="#cb7-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">1</span>), <span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb7-26"><a href="#cb7-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Contrast of f (Terrestrial vs marine)"</span>,</span>
<span id="cb7-27"><a href="#cb7-27" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Proportion"</span>) <span class="sc">+</span></span>
<span id="cb7-28"><a href="#cb7-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span>
<span id="cb7-29"><a href="#cb7-29" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-30"><a href="#cb7-30" aria-hidden="true" tabindex="-1"></a>h3_plot <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(h3_contrast, <span class="fu">aes</span>(contrast)) <span class="sc">+</span></span>
<span id="cb7-31"><a href="#cb7-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">y =</span> <span class="fu">after_stat</span>(prop)), <span class="at">position =</span> <span class="st">"dodge"</span>) <span class="sc">+</span></span>
<span id="cb7-32"><a href="#cb7-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> <span class="fu">mean</span>(h3_contrast<span class="sc">$</span>contrast), </span>
<span id="cb7-33"><a href="#cb7-33" aria-hidden="true" tabindex="-1"></a>             <span class="at">color =</span> <span class="st">"firebrick"</span>,</span>
<span id="cb7-34"><a href="#cb7-34" aria-hidden="true" tabindex="-1"></a>             <span class="at">linewidth =</span> <span class="dv">2</span>) <span class="sc">+</span></span>
<span id="cb7-35"><a href="#cb7-35" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="st">"text"</span>, </span>
<span id="cb7-36"><a href="#cb7-36" aria-hidden="true" tabindex="-1"></a>           <span class="at">x =</span> <span class="fu">mean</span>(h3_contrast<span class="sc">$</span>contrast) <span class="sc">+</span> <span class="fl">0.1</span>, <span class="at">y =</span> <span class="fl">0.75</span>,</span>
<span id="cb7-37"><a href="#cb7-37" aria-hidden="true" tabindex="-1"></a>           <span class="at">label =</span> <span class="fu">sprintf</span>(<span class="st">"Mean = %0.2f"</span>, <span class="fu">mean</span>(h3_contrast<span class="sc">$</span>contrast)),</span>
<span id="cb7-38"><a href="#cb7-38" aria-hidden="true" tabindex="-1"></a>           <span class="at">hjust =</span> <span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb7-39"><a href="#cb7-39" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">1</span>), <span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb7-40"><a href="#cb7-40" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Contrast of f (Spatial vs aspatial)"</span>,</span>
<span id="cb7-41"><a href="#cb7-41" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Proportion"</span>) <span class="sc">+</span></span>
<span id="cb7-42"><a href="#cb7-42" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span>
<span id="cb7-43"><a href="#cb7-43" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-44"><a href="#cb7-44" aria-hidden="true" tabindex="-1"></a><span class="fu">plot_grid</span>(h1_plot, h2_plot, h3_plot, <span class="at">ncol =</span> <span class="dv">2</span>, <span class="at">labels =</span> <span class="st">"AUTO"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="fig-hypotheses" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="registeredreport_files/figure-html/fig-hypotheses-1.png" style="width:180mm;height:180mm" class="figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;3: We determine support for our hypotheses through marginal contrasts of <span class="math inline">\(f\)</span>. A: Simulated studies in 2023 were on average 22% more likely to have a DOI than in 2016 (marginal over ecosystem and sensor type), providing support for hypothesis 1. B: Simulated terrestrial studies in 2023 were on average 15% more likely to have a DOI than marine studies (marginal over sensor type), providing support for hypothesis 2. C: Simulated spatial biologging data in 2023 were on average 25% more likely to have a DOI than aspatial data (marginal over ecosystem), providing support for hypothesis 3.</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="preliminary-results" class="level3">
<h3 class="anchored" data-anchor-id="preliminary-results">Preliminary results</h3>
<p>We have initiated the initial classification step, which will quantify our independent variables and is a prerequisite for sampling studies for the assessment of open data practices.</p>
<p>As of February 11, 2024, we have reviewed 1521 papers, 51 of which have been reviewed by two individuals. 63.5% of papers met our standards for relevance, 59.5% of papers presented novel biologging data, and 20.9% of papers contained a data availability statement.</p>
<p>Location sensors (e.g., GPS, depth, or altitude) were deployed most frequently, in 93.1% of relevant studies that presented novel biologging data. Intrinsic (e.g., accelerometers, heart rate monitors) and environmental (e.g., ambient temperature, cameras) sensors were deployed in 23.8% and 20.3% of studies, respectively.</p>
<p>We identified 1108 species that have been instrumented with biologgers. 99.1% of species were vertebrates. Most instrumented vertebrates were mammals (52.2%) or birds (26.1%). The most commonly instrumented mammals were carnivores (42.6%), ungulates (Perrisodactyla and non-cetacean Artiodatyla, 27.9%), and cetaceans (14.3%). Among birds, seabirds were instrumented most often (orders Charadriiformes, Sphenisciformes, Procellariiformes, Suliformes, and Pelecaniformes, 44.6%), followed by raptors (orders Accipitriformes, Falconiformes, and Strigiformes, 21.3%) and fowl (orders Anseriformes and Galliformes, 20.9%).</p>

</section>
</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body" data-line-spacing="2" role="list">
<div id="ref-rutz2009" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Rutz, C. &amp; Hays, G. C. <a href="https://doi.org/10.1098/rsbl.2009.0089">New frontiers in biologging science</a>. <em>Biology Letters</em> <strong>5</strong>, 289–292 (2009).</div>
</div>
<div id="ref-kooyman1966" class="csl-entry" role="listitem">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Kooyman, G. L. <a href="https://doi.org/10.1126/science.151.3717.1553">Maximum Diving Capacities of the Weddell Seal,<span><em>Leptonychotes weddelli</em></span></a>. <em>Science</em> <strong>151</strong>, 1553–1554 (1966).</div>
</div>
<div id="ref-wilmers2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Wilmers, C. C. <em>et al.</em> <a href="https://doi.org/10.1890/14-1401.1">The golden age of bio<span>-</span>logging: how animal<span>-</span>borne sensors are advancing the frontiers of ecology</a>. <em>Ecology</em> <strong>96</strong>, 1741–1753 (2015).</div>
</div>
<div id="ref-weinzierl2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Weinzierl, R. <em>et al.</em> <a href="https://doi.org/10.1002/ece3.2585">Wind estimation based on thermal soaring of birds</a>. <em>Ecology and Evolution</em> <strong>6</strong>, 8706–8718 (2016).</div>
</div>
<div id="ref-yonehara2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Yonehara, Y. <em>et al.</em> <a href="https://doi.org/10.1073/pnas.1523853113">Flight paths of seabirds soaring over the ocean surface enable measurement of fine-scale wind speed and direction</a>. <em>Proceedings of the National Academy of Sciences</em> <strong>113</strong>, 9039–9044 (2016).</div>
</div>
<div id="ref-treasure2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Treasure, A. <em>et al.</em> <a href="https://doi.org/10.5670/oceanog.2017.234">Marine mammals exploring the oceans pole to pole: A review of the MEOP consortium</a>. <em>Oceanography</em> <strong>30</strong>, 132–138 (2017).</div>
</div>
<div id="ref-kays2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Kays, R. <em>et al.</em> <a href="https://doi.org/10.1111/2041-210x.13767">The Movebank system for studying global animal movement and demography</a>. <em>Methods in Ecology and Evolution</em> <strong>13</strong>, 419–431 (2021).</div>
</div>
<div id="ref-denouden2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Ouden, O. F. C. den, Smets, P. S. M., Assink, J. D. &amp; Evers, L. G. <a href="https://doi.org/10.1029/2021gl094555">A Bird<span>’</span>s<span>-</span>Eye View on Ambient Infrasonic Soundscapes</a>. <em>Geophysical Research Letters</em> <strong>48</strong>, (2021).</div>
</div>
<div id="ref-nathan2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Nathan, R. <em>et al.</em> <a href="https://doi.org/10.1126/science.abg1780">Big-data approaches lead to an increased understanding of the ecology of animal movement</a>. <em>Science</em> <strong>375</strong>, (2022).</div>
</div>
<div id="ref-grémillet2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Grémillet, D., Chevallier, D. &amp; Guinet, C. <a href="https://doi.org/10.1093/icesjms/fsac059">Big data approaches to the spatial ecology and conservation of marine megafauna</a>. <em>ICES Journal of Marine Science</em> <strong>79</strong>, 975–986 (2022).</div>
</div>
<div id="ref-beal2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Beal, M. <em>et al.</em> <a href="https://doi.org/10.1126/sciadv.abd7225">Global political responsibility for the conservation of albatrosses and large petrels</a>. <em>Science Advances</em> <strong>7</strong>, (2021).</div>
</div>
<div id="ref-queiroz2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Queiroz, N. <em>et al.</em> <a href="https://doi.org/10.1038/s41586-019-1444-4">Global spatial risk assessment of sharks under the footprint of fisheries</a>. <em>Nature</em> <strong>572</strong>, 461–466 (2019).</div>
</div>
<div id="ref-hindell2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Hindell, M. A. <em>et al.</em> <a href="https://doi.org/10.1038/s41586-020-2126-y">Tracking of marine predators to protect Southern Ocean ecosystems</a>. <em>Nature</em> <strong>580</strong>, 87–92 (2020).</div>
</div>
<div id="ref-rutz2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Rutz, C. <a href="https://doi.org/10.1038/d41586-022-02821-6">Register animal-tracking tags to boost conservation</a>. <em>Nature</em> <strong>609</strong>, 221–221 (2022).</div>
</div>
<div id="ref-lowndes2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Lowndes, J. S. S. <em>et al.</em> <a href="https://doi.org/10.1038/s41559-017-0160">Our path to better science in less time using open data science tools</a>. <em>Nature Ecology &amp; Evolution</em> <strong>1</strong>, 1–7 (2017).</div>
</div>
<div id="ref-hampton2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Hampton, S. E. <em>et al.</em> <a href="https://doi.org/10.1093/biosci/bix025">Skills and Knowledge for Data-Intensive Environmental Research</a>. <em>BioScience</em> <strong>67</strong>, 546–557 (2017).</div>
</div>
<div id="ref-peterson2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">Peterson, A. T., Asase, A., Canhos, D., Souza, S. de &amp; Wieczorek, J. <a href="https://doi.org/10.3897/bdj.6.e26826">Data leakage and loss in biodiversity informatics</a>. <em>Biodiversity Data Journal</em> <strong>6</strong>, (2018).</div>
</div>
<div id="ref-wilkinson2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Wilkinson, M. D. <em>et al.</em> <a href="https://doi.org/10.1038/sdata.2016.18">The FAIR Guiding Principles for scientific data management and stewardship</a>. <em>Scientific Data</em> <strong>3</strong>, 160018 (2016).</div>
</div>
<div id="ref-halpern2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Halpern, B. S. <em>et al.</em> Ecological Synthesis and Its Role in Advancing Knowledge. <em>BioScience</em> (2020) doi:<a href="https://doi.org/10.1093/biosci/biaa105">10.1093/biosci/biaa105</a>.</div>
</div>
<div id="ref-fredston2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Fredston, A. L. &amp; Lowndes, J. S. S. <a href="https://doi.org/10.1146/annurev-marine-041723-094741">Welcoming More Participation in Open Data Science for the Oceans</a>. <em>Annual Review of Marine Science</em> <strong>16</strong>, (2023).</div>
</div>
<div id="ref-roche2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">Roche, D. G. <em>et al.</em> <a href="https://doi.org/10.1111/cobi.13835">Closing the knowledge<span>-</span>action gap in conservation with open science</a>. <em>Conservation Biology</em> <strong>36</strong>, (2021).</div>
</div>
<div id="ref-buxton2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Buxton, R. T. <em>et al.</em> <a href="https://doi.org/10.1111/csp2.329">Avoiding wasted research resources in conservation science</a>. <em>Conservation Science and Practice</em> <strong>3</strong>, (2021).</div>
</div>
<div id="ref-urbano2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Urbano, F. &amp; Cagnacci, F. <a href="https://doi.org/10.3389/fevo.2021.727023">Data management and sharing for collaborative science: Lessons learnt from the euromammals initiative</a>. <em>Frontiers in Ecology and Evolution</em> <strong>9</strong>, (2021).</div>
</div>
<div id="ref-sequeira2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Sequeira, A. M. M. <em>et al.</em> <a href="https://doi.org/10.1111/2041-210x.13593">A standardisation framework for bio<span>-</span>logging data to advance ecological research and conservation</a>. <em>Methods in Ecology and Evolution</em> <strong>12</strong>, 996–1007 (2021).</div>
</div>
<div id="ref-dettki2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">25. </div><div class="csl-right-inline">Dettki, H., Arlt, D., Bäckman, J. &amp; Blanchet, M. <a href="https://doi.org/10.3897/biss.7.111438">Key-value pairs and NoSQL databases: A novel concept to manage biologging data in data repositories</a>. <em>Biodiversity Information Science and Standards</em> <strong>7</strong>, (2023).</div>
</div>
<div id="ref-kranstauber2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Kranstauber, B. <em>et al.</em> <a href="https://doi.org/10.1016/j.envsoft.2010.12.005">The Movebank data model for animal tracking</a>. <em>Environmental Modelling &amp; Software</em> <strong>26</strong>, 834–835 (2011).</div>
</div>
<div id="ref-kranstauber2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Kranstauber, B., Safi, K. &amp; Scharf, A. K. <em><a href="https://CRAN.R-project.org/package=move2">move2: Processing and analysing animal trajectories</a></em>. (2023).</div>
</div>
<div id="ref-wieczorek2012" class="csl-entry" role="listitem">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Wieczorek, J. <em>et al.</em> <a href="https://doi.org/10.1371/journal.pone.0029715">Darwin Core: an evolving community-developed biodiversity data standard</a>. <em>PLoS ONE</em> <strong>7</strong>, e29715 (2012).</div>
</div>
<div id="ref-MPIAB2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">Animal Behavior, M. P. I. of. <a href="http://dx.doi.org/10.1038/s41597-019-0042-5">Movebank Attribute Dictionary</a>.</div>
</div>
<div id="ref-campbell2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">30. </div><div class="csl-right-inline">Campbell, H. A. <em>et al.</em> <a href="https://doi.org/10.1016/j.scitotenv.2015.01.089">Finding our way: On the sharing and reuse of animal telemetry data in Australasia</a>. <em>Science of The Total Environment</em> <strong>534</strong>, 79–84 (2015).</div>
</div>
<div id="ref-scarpignato2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">31. </div><div class="csl-right-inline">Scarpignato, A. L. <em>et al.</em> <a href="https://doi.org/10.1016/j.biocon.2023.110224">Shortfalls in tracking data available to inform North American migratory bird conservation</a>. <em>Biological Conservation</em> <strong>286</strong>, 110224 (2023).</div>
</div>
<div id="ref-taylor2007" class="csl-entry" role="listitem">
<div class="csl-left-margin">32. </div><div class="csl-right-inline">Taylor, C. F. <em>et al.</em> <a href="https://doi.org/10.1038/nbt1329">The minimum information about a proteomics experiment (MIAPE)</a>. <em>Nature Biotechnology</em> <strong>25</strong>, 887–893 (2007).</div>
</div>
<div id="ref-rund2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">33. </div><div class="csl-right-inline">Rund, S. S. C. <em>et al.</em> <a href="https://doi.org/10.1038/s41597-019-0042-5">MIReAD, a minimum information standard for reporting arthropod abundance data</a>. <em>Scientific Data</em> <strong>6</strong>, (2019).</div>
</div>
<div id="ref-cooke2004" class="csl-entry" role="listitem">
<div class="csl-left-margin">34. </div><div class="csl-right-inline">Cooke, S. J. <em>et al.</em> <a href="https://doi.org/10.1016/j.tree.2004.04.003">Biotelemetry: a mechanistic approach to ecology</a>. <em>Trends in Ecology &amp; Evolution</em> <strong>19</strong>, 334–343 (2004).</div>
</div>
<div id="ref-williams2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">35. </div><div class="csl-right-inline">Williams, H. J. <em>et al.</em> <a href="https://doi.org/10.1111/1365-2656.13094">Optimizing the use of biologgers for movement ecology research</a>. <em>Journal of Animal Ecology</em> <strong>89</strong>, 186–206 (2019).</div>
</div>
<div id="ref-bürkner2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">36. </div><div class="csl-right-inline">Bürkner, P.-C. &amp; Charpentier, E. <a href="https://doi.org/10.1111/bmsp.12195">Modelling monotonic effects of ordinal predictors in Bayesian regression models</a>. <em>British Journal of Mathematical and Statistical Psychology</em> <strong>73</strong>, 420–451 (2020).</div>
</div>
<div id="ref-roche2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">37. </div><div class="csl-right-inline">Roche, D. G. <em>et al.</em> <a href="https://doi.org/10.1098/rspb.2021.2780">Slow improvement to the archiving quality of open datasets shared by researchers in ecology and evolution</a>. <em>Proceedings of the Royal Society B: Biological Sciences</em> <strong>289</strong>, (2022).</div>
</div>
<div id="ref-Stan" class="csl-entry" role="listitem">
<div class="csl-left-margin">38. </div><div class="csl-right-inline">Stan Development Team. <a href="https://mc-stan.org">Stan modeling language users guide and reference manual, version 2.34</a>. (2024).</div>
</div>
<div id="ref-bürkner2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">39. </div><div class="csl-right-inline">Bürkner, P.-C. <a href="https://doi.org/10.32614/rj-2018-017">Advanced Bayesian Multilevel Modeling with the R Package brms</a>. <em>The R Journal</em> <strong>10</strong>, 395 (2018).</div>
</div>
<div id="ref-bürkner2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">40. </div><div class="csl-right-inline">Bürkner, P.-C. <a href="https://doi.org/10.18637/jss.v100.i05">Bayesian Item Response Modeling in <span><em>R</em></span> with <span><strong>brms</strong></span> and <span><em>Stan</em></span></a>. <em>Journal of Statistical Software</em> <strong>100</strong>, (2021).</div>
</div>
<div id="ref-roche2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">41. </div><div class="csl-right-inline">Roche, D. G., Kruuk, L. E. B., Lanfear, R. &amp; Binning, S. A. <a href="https://doi.org/10.1371/journal.pbio.1002295">Public Data Archiving in Ecology and Evolution: How Well Are We Doing?</a> <em>PLOS Biology</em> <strong>13</strong>, e1002295 (2015).</div>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>