<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.433">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Trends, challenges, and opportunities in open biologging data</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="registeredreport_files/libs/clipboard/clipboard.min.js"></script>
<script src="registeredreport_files/libs/quarto-html/quarto.js"></script>
<script src="registeredreport_files/libs/quarto-html/popper.min.js"></script>
<script src="registeredreport_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="registeredreport_files/libs/quarto-html/anchor.min.js"></script>
<link href="registeredreport_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="registeredreport_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="registeredreport_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="registeredreport_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="registeredreport_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Trends, challenges, and opportunities in open biologging data</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="introduction-470500-words" class="level2">
<h2 class="anchored" data-anchor-id="introduction-470500-words">Introduction (470/500 words)</h2>
<p>Biologging is the practice of using animal-borne sensors to remotely observe animals and their environments<span class="citation" data-cites="rutz2009"><sup><a href="#ref-rutz2009" role="doc-biblioref">1</a></sup></span>. In the six decades since a biologist first attached a modified kitchen timer to a seal<span class="citation" data-cites="kooyman1966"><sup><a href="#ref-kooyman1966" role="doc-biblioref">2</a></sup></span>, biologging has become an invaluable, multidisciplinary tool used for ecology<span class="citation" data-cites="wilmers2015"><sup><a href="#ref-wilmers2015" role="doc-biblioref">3</a></sup></span>, atmospheric science<span class="citation" data-cites="weinzierl2016 yonehara2016"><sup><a href="#ref-weinzierl2016" role="doc-biblioref">4</a>,<a href="#ref-yonehara2016" role="doc-biblioref">5</a></sup></span>, and oceanography<span class="citation" data-cites="treasure2017"><sup><a href="#ref-treasure2017" role="doc-biblioref">6</a></sup></span>. As biologging data continue to grow in size and complexity (e.g., millions of new records daily in Movebank<span class="citation" data-cites="kays2021"><sup><a href="#ref-kays2021" role="doc-biblioref">7</a></sup></span> and novel sensors like infrasound recorders<span class="citation" data-cites="denouden2021"><sup><a href="#ref-denouden2021" role="doc-biblioref">8</a></sup></span>), “big data” methods have been highlighted as a key direction for future developments<span class="citation" data-cites="nathan2022 grémillet2022"><sup><a href="#ref-nathan2022" role="doc-biblioref">9</a>,<a href="#ref-grémillet2022" role="doc-biblioref">10</a></sup></span>. Simultaneously, biologging data are increasingly used to inform conservation efforts across taxonomic groups<span class="citation" data-cites="beal2021 queiroz2019"><sup><a href="#ref-beal2021" role="doc-biblioref">11</a>,<a href="#ref-queiroz2019" role="doc-biblioref">12</a></sup></span> and regions<span class="citation" data-cites="hindell2020"><sup><a href="#ref-hindell2020" role="doc-biblioref">13</a></sup></span>. Yet, for aggregated biologging data to achieve envisioned impacts on research and conservation, increased access is a matter of practical and ethical concern.</p>
<p>Open science practices, such as publishing data and code alongside manuscripts, improve research transparency and efficiency<span class="citation" data-cites="lowndes2017 hampton2017"><sup><a href="#ref-lowndes2017" role="doc-biblioref">14</a>,<a href="#ref-hampton2017" role="doc-biblioref">15</a></sup></span>. However, data “openness” is determined by multiple factors, and poor research data management and sharing practices are a major source of biodiversity data loss<span class="citation" data-cites="peterson2018"><sup><a href="#ref-peterson2018" role="doc-biblioref">16</a></sup></span>. Fortunately, the advent of the FAIR principles<span class="citation" data-cites="wilkinson2016"><sup><a href="#ref-wilkinson2016" role="doc-biblioref">17</a></sup></span> (<a href="#tbl-fair">Table&nbsp;1</a>) has provided a useful framework to facilitate open data sharing, with FAIR data enabling groundbreaking advances in synthesis, biodiversity, and conservation science<span class="citation" data-cites="halpern2020 fredston2023 roche2021 buxton2021"><sup><a href="#ref-halpern2020" role="doc-biblioref">18</a>–<a href="#ref-buxton2021" role="doc-biblioref">21</a></sup></span> .</p>
<p>There have been numerous efforts within the biologging research community to promote open data sharing practices. Essential cyberinfrastructure include domain-specific data repositories (Movebank<span class="citation" data-cites="kays2021"><sup><a href="#ref-kays2021" role="doc-biblioref">7</a></sup></span>, Seabird Tracking Database <a href="https://www.seabirdtracking.org">seabirdtracking.org</a>, Euromammal<span class="citation" data-cites="urbano2021"><sup><a href="#ref-urbano2021" role="doc-biblioref">22</a></sup></span>) and data standards<span class="citation" data-cites="sequeira2021 dettki2023 kranstauber2011"><sup><a href="#ref-sequeira2021" role="doc-biblioref">23</a>–<a href="#ref-kranstauber2011" role="doc-biblioref">25</a></sup></span> that facilitate adoption of the FAIR principles (<a href="#tbl-fair">Table&nbsp;1</a>).</p>
<div id="tbl-fair" class="anchored">
<table class="table">
<caption>Table&nbsp;1: The FAIR principles, as implemented in biologging cyberinfrastructure, support data sharing and improve research transparency and efficiency.</caption>
<colgroup>
<col style="width: 19%">
<col style="width: 19%">
<col style="width: 34%">
<col style="width: 26%">
</colgroup>
<thead>
<tr class="header">
<th>FAIR principle</th>
<th>Definition</th>
<th>Application to biologging</th>
<th>Example</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Findability</td>
<td>Data and metadata have a globally unique and persistent identifier (e.g., a digital object identifier, DOI) and are indexed in a searchable resource</td>
<td>Data repositories, like the Movebank Data Repository, improve data discoverability and may assign DOIs to data (avoiding issues with broken hyperlinks, for example)</td>
<td>A tracking dataset deposited in the Movebank Data Repository is findable by its permanent DOI or by searching the repository.</td>
</tr>
<tr class="even">
<td>Accessibility</td>
<td>Data and metadata are retrievable by open and universal protocols, such as HTTP</td>
<td>Data repositories allow scientists to retrieve biologging data via a web browser or other open source tools</td>
<td>Publicly available data on Movebank may be downloaded via the website, API, or with the <code>move2</code> R package<span class="citation" data-cites="kranstauber2023"><sup><a href="#ref-kranstauber2023" role="doc-biblioref">26</a></sup></span></td>
</tr>
<tr class="odd">
<td>Interoperability</td>
<td>Data use formal and shared formats and vocabularies</td>
<td>Shared protocols like the Darwin Core standard<span class="citation" data-cites="wieczorek2012"><sup><a href="#ref-wieczorek2012" role="doc-biblioref">27</a></sup></span>, Movebank data model<span class="citation" data-cites="MPIAB2023"><sup><a href="#ref-MPIAB2023" role="doc-biblioref">28</a></sup></span>, and proposed bio-logging standardization framework<span class="citation" data-cites="sequeira2021"><sup><a href="#ref-sequeira2021" role="doc-biblioref">23</a></sup></span> reduce barriers to combining datasets and increase uptake within and across scientific disciplines</td>
<td>Location and environmental data from seal-borne biologgers harmonized to a standard netCDF format facilitated their reuse by oceanographers to study polar regions<span class="citation" data-cites="treasure2017"><sup><a href="#ref-treasure2017" role="doc-biblioref">6</a></sup></span></td>
</tr>
<tr class="even">
<td>Reusability</td>
<td>Data and metadata are richly described and reuse permissions are clearly defined</td>
<td>Data repositories and standards together capture essential context for biologging data and provide licensing options for data reuse</td>
<td>The Movebank Data Repository releases datasets under the CC0 license and the Movebank data model<span class="citation" data-cites="MPIAB2023"><sup><a href="#ref-MPIAB2023" role="doc-biblioref">28</a></sup></span> includes fields important for data reuse, e.g., whether the animal was relocated before release.</td>
</tr>
</tbody>
</table>
</div>
<p>Despite these biologging cyberinfrastructure developments, the general state of open data within the community remains unknown, and there are indications that a majority of tracking data remain inaccessible<span class="citation" data-cites="campbell2015 scarpignato2023"><sup><a href="#ref-campbell2015" role="doc-biblioref">29</a>,<a href="#ref-scarpignato2023" role="doc-biblioref">30</a></sup></span>. We will conduct a systematic review of the biologging literature to quantify open data practices, including open data (re)use and sharing by authors of biologging studies published between 2007-2023 (N=4799). The review will quantify the context of biologging studies (e.g., instrumented species and sensor types) and test the following four hypotheses:</p>
<p><strong>H1:</strong> Open biologging data practices are increasing over time.</p>
<p><strong>H2:</strong> Overall rates of open biologging data practices remain low.</p>
<p><strong>H3:</strong> Biologging studies use open data from external sources more than they generate open data.</p>
<p><strong>H4:</strong> Open biologging data practices vary by taxa, habitat, and spatial vs aspatial (e.g., accelerometer) data.</p>
<p>In addition to hypothesis testing, our review will provide the first biologging bibliometric database, itself a valuable form of cyberinfrastrucure for future systematic reviews and meta-analyses. Based on the results of our review, we will also provide a set of minimum reporting standards (<em>sensu</em> MIAPE<span class="citation" data-cites="taylor2007"><sup><a href="#ref-taylor2007" role="doc-biblioref">31</a></sup></span> for proteomics experiments and MIReAD<span class="citation" data-cites="rund2019"><sup><a href="#ref-rund2019" role="doc-biblioref">32</a></sup></span> for arthropod abundances) as a resource for authors, journal editors and funding bodies, to facilitate FAIR sharing of biologging data.</p>
</section>
<section id="methods" class="level2">
<h2 class="anchored" data-anchor-id="methods">Methods</h2>
<section id="literature-review" class="level3">
<h3 class="anchored" data-anchor-id="literature-review">Literature review</h3>
<p>On such and such a date we searched Web of Science…</p>
</section>
<section id="initial-classification" class="level3">
<h3 class="anchored" data-anchor-id="initial-classification">Initial classification</h3>
<p>During the broad-scale review, a minimum of two reviewers collected high-level information for each of the 4,799 selected articles. At this stage, article relevance was confirmed, and articles that discussed topics outside of this review (e.g., biotelemetry, medical applications) were deemed unrelated. Reviewers noted if the article presented novel biologging data, and if not, additional high-level information from the article was omitted. Related articles presenting newly collected biologging data went under further review.</p>
<p>Related articles were categorized as either a study (i.e., new research), review, methods paper, perspective, or data release paper. Additional high-level information for related articles included information about the instrumented species, biologging device, and data use and availability. For the instrumented species, reviewers recorded if the animal was domesticated, captive, or part of a wild population. Reviewers also recorded if the associated habitat of the species was most regularly terrestrial, aquatic, or marine. For the device, reviewers noted the type of information that the biologger recorded. This was separated into three categories: location (e.g., GPS, Argos, geolocation, depth, altitude), intrinsic (e.g., accelerometer, magnetometer, gyroscope heart rate, Hall sensor, stomach temperature, neurological sensors, Pitot tubes, speed sensors, internal body temperature), and environment (e.g., ambient temperature, video, audio, salinity, fluorescence, proximity sensors to other tagged animals, light). Lastly, reviewers recorded information about the data used and/or collected in the article. It was noted if the article used external non-biologging data (e.g., publicly available sea surface temperatures across the study domain) for their analysis and if the article included a biologging data availability statement. At this time, it was only noted if some language about the biologging data’s availability was included, and no other metrics were used to assess the data’s findability, accessibility, interoperability, and/or reusability. The last piece of information recorded during the initial classification was if the text “conserv*” appeared in the article, and specifically within the context of conservation.</p>
<p>The article type, instrumented species, biologging device, data use and availability, and conservation application data collected during the broad-scale review was used to ensure that a diverse and comprehensive pool of sample papers were selected for the fine-scale review.</p>
</section>
<section id="sampling-plan" class="level3">
<h3 class="anchored" data-anchor-id="sampling-plan">Sampling plan</h3>
<p>Stratified sampling across:</p>
<ul>
<li><p>Time (early 2007-2015, middle 2016-2019, recent 2020-2023)</p></li>
<li><p>Taxa (mammal, bird, fish, other)</p></li>
<li><p>Habitat (marine, terrestrial)</p></li>
<li><p>Spatial vs aspatial</p></li>
</ul>
</section>
<section id="open-data-assessment" class="level3">
<h3 class="anchored" data-anchor-id="open-data-assessment">Open data assessment</h3>
<p>Describe the fine-scale rubric (FAIR principle compliance,</p>
</section>
<section id="analysis-plan" class="level3">
<h3 class="anchored" data-anchor-id="analysis-plan">Analysis plan</h3>
<p>We’ll have multiple binary response variables (F1, F4, A1.1…), which we expect will be correlated (e.g., it’s likely <span class="math inline">\(p(F1|F4) &gt; p(F1)\)</span>). Generalized estimating equations<span class="citation" data-cites="liang1986"><sup><a href="#ref-liang1986" role="doc-biblioref">33</a></sup></span>, as implemented in <code>geepack</code><span class="citation" data-cites="halekoh2006"><sup><a href="#ref-halekoh2006" role="doc-biblioref">34</a></sup></span>, are suitable for this type of data, right?</p>
<p><span class="math display">\[
\begin{bmatrix}F1 \\ F4 \\ A1.1 \\ \vdots \end{bmatrix} \sim t + x + h + s
\]</span></p>
<p>Where the response variables are compliance with individual FAIR principles, <span class="math inline">\(t\)</span> is years since 2007, <span class="math inline">\(x\)</span> is taxa (mammal, bird, fish, other), <span class="math inline">\(h\)</span> is habitat (marine, terrestrial) and <span class="math inline">\(s\)</span> is sensor type (spatial, aspatial).</p>
<p>Now this brings up an issue with sensor type. What do you do with GPS+accelerometer tags, for instance? GPS may be shared one way and accelerometer another.</p>
<section id="h1-open-biologging-data-practices-are-increasing-over-time" class="level4">
<h4 class="anchored" data-anchor-id="h1-open-biologging-data-practices-are-increasing-over-time"><strong>H1:</strong> Open biologging data practices are increasing over time</h4>
<p>Coefficients for <span class="math inline">\(t\)</span> increase from early to middle to late.</p>
</section>
<section id="h2-overall-rates-of-open-biologging-data-practices-remain-low" class="level4">
<h4 class="anchored" data-anchor-id="h2-overall-rates-of-open-biologging-data-practices-remain-low"><strong>H2:</strong> Overall rates of open biologging data practices remain low</h4>
<p>How do we quantify “overall rates”? Examine each response individually? Restrict to late period?</p>
</section>
<section id="h3-biologging-studies-use-open-data-from-external-sources-more-than-they-generate-open-data" class="level4">
<h4 class="anchored" data-anchor-id="h3-biologging-studies-use-open-data-from-external-sources-more-than-they-generate-open-data"><strong>H3:</strong> Biologging studies use open data from external sources more than they generate open data</h4>
<p>How do we model this? This is probably wrong but it’s a starting point.</p>
<p><span class="math display">\[
o \sim t + x + h + s
\]</span></p>
<p>Where <span class="math inline">\(o\)</span> is a multinomial variable with four categories: no open data, open external, open biologging, both open.</p>
</section>
<section id="h4-open-biologging-data-practices-vary-by-taxa-habitat-and-spatial-vs-aspatial-e.g.-accelerometer-data" class="level4">
<h4 class="anchored" data-anchor-id="h4-open-biologging-data-practices-vary-by-taxa-habitat-and-spatial-vs-aspatial-e.g.-accelerometer-data"><strong>H4:</strong> Open biologging data practices vary by taxa, habitat, and spatial vs aspatial (e.g., accelerometer) data</h4>
<p><span class="math inline">\(x\)</span>, <span class="math inline">\(h\)</span>, and <span class="math inline">\(s\)</span> are all significant terms in GEE model. <em>A priori</em>, <span class="math inline">\(s_{spatial}\)</span> &gt; <span class="math inline">\(s_{aspatial}\)</span>. But for taxa <span class="math inline">\(x\)</span> and habitat <span class="math inline">\(h\)</span>, just predicting there are significant differences.</p>

</section>
</section>
</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body" data-line-spacing="2" role="list">
<div id="ref-rutz2009" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Rutz, C. &amp; Hays, G. C. <a href="https://doi.org/10.1098/rsbl.2009.0089">New frontiers in biologging science</a>. <em>Biology Letters</em> <strong>5</strong>, 289–292 (2009).</div>
</div>
<div id="ref-kooyman1966" class="csl-entry" role="listitem">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Kooyman, G. L. <a href="https://doi.org/10.1126/science.151.3717.1553">Maximum Diving Capacities of the Weddell Seal,<span><em>Leptonychotes weddelli</em></span></a>. <em>Science</em> <strong>151</strong>, 1553–1554 (1966).</div>
</div>
<div id="ref-wilmers2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Wilmers, C. C. <em>et al.</em> <a href="https://doi.org/10.1890/14-1401.1">The golden age of bio<span>-</span>logging: how animal<span>-</span>borne sensors are advancing the frontiers of ecology</a>. <em>Ecology</em> <strong>96</strong>, 1741–1753 (2015).</div>
</div>
<div id="ref-weinzierl2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Weinzierl, R. <em>et al.</em> <a href="https://doi.org/10.1002/ece3.2585">Wind estimation based on thermal soaring of birds</a>. <em>Ecology and Evolution</em> <strong>6</strong>, 8706–8718 (2016).</div>
</div>
<div id="ref-yonehara2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Yonehara, Y. <em>et al.</em> <a href="https://doi.org/10.1073/pnas.1523853113">Flight paths of seabirds soaring over the ocean surface enable measurement of fine-scale wind speed and direction</a>. <em>Proceedings of the National Academy of Sciences</em> <strong>113</strong>, 9039–9044 (2016).</div>
</div>
<div id="ref-treasure2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Treasure, A. <em>et al.</em> <a href="https://doi.org/10.5670/oceanog.2017.234">Marine mammals exploring the oceans pole to pole: A review of the MEOP consortium</a>. <em>Oceanography</em> <strong>30</strong>, 132–138 (2017).</div>
</div>
<div id="ref-kays2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Kays, R. <em>et al.</em> <a href="https://doi.org/10.1111/2041-210x.13767">The Movebank system for studying global animal movement and demography</a>. <em>Methods in Ecology and Evolution</em> <strong>13</strong>, 419–431 (2021).</div>
</div>
<div id="ref-denouden2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Ouden, O. F. C. den, Smets, P. S. M., Assink, J. D. &amp; Evers, L. G. <a href="https://doi.org/10.1029/2021gl094555">A Bird<span>’</span>s<span>-</span>Eye View on Ambient Infrasonic Soundscapes</a>. <em>Geophysical Research Letters</em> <strong>48</strong>, (2021).</div>
</div>
<div id="ref-nathan2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Nathan, R. <em>et al.</em> <a href="https://doi.org/10.1126/science.abg1780">Big-data approaches lead to an increased understanding of the ecology of animal movement</a>. <em>Science</em> <strong>375</strong>, (2022).</div>
</div>
<div id="ref-grémillet2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Grémillet, D., Chevallier, D. &amp; Guinet, C. <a href="https://doi.org/10.1093/icesjms/fsac059">Big data approaches to the spatial ecology and conservation of marine megafauna</a>. <em>ICES Journal of Marine Science</em> <strong>79</strong>, 975–986 (2022).</div>
</div>
<div id="ref-beal2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Beal, M. <em>et al.</em> <a href="https://doi.org/10.1126/sciadv.abd7225">Global political responsibility for the conservation of albatrosses and large petrels</a>. <em>Science Advances</em> <strong>7</strong>, (2021).</div>
</div>
<div id="ref-queiroz2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Queiroz, N. <em>et al.</em> <a href="https://doi.org/10.1038/s41586-019-1444-4">Global spatial risk assessment of sharks under the footprint of fisheries</a>. <em>Nature</em> <strong>572</strong>, 461–466 (2019).</div>
</div>
<div id="ref-hindell2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Hindell, M. A. <em>et al.</em> <a href="https://doi.org/10.1038/s41586-020-2126-y">Tracking of marine predators to protect Southern Ocean ecosystems</a>. <em>Nature</em> <strong>580</strong>, 87–92 (2020).</div>
</div>
<div id="ref-lowndes2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Lowndes, J. S. S. <em>et al.</em> <a href="https://doi.org/10.1038/s41559-017-0160">Our path to better science in less time using open data science tools</a>. <em>Nature Ecology &amp; Evolution</em> <strong>1</strong>, 1–7 (2017).</div>
</div>
<div id="ref-hampton2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Hampton, S. E. <em>et al.</em> <a href="https://doi.org/10.1093/biosci/bix025">Skills and Knowledge for Data-Intensive Environmental Research</a>. <em>BioScience</em> <strong>67</strong>, 546–557 (2017).</div>
</div>
<div id="ref-peterson2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Peterson, A. T., Asase, A., Canhos, D., Souza, S. de &amp; Wieczorek, J. <a href="https://doi.org/10.3897/bdj.6.e26826">Data leakage and loss in biodiversity informatics</a>. <em>Biodiversity Data Journal</em> <strong>6</strong>, (2018).</div>
</div>
<div id="ref-wilkinson2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">Wilkinson, M. D. <em>et al.</em> <a href="https://doi.org/10.1038/sdata.2016.18">The FAIR Guiding Principles for scientific data management and stewardship</a>. <em>Scientific Data</em> <strong>3</strong>, 160018 (2016).</div>
</div>
<div id="ref-halpern2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Halpern, B. S. <em>et al.</em> Ecological Synthesis and Its Role in Advancing Knowledge. <em>BioScience</em> (2020) doi:<a href="https://doi.org/10.1093/biosci/biaa105">10.1093/biosci/biaa105</a>.</div>
</div>
<div id="ref-fredston2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Fredston, A. L. &amp; Lowndes, J. S. S. <a href="https://doi.org/10.1146/annurev-marine-041723-094741">Welcoming More Participation in Open Data Science for the Oceans</a>. <em>Annual Review of Marine Science</em> <strong>16</strong>, (2023).</div>
</div>
<div id="ref-roche2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Roche, D. G. <em>et al.</em> <a href="https://doi.org/10.1111/cobi.13835">Closing the knowledge<span>-</span>action gap in conservation with open science</a>. <em>Conservation Biology</em> <strong>36</strong>, (2021).</div>
</div>
<div id="ref-buxton2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">Buxton, R. T. <em>et al.</em> <a href="https://doi.org/10.1111/csp2.329">Avoiding wasted research resources in conservation science</a>. <em>Conservation Science and Practice</em> <strong>3</strong>, (2021).</div>
</div>
<div id="ref-urbano2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Urbano, F. &amp; Cagnacci, F. <a href="https://doi.org/10.3389/fevo.2021.727023">Data management and sharing for collaborative science: Lessons learnt from the euromammals initiative</a>. <em>Frontiers in Ecology and Evolution</em> <strong>9</strong>, (2021).</div>
</div>
<div id="ref-sequeira2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Sequeira, A. M. M. <em>et al.</em> <a href="https://doi.org/10.1111/2041-210x.13593">A standardisation framework for bio<span>-</span>logging data to advance ecological research and conservation</a>. <em>Methods in Ecology and Evolution</em> <strong>12</strong>, 996–1007 (2021).</div>
</div>
<div id="ref-dettki2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Dettki, H., Arlt, D., Bäckman, J. &amp; Blanchet, M. <a href="https://doi.org/10.3897/biss.7.111438">Key-value pairs and NoSQL databases: A novel concept to manage biologging data in data repositories</a>. <em>Biodiversity Information Science and Standards</em> <strong>7</strong>, (2023).</div>
</div>
<div id="ref-kranstauber2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">25. </div><div class="csl-right-inline">Kranstauber, B. <em>et al.</em> <a href="https://doi.org/10.1016/j.envsoft.2010.12.005">The Movebank data model for animal tracking</a>. <em>Environmental Modelling &amp; Software</em> <strong>26</strong>, 834–835 (2011).</div>
</div>
<div id="ref-kranstauber2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Kranstauber, B., Safi, K. &amp; Scharf, A. K. <em><a href="https://CRAN.R-project.org/package=move2">move2: Processing and analysing animal trajectories</a></em>. (2023).</div>
</div>
<div id="ref-wieczorek2012" class="csl-entry" role="listitem">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Wieczorek, J. <em>et al.</em> <a href="https://doi.org/10.1371/journal.pone.0029715">Darwin Core: an evolving community-developed biodiversity data standard</a>. <em>PLoS ONE</em> <strong>7</strong>, e29715 (2012).</div>
</div>
<div id="ref-MPIAB2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Animal Behavior, M. P. I. of. <a href="http://dx.doi.org/10.1038/s41597-019-0042-5">Movebank Attribute Dictionary</a>.</div>
</div>
<div id="ref-campbell2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">Campbell, H. A. <em>et al.</em> <a href="https://doi.org/10.1016/j.scitotenv.2015.01.089">Finding our way: On the sharing and reuse of animal telemetry data in Australasia</a>. <em>Science of The Total Environment</em> <strong>534</strong>, 79–84 (2015).</div>
</div>
<div id="ref-scarpignato2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">30. </div><div class="csl-right-inline">Scarpignato, A. L. <em>et al.</em> <a href="https://doi.org/10.1016/j.biocon.2023.110224">Shortfalls in tracking data available to inform North American migratory bird conservation</a>. <em>Biological Conservation</em> <strong>286</strong>, 110224 (2023).</div>
</div>
<div id="ref-taylor2007" class="csl-entry" role="listitem">
<div class="csl-left-margin">31. </div><div class="csl-right-inline">Taylor, C. F. <em>et al.</em> <a href="https://doi.org/10.1038/nbt1329">The minimum information about a proteomics experiment (MIAPE)</a>. <em>Nature Biotechnology</em> <strong>25</strong>, 887–893 (2007).</div>
</div>
<div id="ref-rund2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">32. </div><div class="csl-right-inline">Rund, S. S. C. <em>et al.</em> <a href="https://doi.org/10.1038/s41597-019-0042-5">MIReAD, a minimum information standard for reporting arthropod abundance data</a>. <em>Scientific Data</em> <strong>6</strong>, (2019).</div>
</div>
<div id="ref-liang1986" class="csl-entry" role="listitem">
<div class="csl-left-margin">33. </div><div class="csl-right-inline">Liang, K.-Y. &amp; Zeger, S. L. <a href="https://doi.org/10.1093/biomet/73.1.13">Longitudinal data analysis using generalized linear models</a>. <em>Biometrika</em> <strong>73</strong>, 13–22 (1986).</div>
</div>
<div id="ref-halekoh2006" class="csl-entry" role="listitem">
<div class="csl-left-margin">34. </div><div class="csl-right-inline">Halekoh, U., Højsgaard, S. &amp; Yan, J. <a href="https://doi.org/10.18637/jss.v015.i02">The <span><em>R</em></span> Package <span><strong>geepack</strong></span> for Generalized Estimating Equations</a>. <em>Journal of Statistical Software</em> <strong>15</strong>, (2006).</div>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>