<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Trends, challenges, and opportunities in open biologging data</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="registeredreport_files/libs/clipboard/clipboard.min.js"></script>
<script src="registeredreport_files/libs/quarto-html/quarto.js"></script>
<script src="registeredreport_files/libs/quarto-html/popper.min.js"></script>
<script src="registeredreport_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="registeredreport_files/libs/quarto-html/anchor.min.js"></script>
<link href="registeredreport_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="registeredreport_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="registeredreport_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="registeredreport_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="registeredreport_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Trends, challenges, and opportunities in open biologging data</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="introduction-470500-words" class="level2">
<h2 class="anchored" data-anchor-id="introduction-470500-words">Introduction (470/500 words)</h2>
<p>Biologging is the practice of using animal-borne sensors to remotely observe animals and their environments<span class="citation" data-cites="rutz2009"><sup><a href="#ref-rutz2009" role="doc-biblioref">1</a></sup></span>. In the six decades since a biologist first attached a modified kitchen timer to a seal<span class="citation" data-cites="kooyman1966"><sup><a href="#ref-kooyman1966" role="doc-biblioref">2</a></sup></span>, biologging has become an invaluable, multidisciplinary tool used for ecology<span class="citation" data-cites="wilmers2015"><sup><a href="#ref-wilmers2015" role="doc-biblioref">3</a></sup></span>, atmospheric science<span class="citation" data-cites="weinzierl2016 yonehara2016"><sup><a href="#ref-weinzierl2016" role="doc-biblioref">4</a>,<a href="#ref-yonehara2016" role="doc-biblioref">5</a></sup></span>, and oceanography<span class="citation" data-cites="treasure2017"><sup><a href="#ref-treasure2017" role="doc-biblioref">6</a></sup></span>. As biologging data continue to grow in size and complexity (e.g., millions of new records daily in Movebank<span class="citation" data-cites="kays2021"><sup><a href="#ref-kays2021" role="doc-biblioref">7</a></sup></span> and novel sensors like infrasound recorders<span class="citation" data-cites="denouden2021"><sup><a href="#ref-denouden2021" role="doc-biblioref">8</a></sup></span>), “big data” methods have been highlighted as a key direction for future developments<span class="citation" data-cites="nathan2022 grémillet2022"><sup><a href="#ref-nathan2022" role="doc-biblioref">9</a>,<a href="#ref-grémillet2022" role="doc-biblioref">10</a></sup></span>. Simultaneously, biologging data are increasingly used to inform conservation efforts across taxonomic groups<span class="citation" data-cites="beal2021 queiroz2019"><sup><a href="#ref-beal2021" role="doc-biblioref">11</a>,<a href="#ref-queiroz2019" role="doc-biblioref">12</a></sup></span> and regions<span class="citation" data-cites="hindell2020"><sup><a href="#ref-hindell2020" role="doc-biblioref">13</a></sup></span>. Yet, for aggregated biologging data to achieve envisioned impacts on research and conservation, increased access is a matter of practical and ethical concern.</p>
<p>Open science practices, such as publishing data and code alongside manuscripts, improve research transparency and efficiency<span class="citation" data-cites="lowndes2017 hampton2017"><sup><a href="#ref-lowndes2017" role="doc-biblioref">14</a>,<a href="#ref-hampton2017" role="doc-biblioref">15</a></sup></span>. However, data “openness” is determined by multiple factors, and poor research data management and sharing practices are a major source of biodiversity data loss<span class="citation" data-cites="peterson2018"><sup><a href="#ref-peterson2018" role="doc-biblioref">16</a></sup></span>. Fortunately, the advent of the FAIR principles<span class="citation" data-cites="wilkinson2016"><sup><a href="#ref-wilkinson2016" role="doc-biblioref">17</a></sup></span> (<a href="#tbl-fair">Table&nbsp;1</a>) has provided a useful framework to facilitate open data sharing, with FAIR data enabling groundbreaking advances in synthesis, biodiversity, and conservation science<span class="citation" data-cites="halpern2020 fredston2023 roche2021 buxton2021"><sup><a href="#ref-halpern2020" role="doc-biblioref">18</a>–<a href="#ref-buxton2021" role="doc-biblioref">21</a></sup></span> .</p>
<p>There have been numerous efforts within the biologging research community to promote open data sharing practices. Essential cyberinfrastructure include domain-specific data repositories (Movebank<span class="citation" data-cites="kays2021"><sup><a href="#ref-kays2021" role="doc-biblioref">7</a></sup></span>, Seabird Tracking Database <a href="https://www.seabirdtracking.org">seabirdtracking.org</a>, Euromammal<span class="citation" data-cites="urbano2021"><sup><a href="#ref-urbano2021" role="doc-biblioref">22</a></sup></span>) and data standards<span class="citation" data-cites="sequeira2021 dettki2023 kranstauber2011"><sup><a href="#ref-sequeira2021" role="doc-biblioref">23</a>–<a href="#ref-kranstauber2011" role="doc-biblioref">25</a></sup></span> that facilitate adoption of the FAIR principles (<a href="#tbl-fair">Table&nbsp;1</a>).</p>
<div id="tbl-fair" class="anchored">
<table class="table">
<caption>Table&nbsp;1: The FAIR principles, as implemented in biologging cyberinfrastructure, support data sharing and improve research transparency and efficiency.</caption>
<colgroup>
<col style="width: 23%">
<col style="width: 23%">
<col style="width: 30%">
<col style="width: 23%">
</colgroup>
<thead>
<tr class="header">
<th>FAIR principle</th>
<th>Definition</th>
<th>Application to biologging</th>
<th>Example</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Findability</td>
<td>Data and metadata have a globally unique and persistent identifier (e.g., a digital object identifier, DOI) and are indexed in a searchable resource</td>
<td>Data repositories, like the Movebank Data Repository, improve data discoverability and may assign DOIs to data (avoiding issues with broken hyperlinks, for example)</td>
<td>A tracking dataset deposited in the Movebank Data Repository is findable by its permanent DOI or by searching the repository.</td>
</tr>
<tr class="even">
<td>Accessibility</td>
<td>Data and metadata are retrievable by open and universal protocols, such as HTTP</td>
<td>Data repositories allow scientists to retrieve biologging data via a web browser or other open source tools</td>
<td>Publicly available data on Movebank may be downloaded via the website, API, or with the <code>move2</code> R package<span class="citation" data-cites="kranstauber2023"><sup><a href="#ref-kranstauber2023" role="doc-biblioref">26</a></sup></span></td>
</tr>
<tr class="odd">
<td>Interoperability</td>
<td>Data use formal and shared formats and vocabularies</td>
<td>Shared protocols like the Darwin Core standard<span class="citation" data-cites="wieczorek2012"><sup><a href="#ref-wieczorek2012" role="doc-biblioref">27</a></sup></span>, Movebank data model<span class="citation" data-cites="MPIAB2023"><sup><a href="#ref-MPIAB2023" role="doc-biblioref">28</a></sup></span>, and proposed bio-logging standardization framework<span class="citation" data-cites="sequeira2021"><sup><a href="#ref-sequeira2021" role="doc-biblioref">23</a></sup></span> reduce barriers to combining datasets and increase uptake within and across scientific disciplines</td>
<td>Location and environmental data from seal-borne biologgers harmonized to a standard netCDF format facilitated their reuse by oceanographers to study polar regions<span class="citation" data-cites="treasure2017"><sup><a href="#ref-treasure2017" role="doc-biblioref">6</a></sup></span></td>
</tr>
<tr class="even">
<td>Reusability</td>
<td>Data and metadata are richly described and reuse permissions are clearly defined</td>
<td>Data repositories and standards together capture essential context for biologging data and provide licensing options for data reuse</td>
<td>The Movebank Data Repository releases datasets under the CC0 license and the Movebank data model<span class="citation" data-cites="MPIAB2023"><sup><a href="#ref-MPIAB2023" role="doc-biblioref">28</a></sup></span> includes fields important for data reuse, e.g., whether the animal was relocated before release.</td>
</tr>
</tbody>
</table>
</div>
<p>Despite these biologging cyberinfrastructure developments, the general state of open data within the community remains unknown, and there are indications that a majority of tracking data remain inaccessible<span class="citation" data-cites="campbell2015 scarpignato2023"><sup><a href="#ref-campbell2015" role="doc-biblioref">29</a>,<a href="#ref-scarpignato2023" role="doc-biblioref">30</a></sup></span>. We will conduct a systematic review of the biologging literature to quantify open data practices, including open data (re)use and sharing by authors of biologging studies published between 2007-2023 (N=4799). The review will quantify the context of biologging studies (e.g., instrumented species and sensor types) and test the following four hypotheses:</p>
<p><strong>H1:</strong> Open biologging data practices are increasing over time.</p>
<p><strong>H2:</strong> Open biologging data practices vary by ecosystem (marine or terrestrial).</p>
<p><strong>H3:</strong> Spatial biologging data are shared more often than aspatial (e.g., accelerometer) data.</p>
<p>In addition to hypothesis testing, our review will provide the first biologging bibliometric database, itself a valuable form of cyberinfrastrucure for future systematic reviews and meta-analyses. Based on the results of our review, we will also provide a set of minimum reporting standards (<em>sensu</em> MIAPE<span class="citation" data-cites="taylor2007"><sup><a href="#ref-taylor2007" role="doc-biblioref">31</a></sup></span> for proteomics experiments and MIReAD<span class="citation" data-cites="rund2019"><sup><a href="#ref-rund2019" role="doc-biblioref">32</a></sup></span> for arthropod abundances) as a resource for authors, journal editors and funding bodies, to facilitate FAIR sharing of biologging data.</p>
</section>
<section id="methods" class="level2">
<h2 class="anchored" data-anchor-id="methods">Methods</h2>
<p>The scientific literature has yet to arrive on consensus definitions for biologging and related terms, such as biotelemetry, tagging, and tracking. Rutz and Hays<span class="citation" data-cites="rutz2009"><sup><a href="#ref-rutz2009" role="doc-biblioref">1</a></sup></span> defined biologging as “the use of miniaturized animal-attached tags for logging and/or relaying of data about an animal’s movements, behaviour, physiology and/or environment” and Cooke et al.<span class="citation" data-cites="cooke2004"><sup><a href="#ref-cooke2004" role="doc-biblioref">33</a></sup></span> defined biotelemetry as “remote measurement of physiological, behavioural, or energetic data”. These and other definitions share the following two characteristics:</p>
<ul>
<li><p>An electronic device is attached to an animal, externally or internally.</p></li>
<li><p>The device uses sensors and memory to record observations (e.g.&nbsp;GPS and accelerometers) <em>or</em> produces a signal that may be recorded by other sensors (e.g.&nbsp;acoustic telemetry and Argos satellites).</p>
<ul>
<li>Hybrid devices may do both, either separately (e.g.&nbsp;a tag that transmits to Argos but also records depth from a pressure sensor, which may be downloaded after recovering the device) or simultaneously (e.g.&nbsp;a tag that records location using a GPS sensor, then transmits that location to Argos).</li>
</ul></li>
</ul>
<p>Further complicating matters, biologging may be used in many research contexts. Much of the biologger literature involves deployments on unrestrained, undomesticated animals for the purposes of addressing questions in biology, conservation, and related sciences. But agricultural research also uses biologging devices to assess animal welfare and resource use (domesticated animals) and biomedical research uses animal-borne heart rate sensors in lab experiments (restrained).</p>
<p>The focus of this review is:</p>
<ol type="1">
<li>Biologging devices that use sensors (as opposed to acoustic or radio transmitters)</li>
<li>Deployed on unrestrained, undomesticated animals</li>
</ol>
<p>The choice to exclude transmitters was based on the properties of the data collected. Data from sensor-based biologgers can reasonably be considered self-contained, whereas data from transmitters depend equally on the receiver network. This external dependency complicates the assessment of open data sharing, the primary goal of this review. We make an exception for one type of transmitter, Argos tags, because the Argos system provides data to researchers with properties more similar to sensor-based biologgers.</p>
<section id="literature-review" class="level3">
<h3 class="anchored" data-anchor-id="literature-review">Literature review</h3>
<p>On 2023-08-21, we queried Web of Science Core Collection for papers published since 2007 related to biologging with the following query:</p>
<p><code>animal AND (biologg* OR bio-logg* OR biotelemetry OR electronic tag* OR satellite track* OR GPS telemetry OR satellite telemetry OR satellite transmit* OR GPS collar* OR depth recorder* OR accelerometer OR archival tag*)</code></p>
<p>This yielded 6654 papers. We limited results to Web of Science categories with 100+ papers we deemed relevant, leaving 4799 papers. The categories we included were: Ecology, Zoology, Marine Freshwater Biology, Biodiversity Conservation, Multidisciplinary Sciences, Environmental Sciences, Oceanography, Biology, Behavioral Sciences, Ornithology, Evolutionary Biology, and Fisheries. We excluded the following categories with 100+ papers: Agriculture Dairy Animal Science, Veterinary Sciences, Engineering Electrical Electronic, Physiology, Food Science Technology, Engineering Biomedical, Agriculture Multidisciplinary, Computer Science Interdisciplinary Applications, Instruments Instrumentation.</p>
</section>
<section id="initial-classification" class="level3">
<h3 class="anchored" data-anchor-id="initial-classification">Initial classification</h3>
<p>Every paper will be reviewed by two individuals to assess its relevance to this review, record the presence or absence of a data availability statement, and to categorize the study by sensor type, taxonomy, and ecosystem. Papers will be considered relevant if they satisfy the following two characteristics:</p>
<ol type="1">
<li>They involved the collection of previously unpublished biologging data. Our definition of biologging excludes telemetry devices (e.g., acoustic pingers and radiotransmitters) with the exception of devices that communicate with Argos satellites.</li>
<li>The biologgers were deployed on non-domesticated animals in unrestrained habitat. Studies involving domesticated animals (e.g., cattle) or animals in managed care (e.g., a zoo or aquarium) will be excluded.</li>
</ol>
<p>We will consider a paper to have a data availability statement (DAS) if it states how to access the data or explicitly states why the data are not available. Any of the following examples would be considered a DAS: “Data were deposited in a repository” with an accompanying DOI, “Data are available at a website” with an accompanying URL, “Data are available upon request”, or “Data withheld for the safety of the tracked animals”.</p>
<p>We will assign sensor types to one or more categories following Williams et al.<span class="citation" data-cites="williams2019"><sup><a href="#ref-williams2019" role="doc-biblioref">34</a></sup></span>: location (e.g., GPS, depth), intrinsic (e.g., accelerometer, internal temperature), or environment (e.g., ambient temperature, camera). Biologgers can combine sensors from multiple categories (e.g., a GPS-accelerometer collar).</p>
<p>We will resolve taxonomy to the species level using the Integrated Taxonomic Information System (www.itis.gov). Each study’s ecosystem will be classified as one or more of aquatic, marine, and terrestrial according to the IUCN Red List (https://www.iucnredlist.org).</p>
</section>
<section id="assessing-open-data" class="level3">
<h3 class="anchored" data-anchor-id="assessing-open-data">Assessing open data</h3>
<p>Following our initial classification, we will randomly sample 600 papers to assess the openness of biologging data. We will limit our sample to papers with a DAS, and we will stratify our sample by year, ecosystem, and sensor type. We will attempt to sample equal numbers of papers across the following categories:</p>
<ul>
<li><p>Time (early 2007-2015, middle 2016-2019, recent 2020-2023)</p></li>
<li><p>Ecosystem (marine, terrestrial)</p></li>
<li><p>Sensor type (spatial, aspatial)</p></li>
</ul>
<p>The sensor type of a study will be considered spatial if it includes a location sensor and aspatial if it includes an intrinsic or environmental sensor.</p>
<p>For each sampled paper, we will attempt to retrieve the biologging dataset underlying the study. If data are available upon request, we will email the corresponding author to request the data and send one follow-up email two weeks later if we do not receive a response. We will record whether each dataset satisfies the following properties:</p>
<ul>
<li><p>Do the data have a permanent identifier, such as a DOI? (FAIR principle F1)</p></li>
<li><p>Are the data retrievable via a standard communications protocol, such as HTTP? (FAIR principle A1)</p></li>
<li><p>Do the data contain re-usable biologging data, as opposed to summaries?</p></li>
<li><p>Are the data sufficiently complete to reproduce the analyses in the study? I.e., are all necessary data and metadata included within the dataset.</p></li>
</ul>
<p>If we cannot retrieve the dataset for a study, we will consider the study to not satisfy any of the four properties listed above.</p>
</section>
<section id="analysis" class="level3">
<h3 class="anchored" data-anchor-id="analysis">Analysis</h3>
<p>Our independent variables will be: year of publication (<span class="math inline">\(t\)</span>), ecosystem (<span class="math inline">\(e\)</span>), and sensor type (<span class="math inline">\(s\)</span>) [see “Initial classification”]. Our dependent variables will be: the presence of a permanent identifier (<span class="math inline">\(f\)</span>), whether data are retrievable by a standard protocol (<span class="math inline">\(a\)</span>), whether data include re-usable biologging data (<span class="math inline">\(r\)</span>), and whether data are complete (<span class="math inline">\(c\)</span>) [see “Assessing open data”]. We will fit four Bayesian generalized linear models of the form:</p>
<p><span class="math display">\[
\begin{align}
DV_{i} &amp;\sim Bernoulli(p) \\
logit(p_i) &amp;= \alpha - \beta_e e_i + \beta_s s_i + \phi_i \\
\phi_i &amp;= \beta_t mo(t, \delta) \\
\alpha &amp;\sim Normal(0, 1) \\
\beta_e, \beta_s, \beta_t &amp;\sim Normal(0, 1) \\
\delta &amp;\sim Dirichlet(1, ..., 1)
\end{align}
\]</span></p>
<p>Where <span class="math inline">\(DV\)</span> is dependent variable (<span class="math inline">\(f\)</span>, <span class="math inline">\(a\)</span>, <span class="math inline">\(r\)</span>, or <span class="math inline">\(c\)</span>). <span class="math inline">\(e\)</span> and <span class="math inline">\(s\)</span> encode ecosystem (0 = marine, 1 = terrestrial) and sensor type (0 = aspatial, 1 = spatial), respectively. Year of publication (<span class="math inline">\(t\)</span>) is treated as an ordinal categorical predictor<span class="citation" data-cites="bürkner2020"><sup><a href="#ref-bürkner2020" role="doc-biblioref">35</a></sup></span>. Modeling <span class="math inline">\(t\)</span> this way, instead of as a metric continuous variable, allows the rate of change of the dependent variables to be nonlinear in time. <span class="math inline">\(\phi\)</span> represents the effect of year of publication, which is the product of a coefficient, <span class="math inline">\(\beta_t\)</span>, and a monotonically increasing component, <span class="math inline">\(\delta\)</span>, between 0 and 1. Thus <span class="math inline">\(\phi\)</span> takes on the value 0 at the beginning of the time period (2007) and increases to <span class="math inline">\(\beta_t\)</span> at the end (2023). <span class="math inline">\(\delta\)</span> controls the incremental steps between the two periods. The priors for <span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\beta_e, \beta_s, \beta_t\)</span> are standard normal, and the prior for <span class="math inline">\(\delta\)</span> is a Dirichlet distribution with 1 for each step between years.</p>
<p>We will use Stan<span class="citation" data-cites="Stan"><sup><a href="#ref-Stan" role="doc-biblioref">36</a></sup></span> and brms<span class="citation" data-cites="bürkner2018 bürkner2021"><sup><a href="#ref-bürkner2018" role="doc-biblioref">37</a>,<a href="#ref-bürkner2021" role="doc-biblioref">38</a></sup></span> for model fitting.</p>
<section id="h1-open-biologging-data-practices-are-increasing-over-time" class="level4">
<h4 class="anchored" data-anchor-id="h1-open-biologging-data-practices-are-increasing-over-time">H1: Open biologging data practices are increasing over time</h4>
<p>We will test hypothesis <strong>H1</strong> by comparing posterior predictions for 2023 and 2016. We will use 2016 as the baseline instead of the beginning of our study period (2007) to undertake a more stringent test of our hypothesis. Widespread adoption of open data policies by ecology and evolutionary biology journals began circa 2012-2014<span class="citation" data-cites="roche2015"><sup><a href="#ref-roche2015" role="doc-biblioref">39</a></sup></span> and the FAIR data principles were published in 2016<span class="citation" data-cites="wilkinson2016"><sup><a href="#ref-wilkinson2016" role="doc-biblioref">17</a></sup></span>. Therefore we consider 2016 a more meaningful point of comparison.</p>
</section>
<section id="h2-open-biologging-data-practices-vary-by-ecosystem" class="level4">
<h4 class="anchored" data-anchor-id="h2-open-biologging-data-practices-vary-by-ecosystem">H2: Open biologging data practices vary by ecosystem</h4>
<p>We will test hypothesis <strong>H2</strong> by comparing posterior predictions for marine and terrestrial ecosystems for the year 2023.</p>
</section>
<section id="h3-open-biologging-data-practices-vary-by-ecosystem" class="level4">
<h4 class="anchored" data-anchor-id="h3-open-biologging-data-practices-vary-by-ecosystem">H3: Open biologging data practices vary by ecosystem</h4>
<p>We will test hypothesis <strong>H3</strong> by comparing posterior predictions for aspatial and spatial ecosystems for the year 2023.</p>
</section>
</section>
<section id="simulation" class="level3">
<h3 class="anchored" data-anchor-id="simulation">Simulation</h3>
<p>We simulated reasonable values for <span class="math inline">\(t\)</span>, <span class="math inline">\(e\)</span>, <span class="math inline">\(s\)</span>, and <span class="math inline">\(f\)</span> to demonstrate how we will perform our analysis.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(brms)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Loading required package: Rcpp</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Loading 'brms' package (version 2.20.4). Useful instructions
can be found by typing help('brms'). A more detailed introduction
to the package is available through vignette('brms_overview').</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>
Attaching package: 'brms'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following object is masked from 'package:stats':

    ar</code></pre>
</div>
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ dplyr     1.1.4     ✔ readr     2.1.5
✔ forcats   1.0.0     ✔ stringr   1.5.1
✔ ggplot2   3.4.4     ✔ tibble    3.2.1
✔ lubridate 1.9.3     ✔ tidyr     1.3.1
✔ purrr     1.0.2     </code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()
ℹ Use the conflicted package (&lt;http://conflicted.r-lib.org/&gt;) to force all conflicts to become errors</code></pre>
</div>
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">1234</span>)</span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a><span class="co"># Grid of independent variables</span></span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a>biolog_grid <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">era =</span> <span class="fu">c</span>(<span class="st">"early"</span>, <span class="st">"middle"</span>, <span class="st">"recent"</span>),</span>
<span id="cb9-6"><a href="#cb9-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="fu">c</span>(<span class="st">"marine"</span>, <span class="st">"terrestrial"</span>),</span>
<span id="cb9-7"><a href="#cb9-7" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="fu">c</span>(<span class="st">"aspatial"</span>, <span class="st">"spatial"</span>)</span>
<span id="cb9-8"><a href="#cb9-8" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb9-9"><a href="#cb9-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">slice</span>(<span class="fu">rep</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">12</span>, <span class="at">each =</span> <span class="dv">50</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb9-10"><a href="#cb9-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">T =</span> <span class="fu">case_when</span>(</span>
<span id="cb9-11"><a href="#cb9-11" aria-hidden="true" tabindex="-1"></a>    era <span class="sc">==</span> <span class="st">"early"</span>  <span class="sc">~</span> <span class="fu">sample</span>(<span class="dv">2007</span><span class="sc">:</span><span class="dv">2015</span>, <span class="dv">600</span>, <span class="at">replace =</span> <span class="cn">TRUE</span>),</span>
<span id="cb9-12"><a href="#cb9-12" aria-hidden="true" tabindex="-1"></a>    era <span class="sc">==</span> <span class="st">"middle"</span> <span class="sc">~</span> <span class="fu">sample</span>(<span class="dv">2016</span><span class="sc">:</span><span class="dv">2019</span>, <span class="dv">600</span>, <span class="at">replace =</span> <span class="cn">TRUE</span>),</span>
<span id="cb9-13"><a href="#cb9-13" aria-hidden="true" tabindex="-1"></a>    era <span class="sc">==</span> <span class="st">"recent"</span> <span class="sc">~</span> <span class="fu">sample</span>(<span class="dv">2020</span><span class="sc">:</span><span class="dv">2023</span>, <span class="dv">600</span>, <span class="at">replace =</span> <span class="cn">TRUE</span>)</span>
<span id="cb9-14"><a href="#cb9-14" aria-hidden="true" tabindex="-1"></a>  ))</span>
<span id="cb9-15"><a href="#cb9-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-16"><a href="#cb9-16" aria-hidden="true" tabindex="-1"></a><span class="co"># Parameters for simulation</span></span>
<span id="cb9-17"><a href="#cb9-17" aria-hidden="true" tabindex="-1"></a><span class="co"># delta</span></span>
<span id="cb9-18"><a href="#cb9-18" aria-hidden="true" tabindex="-1"></a>delta <span class="ot">&lt;-</span> <span class="fu">c</span>(</span>
<span id="cb9-19"><a href="#cb9-19" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Dummy for 2007</span></span>
<span id="cb9-20"><a href="#cb9-20" aria-hidden="true" tabindex="-1"></a>  <span class="dv">0</span>,</span>
<span id="cb9-21"><a href="#cb9-21" aria-hidden="true" tabindex="-1"></a>  <span class="co"># slow increases in early period</span></span>
<span id="cb9-22"><a href="#cb9-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rep</span>(<span class="dv">1</span>, <span class="dv">8</span>),</span>
<span id="cb9-23"><a href="#cb9-23" aria-hidden="true" tabindex="-1"></a>  <span class="co"># fastest increases in middle period</span></span>
<span id="cb9-24"><a href="#cb9-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rep</span>(<span class="dv">3</span>, <span class="dv">4</span>),</span>
<span id="cb9-25"><a href="#cb9-25" aria-hidden="true" tabindex="-1"></a>  <span class="co"># moderate increases in recent period</span></span>
<span id="cb9-26"><a href="#cb9-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rep</span>(<span class="dv">2</span>, <span class="dv">4</span>)</span>
<span id="cb9-27"><a href="#cb9-27" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb9-28"><a href="#cb9-28" aria-hidden="true" tabindex="-1"></a>delta <span class="ot">&lt;-</span> delta <span class="sc">/</span> <span class="fu">sum</span>(delta)</span>
<span id="cb9-29"><a href="#cb9-29" aria-hidden="true" tabindex="-1"></a><span class="co"># betas (E, S, T)</span></span>
<span id="cb9-30"><a href="#cb9-30" aria-hidden="true" tabindex="-1"></a>beta_E <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="at">marine =</span> <span class="dv">0</span>, <span class="at">terrestrial =</span> <span class="dv">1</span>)</span>
<span id="cb9-31"><a href="#cb9-31" aria-hidden="true" tabindex="-1"></a>beta_S <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="at">aspatial =</span> <span class="dv">0</span>, <span class="at">spatial =</span> <span class="dv">1</span>)</span>
<span id="cb9-32"><a href="#cb9-32" aria-hidden="true" tabindex="-1"></a>beta_T <span class="ot">&lt;-</span> <span class="dv">3</span></span>
<span id="cb9-33"><a href="#cb9-33" aria-hidden="true" tabindex="-1"></a>alpha <span class="ot">&lt;-</span> <span class="sc">-</span><span class="dv">4</span></span>
<span id="cb9-34"><a href="#cb9-34" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-35"><a href="#cb9-35" aria-hidden="true" tabindex="-1"></a><span class="co"># Run simulation</span></span>
<span id="cb9-36"><a href="#cb9-36" aria-hidden="true" tabindex="-1"></a>inv_logit <span class="ot">&lt;-</span> \(x) <span class="fu">exp</span>(x) <span class="sc">/</span> (<span class="dv">1</span> <span class="sc">+</span> <span class="fu">exp</span>(x))</span>
<span id="cb9-37"><a href="#cb9-37" aria-hidden="true" tabindex="-1"></a>biolog <span class="ot">&lt;-</span> biolog_grid <span class="sc">%&gt;%</span></span>
<span id="cb9-38"><a href="#cb9-38" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">cumsum_delta =</span> <span class="fu">map_dbl</span>(T, \(t) <span class="fu">sum</span>(delta[<span class="dv">1</span><span class="sc">:</span>(t <span class="sc">-</span> <span class="dv">2007</span>)])),</span>
<span id="cb9-39"><a href="#cb9-39" aria-hidden="true" tabindex="-1"></a>         <span class="at">phi =</span> alpha <span class="sc">+</span> beta_E[E] <span class="sc">+</span> beta_S[S] <span class="sc">+</span> beta_T <span class="sc">*</span> cumsum_delta,</span>
<span id="cb9-40"><a href="#cb9-40" aria-hidden="true" tabindex="-1"></a>         <span class="at">F =</span> <span class="fu">rbinom</span>(<span class="dv">600</span>, <span class="at">size =</span> <span class="dv">1</span>, <span class="at">prob =</span> <span class="fu">inv_logit</span>(phi)))</span>
<span id="cb9-41"><a href="#cb9-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-42"><a href="#cb9-42" aria-hidden="true" tabindex="-1"></a><span class="co"># Visualize data</span></span>
<span id="cb9-43"><a href="#cb9-43" aria-hidden="true" tabindex="-1"></a>biolog_summ <span class="ot">&lt;-</span> biolog <span class="sc">%&gt;%</span></span>
<span id="cb9-44"><a href="#cb9-44" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(T, E, S) <span class="sc">%&gt;%</span></span>
<span id="cb9-45"><a href="#cb9-45" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarize</span>(<span class="at">F =</span> <span class="fu">mean</span>(F),</span>
<span id="cb9-46"><a href="#cb9-46" aria-hidden="true" tabindex="-1"></a>            <span class="at">.groups =</span> <span class="st">"drop"</span>)</span>
<span id="cb9-47"><a href="#cb9-47" aria-hidden="true" tabindex="-1"></a><span class="co"># phi ~ T</span></span>
<span id="cb9-48"><a href="#cb9-48" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(biolog, <span class="fu">aes</span>(T, phi, <span class="at">color =</span> E)) <span class="sc">+</span></span>
<span id="cb9-49"><a href="#cb9-49" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>() <span class="sc">+</span></span>
<span id="cb9-50"><a href="#cb9-50" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span>S) <span class="sc">+</span></span>
<span id="cb9-51"><a href="#cb9-51" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb9-52"><a href="#cb9-52" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"bottom"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="registeredreport_files/figure-html/simulation-1.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="co"># F ~ T</span></span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(biolog, <span class="fu">aes</span>(T, F, <span class="at">color =</span> E)) <span class="sc">+</span></span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_jitter</span>(<span class="fu">aes</span>(<span class="at">shape =</span> S), <span class="at">alpha =</span> <span class="fl">0.75</span>, <span class="at">width =</span> <span class="fl">0.25</span>, <span class="at">height =</span> <span class="fl">0.05</span>) <span class="sc">+</span></span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="fu">aes</span>(<span class="at">linetype =</span> S), biolog_summ) <span class="sc">+</span></span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">breaks =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fl">0.5</span>, <span class="dv">1</span>)) <span class="sc">+</span></span>
<span id="cb10-6"><a href="#cb10-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb10-7"><a href="#cb10-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"bottom"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="registeredreport_files/figure-html/simulation-2.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Fit model</span></span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a>biolog_prior <span class="ot">&lt;-</span> <span class="fu">c</span>(</span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_prior</span>(<span class="at">prior =</span> <span class="st">"normal(0, 1)"</span>, <span class="at">class =</span> <span class="st">"Intercept"</span>),</span>
<span id="cb11-4"><a href="#cb11-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_prior</span>(<span class="at">prior =</span> <span class="st">"normal(0, 1)"</span>, <span class="at">coef =</span> <span class="st">"Eterrestrial"</span>),</span>
<span id="cb11-5"><a href="#cb11-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_prior</span>(<span class="at">prior =</span> <span class="st">"normal(0, 1)"</span>, <span class="at">coef =</span> <span class="st">"Sspatial"</span>),</span>
<span id="cb11-6"><a href="#cb11-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_prior</span>(<span class="at">prior =</span> <span class="st">"normal(0, 1)"</span>, <span class="at">coef =</span> <span class="st">"moT"</span>),</span>
<span id="cb11-7"><a href="#cb11-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">set_prior</span>(<span class="at">prior =</span> <span class="st">"dirichlet(1)"</span>, <span class="at">class =</span> <span class="st">"simo"</span>, <span class="at">coef =</span> <span class="st">"moT1"</span>)</span>
<span id="cb11-8"><a href="#cb11-8" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb11-9"><a href="#cb11-9" aria-hidden="true" tabindex="-1"></a>biolog_mod <span class="ot">&lt;-</span> <span class="fu">brm</span>(F <span class="sc">~</span> E <span class="sc">+</span> S <span class="sc">+</span> <span class="fu">mo</span>(T),</span>
<span id="cb11-10"><a href="#cb11-10" aria-hidden="true" tabindex="-1"></a>                  <span class="at">data =</span> biolog,</span>
<span id="cb11-11"><a href="#cb11-11" aria-hidden="true" tabindex="-1"></a>                  <span class="at">family =</span> <span class="fu">bernoulli</span>(<span class="at">link =</span> <span class="st">"logit"</span>),</span>
<span id="cb11-12"><a href="#cb11-12" aria-hidden="true" tabindex="-1"></a>                  <span class="at">prior =</span> biolog_prior)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Compiling Stan program...
Trying to compile a simple C file</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>Running /Library/Frameworks/R.framework/Resources/bin/R CMD SHLIB foo.c
using C compiler: ‘Apple clang version 12.0.0 (clang-1200.0.32.27)’
using SDK: ‘MacOSX11.0.sdk’
clang -arch arm64 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I"/Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/Rcpp/include/"  -I"/Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/RcppEigen/include/"  -I"/Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/RcppEigen/include/unsupported"  -I"/Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/BH/include" -I"/Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/StanHeaders/include/src/"  -I"/Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/StanHeaders/include/"  -I"/Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/RcppParallel/include/"  -I"/Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/rstan/include" -DEIGEN_NO_DEBUG  -DBOOST_DISABLE_ASSERTS  -DBOOST_PENDING_INTEGER_LOG2_HPP  -DSTAN_THREADS  -DUSE_STANC3 -DSTRICT_R_HEADERS  -DBOOST_PHOENIX_NO_VARIADIC_EXPRESSION  -D_HAS_AUTO_PTR_ETC=0  -include '/Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/StanHeaders/include/stan/math/prim/fun/Eigen.hpp'  -D_REENTRANT -DRCPP_PARALLEL_USE_TBB=1   -I/opt/R/arm64/include    -fPIC  -falign-functions=64 -Wall -g -O2  -c foo.c -o foo.o
In file included from &lt;built-in&gt;:1:
In file included from /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/StanHeaders/include/stan/math/prim/fun/Eigen.hpp:22:
In file included from /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/RcppEigen/include/Eigen/Dense:1:
In file included from /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/RcppEigen/include/Eigen/Core:88:
/Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/RcppEigen/include/Eigen/src/Core/util/Macros.h:628:1: error: unknown type name 'namespace'
namespace Eigen {
^
/Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/RcppEigen/include/Eigen/src/Core/util/Macros.h:628:16: error: expected ';' after top level declarator
namespace Eigen {
               ^
               ;
In file included from &lt;built-in&gt;:1:
In file included from /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/StanHeaders/include/stan/math/prim/fun/Eigen.hpp:22:
In file included from /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/RcppEigen/include/Eigen/Dense:1:
/Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/library/RcppEigen/include/Eigen/Core:96:10: fatal error: 'complex' file not found
#include &lt;complex&gt;
         ^~~~~~~~~
3 errors generated.
make: *** [foo.o] Error 1</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Start sampling</code></pre>
</div>
<div class="cell-output cell-output-stdout">
<pre><code>
SAMPLING FOR MODEL 'anon_model' NOW (CHAIN 1).
Chain 1: 
Chain 1: Gradient evaluation took 8e-05 seconds
Chain 1: 1000 transitions using 10 leapfrog steps per transition would take 0.8 seconds.
Chain 1: Adjust your expectations accordingly!
Chain 1: 
Chain 1: 
Chain 1: Iteration:    1 / 2000 [  0%]  (Warmup)
Chain 1: Iteration:  200 / 2000 [ 10%]  (Warmup)
Chain 1: Iteration:  400 / 2000 [ 20%]  (Warmup)
Chain 1: Iteration:  600 / 2000 [ 30%]  (Warmup)
Chain 1: Iteration:  800 / 2000 [ 40%]  (Warmup)
Chain 1: Iteration: 1000 / 2000 [ 50%]  (Warmup)
Chain 1: Iteration: 1001 / 2000 [ 50%]  (Sampling)
Chain 1: Iteration: 1200 / 2000 [ 60%]  (Sampling)
Chain 1: Iteration: 1400 / 2000 [ 70%]  (Sampling)
Chain 1: Iteration: 1600 / 2000 [ 80%]  (Sampling)
Chain 1: Iteration: 1800 / 2000 [ 90%]  (Sampling)
Chain 1: Iteration: 2000 / 2000 [100%]  (Sampling)
Chain 1: 
Chain 1:  Elapsed Time: 2.559 seconds (Warm-up)
Chain 1:                0.846 seconds (Sampling)
Chain 1:                3.405 seconds (Total)
Chain 1: 

SAMPLING FOR MODEL 'anon_model' NOW (CHAIN 2).
Chain 2: 
Chain 2: Gradient evaluation took 4.6e-05 seconds
Chain 2: 1000 transitions using 10 leapfrog steps per transition would take 0.46 seconds.
Chain 2: Adjust your expectations accordingly!
Chain 2: 
Chain 2: 
Chain 2: Iteration:    1 / 2000 [  0%]  (Warmup)
Chain 2: Iteration:  200 / 2000 [ 10%]  (Warmup)
Chain 2: Iteration:  400 / 2000 [ 20%]  (Warmup)
Chain 2: Iteration:  600 / 2000 [ 30%]  (Warmup)
Chain 2: Iteration:  800 / 2000 [ 40%]  (Warmup)
Chain 2: Iteration: 1000 / 2000 [ 50%]  (Warmup)
Chain 2: Iteration: 1001 / 2000 [ 50%]  (Sampling)
Chain 2: Iteration: 1200 / 2000 [ 60%]  (Sampling)
Chain 2: Iteration: 1400 / 2000 [ 70%]  (Sampling)
Chain 2: Iteration: 1600 / 2000 [ 80%]  (Sampling)
Chain 2: Iteration: 1800 / 2000 [ 90%]  (Sampling)
Chain 2: Iteration: 2000 / 2000 [100%]  (Sampling)
Chain 2: 
Chain 2:  Elapsed Time: 2.535 seconds (Warm-up)
Chain 2:                1.016 seconds (Sampling)
Chain 2:                3.551 seconds (Total)
Chain 2: 

SAMPLING FOR MODEL 'anon_model' NOW (CHAIN 3).
Chain 3: 
Chain 3: Gradient evaluation took 4.4e-05 seconds
Chain 3: 1000 transitions using 10 leapfrog steps per transition would take 0.44 seconds.
Chain 3: Adjust your expectations accordingly!
Chain 3: 
Chain 3: 
Chain 3: Iteration:    1 / 2000 [  0%]  (Warmup)
Chain 3: Iteration:  200 / 2000 [ 10%]  (Warmup)
Chain 3: Iteration:  400 / 2000 [ 20%]  (Warmup)
Chain 3: Iteration:  600 / 2000 [ 30%]  (Warmup)
Chain 3: Iteration:  800 / 2000 [ 40%]  (Warmup)
Chain 3: Iteration: 1000 / 2000 [ 50%]  (Warmup)
Chain 3: Iteration: 1001 / 2000 [ 50%]  (Sampling)
Chain 3: Iteration: 1200 / 2000 [ 60%]  (Sampling)
Chain 3: Iteration: 1400 / 2000 [ 70%]  (Sampling)
Chain 3: Iteration: 1600 / 2000 [ 80%]  (Sampling)
Chain 3: Iteration: 1800 / 2000 [ 90%]  (Sampling)
Chain 3: Iteration: 2000 / 2000 [100%]  (Sampling)
Chain 3: 
Chain 3:  Elapsed Time: 2.673 seconds (Warm-up)
Chain 3:                0.676 seconds (Sampling)
Chain 3:                3.349 seconds (Total)
Chain 3: 

SAMPLING FOR MODEL 'anon_model' NOW (CHAIN 4).
Chain 4: 
Chain 4: Gradient evaluation took 4.8e-05 seconds
Chain 4: 1000 transitions using 10 leapfrog steps per transition would take 0.48 seconds.
Chain 4: Adjust your expectations accordingly!
Chain 4: 
Chain 4: 
Chain 4: Iteration:    1 / 2000 [  0%]  (Warmup)
Chain 4: Iteration:  200 / 2000 [ 10%]  (Warmup)
Chain 4: Iteration:  400 / 2000 [ 20%]  (Warmup)
Chain 4: Iteration:  600 / 2000 [ 30%]  (Warmup)
Chain 4: Iteration:  800 / 2000 [ 40%]  (Warmup)
Chain 4: Iteration: 1000 / 2000 [ 50%]  (Warmup)
Chain 4: Iteration: 1001 / 2000 [ 50%]  (Sampling)
Chain 4: Iteration: 1200 / 2000 [ 60%]  (Sampling)
Chain 4: Iteration: 1400 / 2000 [ 70%]  (Sampling)
Chain 4: Iteration: 1600 / 2000 [ 80%]  (Sampling)
Chain 4: Iteration: 1800 / 2000 [ 90%]  (Sampling)
Chain 4: Iteration: 2000 / 2000 [100%]  (Sampling)
Chain 4: 
Chain 4:  Elapsed Time: 2.6 seconds (Warm-up)
Chain 4:                0.663 seconds (Sampling)
Chain 4:                3.263 seconds (Total)
Chain 4: </code></pre>
</div>
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="fu">posterior_summary</span>(biolog_mod) <span class="sc">%&gt;%</span></span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">round</span>(<span class="at">digits =</span> <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>               Estimate Est.Error     Q2.5    Q97.5
b_Intercept      -3.533     0.364   -4.303   -2.866
b_Eterrestrial    1.024     0.217    0.605    1.441
b_Sspatial        0.542     0.202    0.142    0.942
bsp_moT           0.158     0.025    0.113    0.211
simo_moT1[1]      0.047     0.043    0.001    0.160
simo_moT1[2]      0.042     0.041    0.001    0.154
simo_moT1[3]      0.049     0.044    0.002    0.164
simo_moT1[4]      0.054     0.050    0.001    0.186
simo_moT1[5]      0.044     0.039    0.002    0.146
simo_moT1[6]      0.041     0.038    0.001    0.138
simo_moT1[7]      0.040     0.037    0.001    0.137
simo_moT1[8]      0.038     0.036    0.001    0.130
simo_moT1[9]      0.032     0.031    0.001    0.116
simo_moT1[10]     0.051     0.046    0.001    0.168
simo_moT1[11]     0.068     0.059    0.002    0.217
simo_moT1[12]     0.141     0.093    0.007    0.343
simo_moT1[13]     0.075     0.063    0.002    0.233
simo_moT1[14]     0.103     0.077    0.004    0.282
simo_moT1[15]     0.114     0.078    0.005    0.291
simo_moT1[16]     0.060     0.052    0.002    0.192
lprior           19.665     0.952   17.477   21.218
lp__           -312.623     3.455 -320.326 -307.002</code></pre>
</div>
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Hypothesis testing</span></span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a><span class="do">## H1</span></span>
<span id="cb18-3"><a href="#cb18-3" aria-hidden="true" tabindex="-1"></a><span class="do">### Calculate contrasts</span></span>
<span id="cb18-4"><a href="#cb18-4" aria-hidden="true" tabindex="-1"></a>h1_data_2016 <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb18-5"><a href="#cb18-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="fu">c</span>(<span class="st">"marine"</span>, <span class="st">"terrestrial"</span>),</span>
<span id="cb18-6"><a href="#cb18-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="fu">c</span>(<span class="st">"aspatial"</span>, <span class="st">"spatial"</span>),</span>
<span id="cb18-7"><a href="#cb18-7" aria-hidden="true" tabindex="-1"></a>  <span class="at">T =</span> <span class="dv">2016</span></span>
<span id="cb18-8"><a href="#cb18-8" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb18-9"><a href="#cb18-9" aria-hidden="true" tabindex="-1"></a>h1_pred_2016 <span class="ot">&lt;-</span> <span class="fu">posterior_predict</span>(biolog_mod,</span>
<span id="cb18-10"><a href="#cb18-10" aria-hidden="true" tabindex="-1"></a>                                  <span class="at">newdata =</span> h1_data_2016,</span>
<span id="cb18-11"><a href="#cb18-11" aria-hidden="true" tabindex="-1"></a>                                  <span class="at">ndraws =</span> <span class="dv">1000</span>)</span>
<span id="cb18-12"><a href="#cb18-12" aria-hidden="true" tabindex="-1"></a>h1_data_2023 <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb18-13"><a href="#cb18-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="fu">c</span>(<span class="st">"marine"</span>, <span class="st">"terrestrial"</span>),</span>
<span id="cb18-14"><a href="#cb18-14" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="fu">c</span>(<span class="st">"aspatial"</span>, <span class="st">"spatial"</span>),</span>
<span id="cb18-15"><a href="#cb18-15" aria-hidden="true" tabindex="-1"></a>  <span class="at">T =</span> <span class="dv">2023</span></span>
<span id="cb18-16"><a href="#cb18-16" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb18-17"><a href="#cb18-17" aria-hidden="true" tabindex="-1"></a>h1_pred_2023 <span class="ot">&lt;-</span> <span class="fu">posterior_predict</span>(biolog_mod,</span>
<span id="cb18-18"><a href="#cb18-18" aria-hidden="true" tabindex="-1"></a>                                  <span class="at">newdata =</span> h1_data_2023,</span>
<span id="cb18-19"><a href="#cb18-19" aria-hidden="true" tabindex="-1"></a>                                  <span class="at">ndraws =</span> <span class="dv">1000</span>)</span>
<span id="cb18-20"><a href="#cb18-20" aria-hidden="true" tabindex="-1"></a>h1_contrast <span class="ot">&lt;-</span> (h1_pred_2023 <span class="sc">-</span> h1_pred_2016) <span class="sc">%&gt;%</span> </span>
<span id="cb18-21"><a href="#cb18-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">t</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb18-22"><a href="#cb18-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cbind</span>(<span class="fu">select</span>(h1_data_2016, E, S), .) <span class="sc">%&gt;%</span> </span>
<span id="cb18-23"><a href="#cb18-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="sc">-</span><span class="fu">c</span>(E, S), <span class="at">names_to =</span> <span class="st">"draw"</span>, <span class="at">values_to =</span> <span class="st">"contrast"</span>)</span>
<span id="cb18-24"><a href="#cb18-24" aria-hidden="true" tabindex="-1"></a><span class="do">### Plot contrasts</span></span>
<span id="cb18-25"><a href="#cb18-25" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(h1_contrast, <span class="fu">aes</span>(contrast, <span class="at">fill =</span> S)) <span class="sc">+</span></span>
<span id="cb18-26"><a href="#cb18-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">y =</span> <span class="fu">after_stat</span>(prop)), <span class="at">position =</span> <span class="st">"dodge"</span>) <span class="sc">+</span></span>
<span id="cb18-27"><a href="#cb18-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span>E) <span class="sc">+</span></span>
<span id="cb18-28"><a href="#cb18-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">"firebrick"</span>, <span class="st">"cornflowerblue"</span>)) <span class="sc">+</span></span>
<span id="cb18-29"><a href="#cb18-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">1</span>), <span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb18-30"><a href="#cb18-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Contrast of F</span><span class="sc">\n</span><span class="st">(2023 vs 2016)"</span>,</span>
<span id="cb18-31"><a href="#cb18-31" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Proportion"</span>,</span>
<span id="cb18-32"><a href="#cb18-32" aria-hidden="true" tabindex="-1"></a>       <span class="at">fill =</span> <span class="st">"Sensor type"</span>) <span class="sc">+</span></span>
<span id="cb18-33"><a href="#cb18-33" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="registeredreport_files/figure-html/simulation-3.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a><span class="do">### Table of contrasts</span></span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>h1_contrast <span class="sc">%&gt;%</span> </span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(E, S, contrast) <span class="sc">%&gt;%</span> </span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(E, S) <span class="sc">%&gt;%</span> </span>
<span id="cb19-5"><a href="#cb19-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">n =</span> n <span class="sc">/</span> <span class="fu">sum</span>(n)) <span class="sc">%&gt;%</span> </span>
<span id="cb19-6"><a href="#cb19-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb19-7"><a href="#cb19-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_wider</span>(<span class="at">names_from =</span> <span class="st">"contrast"</span>, <span class="at">values_from =</span> <span class="st">"n"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb19-8"><a href="#cb19-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="fu">across</span>(<span class="sc">-</span><span class="fu">c</span>(E, S), scales<span class="sc">::</span>percent))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 4 × 5
  E           S        `-1`  `0`   `1`  
  &lt;chr&gt;       &lt;chr&gt;    &lt;chr&gt; &lt;chr&gt; &lt;chr&gt;
1 marine      aspatial 4.5%  71.0% 24.5%
2 marine      spatial  6.4%  57.8% 35.8%
3 terrestrial aspatial 10.4% 45.5% 44.1%
4 terrestrial spatial  10.3% 45.3% 44.4%</code></pre>
</div>
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a><span class="do">## H2</span></span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a><span class="do">### Calculate contrasts</span></span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a>h2_data_marine <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="st">"marine"</span>,</span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="fu">c</span>(<span class="st">"aspatial"</span>, <span class="st">"spatial"</span>),</span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">T =</span> <span class="dv">2023</span></span>
<span id="cb21-7"><a href="#cb21-7" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb21-8"><a href="#cb21-8" aria-hidden="true" tabindex="-1"></a>h2_pred_marine <span class="ot">&lt;-</span> <span class="fu">posterior_predict</span>(biolog_mod,</span>
<span id="cb21-9"><a href="#cb21-9" aria-hidden="true" tabindex="-1"></a>                                    <span class="at">newdata =</span> h2_data_marine,</span>
<span id="cb21-10"><a href="#cb21-10" aria-hidden="true" tabindex="-1"></a>                                    <span class="at">ndraws =</span> <span class="dv">1000</span>)</span>
<span id="cb21-11"><a href="#cb21-11" aria-hidden="true" tabindex="-1"></a>h2_data_terrestrial <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb21-12"><a href="#cb21-12" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="st">"terrestrial"</span>,</span>
<span id="cb21-13"><a href="#cb21-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="fu">c</span>(<span class="st">"aspatial"</span>, <span class="st">"spatial"</span>),</span>
<span id="cb21-14"><a href="#cb21-14" aria-hidden="true" tabindex="-1"></a>  <span class="at">T =</span> <span class="dv">2023</span></span>
<span id="cb21-15"><a href="#cb21-15" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb21-16"><a href="#cb21-16" aria-hidden="true" tabindex="-1"></a>h2_pred_terrestrial <span class="ot">&lt;-</span> <span class="fu">posterior_predict</span>(biolog_mod,</span>
<span id="cb21-17"><a href="#cb21-17" aria-hidden="true" tabindex="-1"></a>                                         <span class="at">newdata =</span> h2_data_terrestrial,</span>
<span id="cb21-18"><a href="#cb21-18" aria-hidden="true" tabindex="-1"></a>                                         <span class="at">ndraws =</span> <span class="dv">1000</span>)</span>
<span id="cb21-19"><a href="#cb21-19" aria-hidden="true" tabindex="-1"></a>h2_contrast <span class="ot">&lt;-</span> (h2_pred_terrestrial <span class="sc">-</span> h2_pred_marine) <span class="sc">%&gt;%</span> </span>
<span id="cb21-20"><a href="#cb21-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">t</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb21-21"><a href="#cb21-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cbind</span>(<span class="fu">select</span>(h2_data_marine, S), .) <span class="sc">%&gt;%</span> </span>
<span id="cb21-22"><a href="#cb21-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="sc">-</span><span class="fu">c</span>(S), <span class="at">names_to =</span> <span class="st">"draw"</span>, <span class="at">values_to =</span> <span class="st">"contrast"</span>)</span>
<span id="cb21-23"><a href="#cb21-23" aria-hidden="true" tabindex="-1"></a><span class="do">### Plot contrasts</span></span>
<span id="cb21-24"><a href="#cb21-24" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(h2_contrast, <span class="fu">aes</span>(contrast, <span class="at">fill =</span> S)) <span class="sc">+</span></span>
<span id="cb21-25"><a href="#cb21-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">y =</span> <span class="fu">after_stat</span>(prop)), <span class="at">position =</span> <span class="st">"dodge"</span>) <span class="sc">+</span></span>
<span id="cb21-26"><a href="#cb21-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">"firebrick"</span>, <span class="st">"cornflowerblue"</span>)) <span class="sc">+</span></span>
<span id="cb21-27"><a href="#cb21-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">1</span>), <span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb21-28"><a href="#cb21-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Contrast of F</span><span class="sc">\n</span><span class="st">(Terrestrial vs marine)"</span>,</span>
<span id="cb21-29"><a href="#cb21-29" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Proportion"</span>,</span>
<span id="cb21-30"><a href="#cb21-30" aria-hidden="true" tabindex="-1"></a>       <span class="at">fill =</span> <span class="st">"Sensor type"</span>) <span class="sc">+</span></span>
<span id="cb21-31"><a href="#cb21-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="registeredreport_files/figure-html/simulation-4.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a><span class="do">### Table of contrasts</span></span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a>h2_contrast <span class="sc">%&gt;%</span> </span>
<span id="cb22-3"><a href="#cb22-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(S, contrast) <span class="sc">%&gt;%</span> </span>
<span id="cb22-4"><a href="#cb22-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(S) <span class="sc">%&gt;%</span> </span>
<span id="cb22-5"><a href="#cb22-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">n =</span> n <span class="sc">/</span> <span class="fu">sum</span>(n)) <span class="sc">%&gt;%</span> </span>
<span id="cb22-6"><a href="#cb22-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb22-7"><a href="#cb22-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_wider</span>(<span class="at">names_from =</span> <span class="st">"contrast"</span>, <span class="at">values_from =</span> <span class="st">"n"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb22-8"><a href="#cb22-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="fu">across</span>(<span class="sc">-</span>S, scales<span class="sc">::</span>percent))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 2 × 4
  S        `-1`  `0`   `1`  
  &lt;chr&gt;    &lt;chr&gt; &lt;chr&gt; &lt;chr&gt;
1 aspatial 12.9% 49.2% 37.9%
2 spatial  16.1% 46.7% 37.2%</code></pre>
</div>
<div class="sourceCode cell-code" id="cb24"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a><span class="do">## H3</span></span>
<span id="cb24-2"><a href="#cb24-2" aria-hidden="true" tabindex="-1"></a><span class="do">### Calculate contrasts</span></span>
<span id="cb24-3"><a href="#cb24-3" aria-hidden="true" tabindex="-1"></a>h3_data_aspatial <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb24-4"><a href="#cb24-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="fu">c</span>(<span class="st">"marine"</span>, <span class="st">"terrestrial"</span>),</span>
<span id="cb24-5"><a href="#cb24-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="st">"aspatial"</span>,</span>
<span id="cb24-6"><a href="#cb24-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">T =</span> <span class="dv">2023</span></span>
<span id="cb24-7"><a href="#cb24-7" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb24-8"><a href="#cb24-8" aria-hidden="true" tabindex="-1"></a>h3_pred_aspatial <span class="ot">&lt;-</span> <span class="fu">posterior_predict</span>(biolog_mod,</span>
<span id="cb24-9"><a href="#cb24-9" aria-hidden="true" tabindex="-1"></a>                                      <span class="at">newdata =</span> h3_data_aspatial,</span>
<span id="cb24-10"><a href="#cb24-10" aria-hidden="true" tabindex="-1"></a>                                      <span class="at">ndraws =</span> <span class="dv">1000</span>)</span>
<span id="cb24-11"><a href="#cb24-11" aria-hidden="true" tabindex="-1"></a>h3_data_spatial <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb24-12"><a href="#cb24-12" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="fu">c</span>(<span class="st">"marine"</span>, <span class="st">"terrestrial"</span>),</span>
<span id="cb24-13"><a href="#cb24-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="st">"spatial"</span>,</span>
<span id="cb24-14"><a href="#cb24-14" aria-hidden="true" tabindex="-1"></a>  <span class="at">T =</span> <span class="dv">2023</span></span>
<span id="cb24-15"><a href="#cb24-15" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb24-16"><a href="#cb24-16" aria-hidden="true" tabindex="-1"></a>h3_pred_spatial <span class="ot">&lt;-</span> <span class="fu">posterior_predict</span>(biolog_mod,</span>
<span id="cb24-17"><a href="#cb24-17" aria-hidden="true" tabindex="-1"></a>                                     <span class="at">newdata =</span> h3_data_spatial,</span>
<span id="cb24-18"><a href="#cb24-18" aria-hidden="true" tabindex="-1"></a>                                     <span class="at">ndraws =</span> <span class="dv">1000</span>)</span>
<span id="cb24-19"><a href="#cb24-19" aria-hidden="true" tabindex="-1"></a>h3_contrast <span class="ot">&lt;-</span> (h3_pred_spatial <span class="sc">-</span> h3_pred_aspatial) <span class="sc">%&gt;%</span> </span>
<span id="cb24-20"><a href="#cb24-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">t</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb24-21"><a href="#cb24-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">cbind</span>(<span class="fu">select</span>(h3_data_aspatial, E), .) <span class="sc">%&gt;%</span> </span>
<span id="cb24-22"><a href="#cb24-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="sc">-</span>E, <span class="at">names_to =</span> <span class="st">"draw"</span>, <span class="at">values_to =</span> <span class="st">"contrast"</span>)</span>
<span id="cb24-23"><a href="#cb24-23" aria-hidden="true" tabindex="-1"></a><span class="do">### Plot contrasts</span></span>
<span id="cb24-24"><a href="#cb24-24" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(h3_contrast, <span class="fu">aes</span>(contrast, <span class="at">fill =</span> E)) <span class="sc">+</span></span>
<span id="cb24-25"><a href="#cb24-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="fu">aes</span>(<span class="at">y =</span> <span class="fu">after_stat</span>(prop)), <span class="at">position =</span> <span class="st">"dodge"</span>) <span class="sc">+</span></span>
<span id="cb24-26"><a href="#cb24-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">"mediumpurple"</span>, <span class="st">"tan"</span>)) <span class="sc">+</span></span>
<span id="cb24-27"><a href="#cb24-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">1</span>), <span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb24-28"><a href="#cb24-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Contrast of F</span><span class="sc">\n</span><span class="st">(Spatial vs aspatial)"</span>,</span>
<span id="cb24-29"><a href="#cb24-29" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Proportion"</span>,</span>
<span id="cb24-30"><a href="#cb24-30" aria-hidden="true" tabindex="-1"></a>       <span class="at">fill =</span> <span class="st">"Ecosystem"</span>) <span class="sc">+</span></span>
<span id="cb24-31"><a href="#cb24-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="registeredreport_files/figure-html/simulation-5.png" class="img-fluid" width="672"></p>
</div>
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a><span class="do">### Table of contrasts</span></span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a>h3_contrast <span class="sc">%&gt;%</span> </span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(E, contrast) <span class="sc">%&gt;%</span> </span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(E) <span class="sc">%&gt;%</span> </span>
<span id="cb25-5"><a href="#cb25-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">n =</span> n <span class="sc">/</span> <span class="fu">sum</span>(n)) <span class="sc">%&gt;%</span> </span>
<span id="cb25-6"><a href="#cb25-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb25-7"><a href="#cb25-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_wider</span>(<span class="at">names_from =</span> <span class="st">"contrast"</span>, <span class="at">values_from =</span> <span class="st">"n"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb25-8"><a href="#cb25-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="fu">across</span>(<span class="sc">-</span>E, scales<span class="sc">::</span>percent))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 2 × 4
  E           `-1`  `0`   `1`  
  &lt;chr&gt;       &lt;chr&gt; &lt;chr&gt; &lt;chr&gt;
1 marine      17.5% 54.8% 27.7%
2 terrestrial 17.0% 52.6% 30.4%</code></pre>
</div>
</div>

</section>
</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body" data-line-spacing="2" role="list">
<div id="ref-rutz2009" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Rutz, C. &amp; Hays, G. C. <a href="https://doi.org/10.1098/rsbl.2009.0089">New frontiers in biologging science</a>. <em>Biology Letters</em> <strong>5</strong>, 289–292 (2009).</div>
</div>
<div id="ref-kooyman1966" class="csl-entry" role="listitem">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Kooyman, G. L. <a href="https://doi.org/10.1126/science.151.3717.1553">Maximum Diving Capacities of the Weddell Seal,<span><em>Leptonychotes weddelli</em></span></a>. <em>Science</em> <strong>151</strong>, 1553–1554 (1966).</div>
</div>
<div id="ref-wilmers2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Wilmers, C. C. <em>et al.</em> <a href="https://doi.org/10.1890/14-1401.1">The golden age of bio<span>-</span>logging: how animal<span>-</span>borne sensors are advancing the frontiers of ecology</a>. <em>Ecology</em> <strong>96</strong>, 1741–1753 (2015).</div>
</div>
<div id="ref-weinzierl2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Weinzierl, R. <em>et al.</em> <a href="https://doi.org/10.1002/ece3.2585">Wind estimation based on thermal soaring of birds</a>. <em>Ecology and Evolution</em> <strong>6</strong>, 8706–8718 (2016).</div>
</div>
<div id="ref-yonehara2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Yonehara, Y. <em>et al.</em> <a href="https://doi.org/10.1073/pnas.1523853113">Flight paths of seabirds soaring over the ocean surface enable measurement of fine-scale wind speed and direction</a>. <em>Proceedings of the National Academy of Sciences</em> <strong>113</strong>, 9039–9044 (2016).</div>
</div>
<div id="ref-treasure2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Treasure, A. <em>et al.</em> <a href="https://doi.org/10.5670/oceanog.2017.234">Marine mammals exploring the oceans pole to pole: A review of the MEOP consortium</a>. <em>Oceanography</em> <strong>30</strong>, 132–138 (2017).</div>
</div>
<div id="ref-kays2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Kays, R. <em>et al.</em> <a href="https://doi.org/10.1111/2041-210x.13767">The Movebank system for studying global animal movement and demography</a>. <em>Methods in Ecology and Evolution</em> <strong>13</strong>, 419–431 (2021).</div>
</div>
<div id="ref-denouden2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Ouden, O. F. C. den, Smets, P. S. M., Assink, J. D. &amp; Evers, L. G. <a href="https://doi.org/10.1029/2021gl094555">A Bird<span>’</span>s<span>-</span>Eye View on Ambient Infrasonic Soundscapes</a>. <em>Geophysical Research Letters</em> <strong>48</strong>, (2021).</div>
</div>
<div id="ref-nathan2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Nathan, R. <em>et al.</em> <a href="https://doi.org/10.1126/science.abg1780">Big-data approaches lead to an increased understanding of the ecology of animal movement</a>. <em>Science</em> <strong>375</strong>, (2022).</div>
</div>
<div id="ref-grémillet2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Grémillet, D., Chevallier, D. &amp; Guinet, C. <a href="https://doi.org/10.1093/icesjms/fsac059">Big data approaches to the spatial ecology and conservation of marine megafauna</a>. <em>ICES Journal of Marine Science</em> <strong>79</strong>, 975–986 (2022).</div>
</div>
<div id="ref-beal2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Beal, M. <em>et al.</em> <a href="https://doi.org/10.1126/sciadv.abd7225">Global political responsibility for the conservation of albatrosses and large petrels</a>. <em>Science Advances</em> <strong>7</strong>, (2021).</div>
</div>
<div id="ref-queiroz2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Queiroz, N. <em>et al.</em> <a href="https://doi.org/10.1038/s41586-019-1444-4">Global spatial risk assessment of sharks under the footprint of fisheries</a>. <em>Nature</em> <strong>572</strong>, 461–466 (2019).</div>
</div>
<div id="ref-hindell2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Hindell, M. A. <em>et al.</em> <a href="https://doi.org/10.1038/s41586-020-2126-y">Tracking of marine predators to protect Southern Ocean ecosystems</a>. <em>Nature</em> <strong>580</strong>, 87–92 (2020).</div>
</div>
<div id="ref-lowndes2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Lowndes, J. S. S. <em>et al.</em> <a href="https://doi.org/10.1038/s41559-017-0160">Our path to better science in less time using open data science tools</a>. <em>Nature Ecology &amp; Evolution</em> <strong>1</strong>, 1–7 (2017).</div>
</div>
<div id="ref-hampton2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Hampton, S. E. <em>et al.</em> <a href="https://doi.org/10.1093/biosci/bix025">Skills and Knowledge for Data-Intensive Environmental Research</a>. <em>BioScience</em> <strong>67</strong>, 546–557 (2017).</div>
</div>
<div id="ref-peterson2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Peterson, A. T., Asase, A., Canhos, D., Souza, S. de &amp; Wieczorek, J. <a href="https://doi.org/10.3897/bdj.6.e26826">Data leakage and loss in biodiversity informatics</a>. <em>Biodiversity Data Journal</em> <strong>6</strong>, (2018).</div>
</div>
<div id="ref-wilkinson2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">Wilkinson, M. D. <em>et al.</em> <a href="https://doi.org/10.1038/sdata.2016.18">The FAIR Guiding Principles for scientific data management and stewardship</a>. <em>Scientific Data</em> <strong>3</strong>, 160018 (2016).</div>
</div>
<div id="ref-halpern2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Halpern, B. S. <em>et al.</em> Ecological Synthesis and Its Role in Advancing Knowledge. <em>BioScience</em> (2020) doi:<a href="https://doi.org/10.1093/biosci/biaa105">10.1093/biosci/biaa105</a>.</div>
</div>
<div id="ref-fredston2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Fredston, A. L. &amp; Lowndes, J. S. S. <a href="https://doi.org/10.1146/annurev-marine-041723-094741">Welcoming More Participation in Open Data Science for the Oceans</a>. <em>Annual Review of Marine Science</em> <strong>16</strong>, (2023).</div>
</div>
<div id="ref-roche2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Roche, D. G. <em>et al.</em> <a href="https://doi.org/10.1111/cobi.13835">Closing the knowledge<span>-</span>action gap in conservation with open science</a>. <em>Conservation Biology</em> <strong>36</strong>, (2021).</div>
</div>
<div id="ref-buxton2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">Buxton, R. T. <em>et al.</em> <a href="https://doi.org/10.1111/csp2.329">Avoiding wasted research resources in conservation science</a>. <em>Conservation Science and Practice</em> <strong>3</strong>, (2021).</div>
</div>
<div id="ref-urbano2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Urbano, F. &amp; Cagnacci, F. <a href="https://doi.org/10.3389/fevo.2021.727023">Data management and sharing for collaborative science: Lessons learnt from the euromammals initiative</a>. <em>Frontiers in Ecology and Evolution</em> <strong>9</strong>, (2021).</div>
</div>
<div id="ref-sequeira2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Sequeira, A. M. M. <em>et al.</em> <a href="https://doi.org/10.1111/2041-210x.13593">A standardisation framework for bio<span>-</span>logging data to advance ecological research and conservation</a>. <em>Methods in Ecology and Evolution</em> <strong>12</strong>, 996–1007 (2021).</div>
</div>
<div id="ref-dettki2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Dettki, H., Arlt, D., Bäckman, J. &amp; Blanchet, M. <a href="https://doi.org/10.3897/biss.7.111438">Key-value pairs and NoSQL databases: A novel concept to manage biologging data in data repositories</a>. <em>Biodiversity Information Science and Standards</em> <strong>7</strong>, (2023).</div>
</div>
<div id="ref-kranstauber2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">25. </div><div class="csl-right-inline">Kranstauber, B. <em>et al.</em> <a href="https://doi.org/10.1016/j.envsoft.2010.12.005">The Movebank data model for animal tracking</a>. <em>Environmental Modelling &amp; Software</em> <strong>26</strong>, 834–835 (2011).</div>
</div>
<div id="ref-kranstauber2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Kranstauber, B., Safi, K. &amp; Scharf, A. K. <em><a href="https://CRAN.R-project.org/package=move2">move2: Processing and analysing animal trajectories</a></em>. (2023).</div>
</div>
<div id="ref-wieczorek2012" class="csl-entry" role="listitem">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Wieczorek, J. <em>et al.</em> <a href="https://doi.org/10.1371/journal.pone.0029715">Darwin Core: an evolving community-developed biodiversity data standard</a>. <em>PLoS ONE</em> <strong>7</strong>, e29715 (2012).</div>
</div>
<div id="ref-MPIAB2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Animal Behavior, M. P. I. of. <a href="http://dx.doi.org/10.1038/s41597-019-0042-5">Movebank Attribute Dictionary</a>.</div>
</div>
<div id="ref-campbell2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">Campbell, H. A. <em>et al.</em> <a href="https://doi.org/10.1016/j.scitotenv.2015.01.089">Finding our way: On the sharing and reuse of animal telemetry data in Australasia</a>. <em>Science of The Total Environment</em> <strong>534</strong>, 79–84 (2015).</div>
</div>
<div id="ref-scarpignato2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">30. </div><div class="csl-right-inline">Scarpignato, A. L. <em>et al.</em> <a href="https://doi.org/10.1016/j.biocon.2023.110224">Shortfalls in tracking data available to inform North American migratory bird conservation</a>. <em>Biological Conservation</em> <strong>286</strong>, 110224 (2023).</div>
</div>
<div id="ref-taylor2007" class="csl-entry" role="listitem">
<div class="csl-left-margin">31. </div><div class="csl-right-inline">Taylor, C. F. <em>et al.</em> <a href="https://doi.org/10.1038/nbt1329">The minimum information about a proteomics experiment (MIAPE)</a>. <em>Nature Biotechnology</em> <strong>25</strong>, 887–893 (2007).</div>
</div>
<div id="ref-rund2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">32. </div><div class="csl-right-inline">Rund, S. S. C. <em>et al.</em> <a href="https://doi.org/10.1038/s41597-019-0042-5">MIReAD, a minimum information standard for reporting arthropod abundance data</a>. <em>Scientific Data</em> <strong>6</strong>, (2019).</div>
</div>
<div id="ref-cooke2004" class="csl-entry" role="listitem">
<div class="csl-left-margin">33. </div><div class="csl-right-inline">Cooke, S. J. <em>et al.</em> <a href="https://doi.org/10.1016/j.tree.2004.04.003">Biotelemetry: a mechanistic approach to ecology</a>. <em>Trends in Ecology &amp; Evolution</em> <strong>19</strong>, 334–343 (2004).</div>
</div>
<div id="ref-williams2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">34. </div><div class="csl-right-inline">Williams, H. J. <em>et al.</em> <a href="https://doi.org/10.1111/1365-2656.13094">Optimizing the use of biologgers for movement ecology research</a>. <em>Journal of Animal Ecology</em> <strong>89</strong>, 186–206 (2019).</div>
</div>
<div id="ref-bürkner2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">35. </div><div class="csl-right-inline">Bürkner, P.-C. &amp; Charpentier, E. <a href="https://doi.org/10.1111/bmsp.12195">Modelling monotonic effects of ordinal predictors in Bayesian regression models</a>. <em>British Journal of Mathematical and Statistical Psychology</em> <strong>73</strong>, 420–451 (2020).</div>
</div>
<div id="ref-Stan" class="csl-entry" role="listitem">
<div class="csl-left-margin">36. </div><div class="csl-right-inline">Stan Development Team. <a href="https://mc-stan.org">Stan modeling language users guide and reference manual, version 2.34</a>. (2024).</div>
</div>
<div id="ref-bürkner2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">37. </div><div class="csl-right-inline">Bürkner, P.-C. <a href="https://doi.org/10.32614/rj-2018-017">Advanced Bayesian Multilevel Modeling with the R Package brms</a>. <em>The R Journal</em> <strong>10</strong>, 395 (2018).</div>
</div>
<div id="ref-bürkner2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">38. </div><div class="csl-right-inline">Bürkner, P.-C. <a href="https://doi.org/10.18637/jss.v100.i05">Bayesian Item Response Modeling in <span><em>R</em></span> with <span><strong>brms</strong></span> and <span><em>Stan</em></span></a>. <em>Journal of Statistical Software</em> <strong>100</strong>, (2021).</div>
</div>
<div id="ref-roche2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">39. </div><div class="csl-right-inline">Roche, D. G., Kruuk, L. E. B., Lanfear, R. &amp; Binning, S. A. <a href="https://doi.org/10.1371/journal.pbio.1002295">Public Data Archiving in Ecology and Evolution: How Well Are We Doing?</a> <em>PLOS Biology</em> <strong>13</strong>, e1002295 (2015).</div>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>