<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.551">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Trends, challenges, and opportunities in open biologging data</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="registeredreport_files/libs/clipboard/clipboard.min.js"></script>
<script src="registeredreport_files/libs/quarto-html/quarto.js"></script>
<script src="registeredreport_files/libs/quarto-html/popper.min.js"></script>
<script src="registeredreport_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="registeredreport_files/libs/quarto-html/anchor.min.js"></script>
<link href="registeredreport_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="registeredreport_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="registeredreport_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="registeredreport_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="registeredreport_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction">Introduction</a></li>
  <li><a href="#methods" id="toc-methods" class="nav-link" data-scroll-target="#methods">Methods</a>
  <ul class="collapse">
  <li><a href="#literature-review" id="toc-literature-review" class="nav-link" data-scroll-target="#literature-review">Literature review</a></li>
  <li><a href="#initial-classification" id="toc-initial-classification" class="nav-link" data-scroll-target="#initial-classification">Initial classification</a></li>
  <li><a href="#assessing-open-data" id="toc-assessing-open-data" class="nav-link" data-scroll-target="#assessing-open-data">Assessing open data</a></li>
  <li><a href="#analysis" id="toc-analysis" class="nav-link" data-scroll-target="#analysis">Analysis</a></li>
  <li><a href="#hypothesis-testing" id="toc-hypothesis-testing" class="nav-link" data-scroll-target="#hypothesis-testing">Hypothesis testing</a></li>
  <li><a href="#analysis-of-simulated-data" id="toc-analysis-of-simulated-data" class="nav-link" data-scroll-target="#analysis-of-simulated-data">Analysis of simulated data</a></li>
  </ul></li>
  <li><a href="#planned-tables-and-figures" id="toc-planned-tables-and-figures" class="nav-link" data-scroll-target="#planned-tables-and-figures">Planned tables and figures</a></li>
  </ul>
<div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="registeredreport.docx"><i class="bi bi-file-word"></i>MS Word</a></li></ul></div></nav>
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Trends, challenges, and opportunities in open biologging data</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="introduction" class="level2">
<h2 class="anchored" data-anchor-id="introduction">Introduction</h2>
<p>(499/500 words)</p>
<p>Biologging uses animal-borne sensors to remotely observe animals and their environments<span class="citation" data-cites="rutz2009"><sup><a href="#ref-rutz2009" role="doc-biblioref">1</a></sup></span>. In the six decades since a biologist first attached a modified kitchen timer to a seal<span class="citation" data-cites="kooyman1966"><sup><a href="#ref-kooyman1966" role="doc-biblioref">2</a></sup></span>, biologging has become an invaluable, multidisciplinary tool to address questions in ecology<span class="citation" data-cites="wilmers2015"><sup><a href="#ref-wilmers2015" role="doc-biblioref">3</a></sup></span>, atmospheric science<span class="citation" data-cites="weinzierl2016 yonehara2016"><sup><a href="#ref-weinzierl2016" role="doc-biblioref">4</a>,<a href="#ref-yonehara2016" role="doc-biblioref">5</a></sup></span>, and oceanography<span class="citation" data-cites="treasure2017"><sup><a href="#ref-treasure2017" role="doc-biblioref">6</a></sup></span>. As biologging data continue to grow in size and complexity (e.g., millions of new records daily in Movebank<span class="citation" data-cites="kays2021"><sup><a href="#ref-kays2021" role="doc-biblioref">7</a></sup></span> and novel sensors like infrasound recorders<span class="citation" data-cites="denouden2021"><sup><a href="#ref-denouden2021" role="doc-biblioref">8</a></sup></span>), “big data” methods are a key direction for future developments<span class="citation" data-cites="nathan2022 grémillet2022"><sup><a href="#ref-nathan2022" role="doc-biblioref">9</a>,<a href="#ref-grémillet2022" role="doc-biblioref">10</a></sup></span>. Simultaneously, biologging data are increasingly used to inform conservation efforts across taxonomic groups<span class="citation" data-cites="beal2021 queiroz2019"><sup><a href="#ref-beal2021" role="doc-biblioref">11</a>,<a href="#ref-queiroz2019" role="doc-biblioref">12</a></sup></span> and regions<span class="citation" data-cites="hindell2020"><sup><a href="#ref-hindell2020" role="doc-biblioref">13</a></sup></span>. However, the envisioned impacts of aggregated biologging data on research and conservation are hindered by a lack of data standardization, discoverability, and accessibility<span class="citation" data-cites="sequeira2021"><sup><a href="#ref-sequeira2021" role="doc-biblioref">14</a></sup></span>, which has become a growing matter of practical and ethical concern<span class="citation" data-cites="rutz2022"><sup><a href="#ref-rutz2022" role="doc-biblioref">15</a></sup></span>. Currently, there is no systematic review of biologging studies, instruments used, species investigated, or open biologging datasets. Insights from such a review would support collaborative data sharing and curation efforts, and direct approaches to drive greater research and conservation impacts.</p>
<p>Open science practices, such as publishing data and code alongside manuscripts, improve research transparency and efficiency<span class="citation" data-cites="lowndes2017 hampton2017"><sup><a href="#ref-lowndes2017" role="doc-biblioref">16</a>,<a href="#ref-hampton2017" role="doc-biblioref">17</a></sup></span>. However, data “openness” is determined by multiple factors, and poor research data management and sharing practices are a major source of biodiversity data loss<span class="citation" data-cites="peterson2018"><sup><a href="#ref-peterson2018" role="doc-biblioref">18</a></sup></span>. Fortunately, the FAIR principles<span class="citation" data-cites="wilkinson2016"><sup><a href="#ref-wilkinson2016" role="doc-biblioref">19</a></sup></span> (<a href="#tbl-fair" class="quarto-xref">Table&nbsp;1</a>) provide a useful framework to facilitate open data, with FAIR data enabling groundbreaking advances in synthesis, biodiversity, and conservation science<span class="citation" data-cites="halpern2020 fredston2023 roche2021 buxton2021"><sup><a href="#ref-halpern2020" role="doc-biblioref">20</a>–<a href="#ref-buxton2021" role="doc-biblioref">23</a></sup></span> .</p>
<p>Numerous efforts within the biologging research community have sought to promote open data. For example, data standards<span class="citation" data-cites="sequeira2021 dettki2023 kranstauber2011"><sup><a href="#ref-sequeira2021" role="doc-biblioref">14</a>,<a href="#ref-dettki2023" role="doc-biblioref">24</a>,<a href="#ref-kranstauber2011" role="doc-biblioref">25</a></sup></span> and domain-specific data repositories (Movebank<span class="citation" data-cites="kays2021"><sup><a href="#ref-kays2021" role="doc-biblioref">7</a></sup></span>, <a href="https://www.seabirdtracking.org">seabirdtracking.org</a>, Euromammal<span class="citation" data-cites="urbano2021"><sup><a href="#ref-urbano2021" role="doc-biblioref">26</a></sup></span>) facilitate the adoption of the FAIR principles (<a href="#tbl-fair" class="quarto-xref">Table&nbsp;1</a>). However, the overall state of open data in biologging research remains unknown, and there are indications that most tracking data remain inaccessible<span class="citation" data-cites="campbell2015 scarpignato2023"><sup><a href="#ref-campbell2015" role="doc-biblioref">27</a>,<a href="#ref-scarpignato2023" role="doc-biblioref">28</a></sup></span>.</p>
<div id="tbl-fair" class="quarto-float anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-fair-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;1: The FAIR principles<span class="citation" data-cites="wilkinson2016"><sup><a href="#ref-wilkinson2016" role="doc-biblioref">19</a></sup></span>, as implemented in biologging cyberinfrastructure, support data sharing and improve research transparency and efficiency.
</figcaption>
<div aria-describedby="tbl-fair-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<table class="table">
<colgroup>
<col style="width: 23%">
<col style="width: 23%">
<col style="width: 28%">
<col style="width: 23%">
</colgroup>
<thead>
<tr class="header">
<th>FAIR principle</th>
<th>Definition</th>
<th>Application to biologging</th>
<th>Example</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Findability</td>
<td>Data and metadata have a globally unique and persistent identifier (e.g., a digital object identifier, DOI) and are indexed in a searchable resource</td>
<td>Data repositories, like the Movebank Data Repository<span class="citation" data-cites="kays2021"><sup><a href="#ref-kays2021" role="doc-biblioref">7</a></sup></span>, improve data discoverability and may assign DOIs to data (avoiding issues with broken hyperlinks, for example)</td>
<td>A tracking dataset deposited in the Movebank Data Repository is findable by its permanent DOI or by searching the repository</td>
</tr>
<tr class="even">
<td>Accessibility</td>
<td>Data and metadata are retrievable by open and universal protocols, such as HTTP</td>
<td>Data repositories allow scientists to retrieve biologging data via a web browser or other open source tools</td>
<td>Publicly available data on Movebank may be downloaded via the website, API, or with the <code>move2</code> R package<span class="citation" data-cites="kranstauber2023"><sup><a href="#ref-kranstauber2023" role="doc-biblioref">29</a></sup></span></td>
</tr>
<tr class="odd">
<td>Interoperability</td>
<td>Data use formal and shared formats and vocabularies</td>
<td>Shared protocols like the Darwin Core standard<span class="citation" data-cites="wieczorek2012"><sup><a href="#ref-wieczorek2012" role="doc-biblioref">30</a></sup></span>, Movebank data model<span class="citation" data-cites="MPIAB2023"><sup><a href="#ref-MPIAB2023" role="doc-biblioref">31</a></sup></span>, and proposed bio-logging standardization framework<span class="citation" data-cites="sequeira2021"><sup><a href="#ref-sequeira2021" role="doc-biblioref">14</a></sup></span> reduce barriers to combining datasets and increase uptake within and across scientific disciplines</td>
<td>Location and environmental data from seal-borne biologgers harmonized to a standard netCDF format facilitated their reuse by oceanographers to study polar regions<span class="citation" data-cites="treasure2017"><sup><a href="#ref-treasure2017" role="doc-biblioref">6</a></sup></span></td>
</tr>
<tr class="even">
<td>Reusability</td>
<td>Data and metadata are richly described and reuse permissions are clearly defined</td>
<td>Data repositories and standards together capture essential context for biologging data and provide licensing options for data reuse</td>
<td>The Movebank Data Repository releases datasets under the CC0 license and the Movebank data model<span class="citation" data-cites="MPIAB2023"><sup><a href="#ref-MPIAB2023" role="doc-biblioref">31</a></sup></span> includes fields important for data reuse (e.g., whether the animal was relocated before release)</td>
</tr>
</tbody>
</table>
</div>
</figure>
</div>
<p>We will conduct a systematic review to quantify the diversity of instruments, species, and open data practices in biologging studies published between 2007-2023. 2007 marked the creation of Movebank, the largest biologging data platform. Our review will test the following three hypotheses:</p>
<p><strong>H1:</strong> Open biologging data practices are increasing over time.</p>
<ul>
<li>Prediction: we expect open data increased slowly 2007-2016, then more rapidly 2016-2023 following the publication of the FAIR principles and many journals adopting open data policies<span class="citation" data-cites="roche2015"><sup><a href="#ref-roche2015" role="doc-biblioref">32</a></sup></span>.</li>
</ul>
<p><strong>H2:</strong> Open biologging data practices vary by ecosystem.</p>
<ul>
<li>Prediction: we expect biologging data to be more open for studies in terrestrial than marine ecosystems, as many leading journals in marine science (e.g.&nbsp;<em>Marine Ecology Progress Series</em>, <em>Marine Mammal Science</em>) do not yet have open data requirements.</li>
</ul>
<p><strong>H3:</strong> Open biologging data practices are greater for spatial than aspatial (e.g., accelerometer) biologging data.</p>
<ul>
<li>Prediction: we expect spatial data to be more open than aspatial data, for which fewer repositories and data standards exist<span class="citation" data-cites="grémillet2022"><sup><a href="#ref-grémillet2022" role="doc-biblioref">10</a></sup></span>.</li>
</ul>
<p>Beyond hypothesis testing, our review will provide the first biologging bibliometric database, itself a valuable form of cyberinfrastrucure for future systematic reviews and meta-analyses. We will also provide a minimum information standard (e.g.&nbsp;MIAPE<span class="citation" data-cites="taylor2007"><sup><a href="#ref-taylor2007" role="doc-biblioref">33</a></sup></span>, MIReAD<span class="citation" data-cites="rund2019"><sup><a href="#ref-rund2019" role="doc-biblioref">34</a></sup></span>) as a resource for authors, journal editors, and funding bodies to facilitate FAIR sharing of biologging data.</p>
</section>
<section id="methods" class="level2">
<h2 class="anchored" data-anchor-id="methods">Methods</h2>
<p>We will systematically review biologging studies of wild animals in the fields of ecology, evolutionary biology, and conservation science published between 2007-2023. We define biologging as animal-borne instruments with sensors and memory for data storage. Studies employing biotelemetry devices (defined as animal-borne instruments that emit a signal for an external receiver)<span class="citation" data-cites="rutz2009 cooke2004"><sup><a href="#ref-rutz2009" role="doc-biblioref">1</a>,<a href="#ref-cooke2004" role="doc-biblioref">35</a></sup></span> will be excluded, because their analyses require data describing the configuration of the receiver network. We make an exception for satellite biotelemetry using the Argos network, because the Argos system resolves transmissions to timestamped locations, and thus provides researchers with data more similar to biologgers.</p>
<section id="literature-review" class="level3">
<h3 class="anchored" data-anchor-id="literature-review">Literature review</h3>
<p>On 2023-08-21, we queried Web of Science Core Collection for papers published between 2007-2023 related to biologging with the following query:</p>
<p><code>animal AND (biologg* OR bio-logg* OR biotelemetry OR electronic tag* OR satellite track* OR GPS telemetry OR satellite telemetry OR satellite transmit* OR GPS collar* OR depth recorder* OR accelerometer OR archival tag*)</code></p>
<p>This yielded 6654 papers. We limited results to relevant Web of Science categories with 100+ papers, leaving 4799 papers. The categories we considered relevant were Ecology, Zoology, Marine Freshwater Biology, Biodiversity Conservation, Multidisciplinary Sciences, Environmental Sciences, Oceanography, Biology, Behavioral Sciences, Ornithology, Evolutionary Biology, and Fisheries. We excluded the following categories with 100+ papers because they typically involve domesticated or lab animals: Agriculture Dairy Animal Science, Veterinary Sciences, Engineering Electrical Electronic, Physiology, Food Science Technology, Engineering Biomedical, Agriculture Multidisciplinary, Computer Science Interdisciplinary Applications, Instruments Instrumentation.</p>
</section>
<section id="initial-classification" class="level3">
<h3 class="anchored" data-anchor-id="initial-classification">Initial classification</h3>
<p>Every paper will be separately reviewed by two individuals to assess its relevance to this review, record the presence or absence of a data availability statement, and categorize the study by sensor type, taxonomy, and ecosystem. In cases where the reviewers disagree, a third reviewer will assess the study to reach a consensus. Papers will be considered relevant if they satisfy the following two characteristics:</p>
<ol type="1">
<li>They involved the collection of previously unpublished biologging data.</li>
<li>The biologgers were deployed on non-domesticated animals in unrestrained habitat. Studies involving domesticated animals (e.g., cattle), animals in managed care (e.g., a zoo or aquarium), or biomedical laboratories will be excluded.</li>
</ol>
<p>We will consider a paper to have a data availability statement (DAS) if it states how to access the data or explicitly states why the data are not available. Any of the following examples would be considered a DAS: “Data were deposited in a repository” with an accompanying DOI, “Data are available at a website” with an accompanying URL, “Data are available upon request”, or “Data withheld for the safety of the tracked animals”.</p>
<p>We will assign sensor types to one or more categories following Williams et al.<span class="citation" data-cites="williams2019"><sup><a href="#ref-williams2019" role="doc-biblioref">36</a></sup></span>: location (e.g., GPS, depth), intrinsic (e.g., accelerometer, internal temperature), or environment (e.g., ambient temperature, camera). Biologgers can combine sensors from multiple categories (e.g., a GPS-accelerometer collar). We will consider all sensors that were reported in the methods.</p>
<p>We will resolve taxonomy to the species level using the Integrated Taxonomic Information System (www.itis.gov). Each study’s ecosystem will be classified as one or more of aquatic (i.e., freshwater), marine (i.e., saltwater), and terrestrial according to the IUCN Red List (https://www.iucnredlist.org).</p>
<p>It was necessary for us to begin the initial classification to properly design the protocol for the open data assessment (next section). We have completed the initial classification for ~25% of studies in our sample. No data have been collected about studies’ open data practices, so our hypotheses are not informed by preliminary data.</p>
</section>
<section id="assessing-open-data" class="level3">
<h3 class="anchored" data-anchor-id="assessing-open-data">Assessing open data</h3>
<p>Following our initial classification, we will randomly sample 600 studies to assess the openness of biologging data. We will stratify our sample by year, ecosystem, and sensor type. We will attempt to sample 50 studies in each combination of the following categories:</p>
<ul>
<li><p>Time period (early 2007-2015, middle 2016-2019, recent 2020-2023)</p></li>
<li><p>Ecosystem (marine, terrestrial)</p></li>
<li><p>Sensor type (spatial, aspatial)</p></li>
</ul>
<p>Freshwater aquatic biologging studies will be excluded from the open data assessment. Our preliminary results from the initial classification indicate freshwater aquatic biologging studies are rare, as most tracking studies in this ecosystem use acoustic telemetry.</p>
<p>Studies may include both spatial (i.e., location) and aspatial (i.e., intrinsic or environmental) sensor data. Therefore, at the time of sampling, we will assign each study to either the aspatial or spatial category and limit our assessment to the corresponding sensor data. Our initial classification preliminary results indicate spatial sensors are deployed in the vast majority of studies (93%), whereas aspatial sensors are deployed less frequently (35% of studies).</p>
<p>For each sampled paper, we will attempt to retrieve the biologging dataset underlying the study. If data are available upon request, we will email the corresponding author to request the data and, if we do not receive a response within two weeks, we will send one follow-up email. Similarly, if we receive a summary of the biologging data, we will send one follow-up email requesting the underlying biologging data. We will record whether each dataset satisfies the following properties:</p>
<ol type="1">
<li><p>Findable ( <span class="math inline">\(f\)</span> ). Do the data have a permanent identifier, such as a DOI? I.e., FAIR principle F1<span class="citation" data-cites="wilkinson2016"><sup><a href="#ref-wilkinson2016" role="doc-biblioref">19</a></sup></span>.</p></li>
<li><p>Accessible ( <span class="math inline">\(a\)</span> ). Are the data retrievable via a standard communications protocol, such as HTTP? I.e., FAIR principle A1<span class="citation" data-cites="wilkinson2016"><sup><a href="#ref-wilkinson2016" role="doc-biblioref">19</a></sup></span>.</p></li>
<li><p>Reusable ( <span class="math inline">\(r\)</span> ). Do the data contain biologging data (raw or processed), as opposed to descriptive statistics or other summaries? I.e., FAIR principle R1<span class="citation" data-cites="wilkinson2016"><sup><a href="#ref-wilkinson2016" role="doc-biblioref">19</a></sup></span>.</p></li>
</ol>
<p>If we cannot retrieve the dataset for a study, we will consider the study to not satisfy any of the three properties listed above.</p>
<p>To clarify our methods for spatial versus aspatial data, consider a hypothetical GPS-accelerometer biologging study sampled in the aspatial category. The authors deposited the GPS data in the Movebank Data Repository (which issues DOIs) and the accelerometer data are available upon request. We contact the author and they provide us with a summary table of the accelerometer data, where each row is a deployment and columns are characteristics of the deployment, such as duration. They do not respond to our follow-up email requesting the underlying biologging data. Then, because this study was sampled in the aspatial category, it would be scored as <span class="math inline">\(f=0,a=0,r=0\)</span> because (<span class="math inline">\(f\)</span>) the accelerometer data do not have a DOI, (<span class="math inline">\(a\)</span>) retrieving the accelerometer data required contacting the author, and (<span class="math inline">\(r\)</span>) only summaries of the accelerometer data were made available. Had this same study been sampled in the spatial category, we would have scored it as <span class="math inline">\(f=1,a=1,r=1\)</span> because (<span class="math inline">\(f\)</span>) the spatial data have a DOI, (<span class="math inline">\(a\)</span>) the data were retrievable via HTTP, and (<span class="math inline">\(r\)</span>) the GPS sensor data were available.</p>
</section>
<section id="analysis" class="level3">
<h3 class="anchored" data-anchor-id="analysis">Analysis</h3>
<p>Our independent variables will be year of publication (<span class="math inline">\(t\)</span>), ecosystem (<span class="math inline">\(e\)</span>), and sensor type (<span class="math inline">\(s\)</span>) [see “Initial classification”]. Our dependent variables will be findability (<span class="math inline">\(f\)</span>), accessibility (<span class="math inline">\(a\)</span>), and reusability (<span class="math inline">\(r\)</span>) [see “Assessing open data”]. We will fit three Bayesian generalized linear models of the form:</p>
<p><span class="math display">\[
\begin{align}
DV &amp;\sim Bernoulli(p) \\
logit(p_i) &amp;= \alpha + \beta_e \times e_i + \beta_s \times s_i + \beta_t \times t_i \\
\end{align}
\]</span></p>
<p>Where <span class="math inline">\(DV\)</span> is the dependent variable (<span class="math inline">\(f\)</span>, <span class="math inline">\(a\)</span>, or <span class="math inline">\(r\)</span>). <span class="math inline">\(e\)</span> and <span class="math inline">\(s\)</span> encode ecosystem (0 for marine, 1 for terrestrial) and sensor type (0 for aspatial, 1 for spatial), respectively. <span class="math inline">\(t\)</span> is the publication year minus the beginning of the study period (i.e., <span class="math inline">\(t=0\)</span> in 2007, …, <span class="math inline">\(t=16\)</span> in 2023).</p>
<p>We use the following priors:</p>
<p><span class="math display">\[
\begin{align}
\alpha &amp;\sim Normal(0, 4) \\
\beta_e &amp;\sim Normal(0, 2) \\
\beta_s &amp;\sim Normal(0, 2) \\
\beta_t &amp;\sim Normal(0, 0.5)
\end{align}
\]</span></p>
<p>We conducted prior predictive checks of our priors. Flatter priors, such as <span class="math inline">\(Normal(0,10)\)</span> for all coefficients, lead to unrealistic values for <span class="math inline">\(p\)</span> (i.e., the probability density of <span class="math inline">\(p\)</span> is concentrated almost entirely near 0 or 1).</p>
<p>We will use Stan<span class="citation" data-cites="Stan"><sup><a href="#ref-Stan" role="doc-biblioref">37</a></sup></span> and brms<span class="citation" data-cites="bürkner2018 bürkner2021"><sup><a href="#ref-bürkner2018" role="doc-biblioref">38</a>,<a href="#ref-bürkner2021" role="doc-biblioref">39</a></sup></span> for model fitting.</p>
<p>We invite feedback from reviewers about statistical design. We considered more complex models, e.g., using splines for <span class="math inline">\(\beta_t\)</span>, treating year as an ordinal variable<span class="citation" data-cites="bürkner2020"><sup><a href="#ref-bürkner2020" role="doc-biblioref">40</a></sup></span>, or allowing interactions between <span class="math inline">\(e\)</span>, <span class="math inline">\(s\)</span>, and <span class="math inline">\(t\)</span>. However, we concluded that the model presented here would answer our research questions with the greatest clarity. If the reviewers think more complex models are essential, we will modify our design accordingly.</p>
</section>
<section id="hypothesis-testing" class="level3">
<h3 class="anchored" data-anchor-id="hypothesis-testing">Hypothesis testing</h3>
<p>We will test hypotheses by testing if the 95% credible interval for the posterior probability distribution of the relevant model coefficients (in all three models) is greater than 0. We will quantify effect sizes using the 95% credible interval for the posterior probability distribution of contrasts in <span class="math inline">\(p(f)\)</span> , <span class="math inline">\(p(a)\)</span>, and <span class="math inline">\(p(r)\)</span> across levels of the relevant category to the hypothesis. We calculate contrasts by sampling the posterior distribution of the probability of a dependent variable (e.g., <span class="math inline">\(p(f)\)</span>) while varying the category of interest. For example, the contrast of <span class="math inline">\(p(f)\)</span> for hypothesis <strong>H1</strong> (open data vary by ecosystem, <span class="math inline">\(e\)</span>), is <span class="math inline">\(p(f|e=terrestrial)-p(f|e=marine)\)</span>, marginal across <span class="math inline">\(s\)</span> and <span class="math inline">\(t\)</span>. See section “Analysis of simulated data” to see our implementation with simulated data.</p>
<section id="h1-open-biologging-data-practices-are-increasing-over-time" class="level4">
<h4 class="anchored" data-anchor-id="h1-open-biologging-data-practices-are-increasing-over-time">H1: Open biologging data practices are increasing over time</h4>
<p>We will test hypothesis <strong>H1</strong> using coefficient <span class="math inline">\(\beta_t\)</span>. We will quantify effect size by contrasting the dependent variable posterior probabilities for three eras: 2007-2015, 2016-2019, and 2020-2023 (see <a href="#fig-h1" class="quarto-xref">Figure&nbsp;2</a> in section “Analysis of simulated data” below).</p>
</section>
<section id="h2-open-biologging-data-practices-vary-by-ecosystem" class="level4">
<h4 class="anchored" data-anchor-id="h2-open-biologging-data-practices-vary-by-ecosystem">H2: Open biologging data practices vary by ecosystem</h4>
<p>We will test hypothesis <strong>H2</strong> using coefficient <span class="math inline">\(\beta_e\)</span>. To quantify effect size, we will contrast posterior probabilities for marine and terrestrial ecosystems (see <a href="#fig-h2" class="quarto-xref">Figure&nbsp;3</a>).</p>
</section>
<section id="h3-open-biologging-data-practices-vary-by-sensor-class" class="level4">
<h4 class="anchored" data-anchor-id="h3-open-biologging-data-practices-vary-by-sensor-class">H3: Open biologging data practices vary by sensor class</h4>
<p>We will test hypothesis <strong>H2</strong> using coefficient <span class="math inline">\(\beta_s\)</span>. To quantify effect size, we will contrast posterior probabilities for aspatial and spatial sensor data (see <a href="#fig-h3" class="quarto-xref">Figure&nbsp;4</a>).</p>
</section>
</section>
<section id="analysis-of-simulated-data" class="level3">
<h3 class="anchored" data-anchor-id="analysis-of-simulated-data">Analysis of simulated data</h3>
<p>We conducted a simulation to demonstrate how we will conduct our analysis. We simulated values for <span class="math inline">\(t\)</span>, <span class="math inline">\(e\)</span>, <span class="math inline">\(s\)</span> (time, ecosystem, and sensor class; independent variables), and <span class="math inline">\(f\)</span> (findability; dependent variable) according to our predictions, to demonstrate how we will perform our analysis. The full analysis will also include dependent variables <span class="math inline">\(a\)</span> (accessibility) and <span class="math inline">\(r\)</span> (reusability).</p>
<p>We set up our analysis by loading necessary packages and setting the seed.</p>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(brms)</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(cowplot)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidybayes)</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">1234</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>The following code simulates our independent and dependent variables. We simulate independent variables according to our stratified sampling plan (<code>biolog_grid</code>). Then we choose values for our parameters <span class="math inline">\(\beta_e\)</span> (<code>beta_E</code>), <span class="math inline">\(\beta_s\)</span> (<code>beta_S</code>), <span class="math inline">\(\beta_t\)</span> (<code>beta_T</code>), and <span class="math inline">\(\alpha\)</span> (<code>alpha</code>), which we use to simulate our dependent variable, <span class="math inline">\(f\)</span> (<code>F</code>). The probability of <span class="math inline">\(f\)</span>, <span class="math inline">\(p(f)\)</span>, increases over time, and <span class="math inline">\(p(f)\)</span> is greater for spatial sensors and terrestrial ecosystems than aspatial sensors and marine ecosystems (<a href="#fig-sim-data" class="quarto-xref">Figure&nbsp;1</a>).</p>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Grid of independent variables</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>biolog_grid <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">era =</span> <span class="fu">c</span>(<span class="st">"early"</span>, <span class="st">"middle"</span>, <span class="st">"recent"</span>),</span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">E =</span> <span class="fu">c</span>(<span class="st">"marine"</span>, <span class="st">"terrestrial"</span>),</span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">S =</span> <span class="fu">c</span>(<span class="st">"aspatial"</span>, <span class="st">"spatial"</span>)</span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">slice</span>(<span class="fu">rep</span>(<span class="fu">seq</span>(<span class="fu">nrow</span>(.)), <span class="at">each =</span> <span class="dv">50</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">T =</span> <span class="fu">case_when</span>(</span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a>    era <span class="sc">==</span> <span class="st">"early"</span>  <span class="sc">~</span> <span class="fu">sample</span>(<span class="dv">2007</span><span class="sc">:</span><span class="dv">2015</span>, <span class="fu">nrow</span>(.), <span class="at">replace =</span> <span class="cn">TRUE</span>),</span>
<span id="cb2-10"><a href="#cb2-10" aria-hidden="true" tabindex="-1"></a>    era <span class="sc">==</span> <span class="st">"middle"</span> <span class="sc">~</span> <span class="fu">sample</span>(<span class="dv">2016</span><span class="sc">:</span><span class="dv">2019</span>, <span class="fu">nrow</span>(.), <span class="at">replace =</span> <span class="cn">TRUE</span>),</span>
<span id="cb2-11"><a href="#cb2-11" aria-hidden="true" tabindex="-1"></a>    era <span class="sc">==</span> <span class="st">"recent"</span> <span class="sc">~</span> <span class="fu">sample</span>(<span class="dv">2020</span><span class="sc">:</span><span class="dv">2023</span>, <span class="fu">nrow</span>(.), <span class="at">replace =</span> <span class="cn">TRUE</span>)</span>
<span id="cb2-12"><a href="#cb2-12" aria-hidden="true" tabindex="-1"></a>  ))</span>
<span id="cb2-13"><a href="#cb2-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-14"><a href="#cb2-14" aria-hidden="true" tabindex="-1"></a><span class="co"># Parameters for simulation</span></span>
<span id="cb2-15"><a href="#cb2-15" aria-hidden="true" tabindex="-1"></a><span class="co"># betas (E, S, T)</span></span>
<span id="cb2-16"><a href="#cb2-16" aria-hidden="true" tabindex="-1"></a>alpha <span class="ot">&lt;-</span> <span class="sc">-</span><span class="dv">4</span></span>
<span id="cb2-17"><a href="#cb2-17" aria-hidden="true" tabindex="-1"></a>beta_E <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="at">marine =</span> <span class="dv">0</span>, <span class="at">terrestrial =</span> <span class="fl">0.5</span>)</span>
<span id="cb2-18"><a href="#cb2-18" aria-hidden="true" tabindex="-1"></a>beta_S <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="at">aspatial =</span> <span class="dv">0</span>, <span class="at">spatial =</span> <span class="fl">1.5</span>)</span>
<span id="cb2-19"><a href="#cb2-19" aria-hidden="true" tabindex="-1"></a>beta_T <span class="ot">=</span> <span class="fl">0.2</span></span>
<span id="cb2-20"><a href="#cb2-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-21"><a href="#cb2-21" aria-hidden="true" tabindex="-1"></a><span class="co"># Run simulation</span></span>
<span id="cb2-22"><a href="#cb2-22" aria-hidden="true" tabindex="-1"></a>inv_logit <span class="ot">&lt;-</span> \(x) <span class="fu">exp</span>(x) <span class="sc">/</span> (<span class="dv">1</span> <span class="sc">+</span> <span class="fu">exp</span>(x))</span>
<span id="cb2-23"><a href="#cb2-23" aria-hidden="true" tabindex="-1"></a>biolog <span class="ot">&lt;-</span> biolog_grid <span class="sc">%&gt;%</span></span>
<span id="cb2-24"><a href="#cb2-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">T2007 =</span> T <span class="sc">-</span> <span class="dv">2007</span>,</span>
<span id="cb2-25"><a href="#cb2-25" aria-hidden="true" tabindex="-1"></a>         <span class="at">logit_p =</span> alpha <span class="sc">+</span> beta_E[E] <span class="sc">+</span> beta_S[S] <span class="sc">+</span> beta_T <span class="sc">*</span> T2007,</span>
<span id="cb2-26"><a href="#cb2-26" aria-hidden="true" tabindex="-1"></a>         <span class="at">p =</span> <span class="fu">inv_logit</span>(logit_p),</span>
<span id="cb2-27"><a href="#cb2-27" aria-hidden="true" tabindex="-1"></a>         <span class="at">F =</span> <span class="fu">rbinom</span>(<span class="fu">nrow</span>(.), <span class="at">size =</span> <span class="dv">1</span>, <span class="at">prob =</span> p))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Visualize data</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>e_palette <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="at">marine =</span> <span class="st">"navy"</span>, <span class="at">terrestrial =</span> <span class="st">"goldenrod"</span>)</span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a>biolog_text <span class="ot">&lt;-</span> <span class="fu">filter</span>(biolog, T <span class="sc">==</span> <span class="dv">2023</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">distinct</span>(E, S, p) <span class="sc">%&gt;%</span> </span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">label =</span> <span class="fu">str_to_title</span>(<span class="fu">paste</span>(E, S, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\n</span><span class="st">"</span>)))</span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(biolog, <span class="fu">aes</span>(T, p, <span class="at">color =</span> E)) <span class="sc">+</span></span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_jitter</span>(<span class="fu">aes</span>(<span class="at">y =</span> F, <span class="at">shape =</span> S), </span>
<span id="cb3-10"><a href="#cb3-10" aria-hidden="true" tabindex="-1"></a>              <span class="at">width =</span> <span class="fl">0.25</span>, <span class="at">height =</span> <span class="fl">0.05</span>,</span>
<span id="cb3-11"><a href="#cb3-11" aria-hidden="true" tabindex="-1"></a>              <span class="at">alpha =</span> <span class="fl">0.5</span>) <span class="sc">+</span></span>
<span id="cb3-12"><a href="#cb3-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>(<span class="fu">aes</span>(<span class="at">linetype =</span> S)) <span class="sc">+</span></span>
<span id="cb3-13"><a href="#cb3-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text</span>(<span class="fu">aes</span>(<span class="at">label =</span> label), biolog_text,</span>
<span id="cb3-14"><a href="#cb3-14" aria-hidden="true" tabindex="-1"></a>            <span class="at">x =</span> <span class="fl">2023.1</span>, <span class="at">hjust =</span> <span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb3-15"><a href="#cb3-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> e_palette) <span class="sc">+</span></span>
<span id="cb3-16"><a href="#cb3-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="cn">NA</span>, <span class="dv">2025</span>), </span>
<span id="cb3-17"><a href="#cb3-17" aria-hidden="true" tabindex="-1"></a>                     <span class="at">breaks =</span> <span class="fu">seq</span>(<span class="dv">2010</span>, <span class="dv">2020</span>, <span class="at">by =</span> <span class="dv">5</span>)) <span class="sc">+</span></span>
<span id="cb3-18"><a href="#cb3-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">y =</span> <span class="fu">expression</span>(<span class="fu">p</span>(f))) <span class="sc">+</span></span>
<span id="cb3-19"><a href="#cb3-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">expand_limits</span>(<span class="at">x =</span> <span class="dv">2025</span>) <span class="sc">+</span></span>
<span id="cb3-20"><a href="#cb3-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>() <span class="sc">+</span></span>
<span id="cb3-21"><a href="#cb3-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>,</span>
<span id="cb3-22"><a href="#cb3-22" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.title.x =</span> <span class="fu">element_blank</span>())</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="fig-sim-data" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-sim-data-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="registeredreport_files/figure-html/fig-sim-data-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-sim-data-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1: In our simulation, probability of findability (<span class="math inline">\(p(f)\)</span>) increases over time. Lines represent <span class="math inline">\(p(f)\)</span>, points are simulated studies. <span class="math inline">\(p(f)\)</span> is greater for terrestrial (gold) than marine (blue) data, and greater for spatial (dashed lines, triangles) than aspatial (solid lines, circles) data.
</figcaption>
</figure>
</div>
</div>
</div>
<p>Here we specify our model, including priors, and fit it to our simulated data.</p>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Fit model</span></span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>biolog_prior <span class="ot">&lt;-</span> <span class="fu">c</span>(</span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>    <span class="fu">set_prior</span>(<span class="at">prior =</span> <span class="st">"normal(0, 4)"</span>, <span class="at">class =</span> <span class="st">"Intercept"</span>),</span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>    <span class="fu">set_prior</span>(<span class="at">prior =</span> <span class="st">"normal(0, 2)"</span>, <span class="at">coef =</span> <span class="st">"Eterrestrial"</span>),</span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">set_prior</span>(<span class="at">prior =</span> <span class="st">"normal(0, 2)"</span>, <span class="at">coef =</span> <span class="st">"Sspatial"</span>),</span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>    <span class="fu">set_prior</span>(<span class="at">prior =</span> <span class="st">"normal(0, 0.5)"</span>, <span class="at">coef =</span> <span class="st">"T2007"</span>)</span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a>biolog_mod <span class="ot">&lt;-</span> <span class="fu">brm</span>(F <span class="sc">~</span> E <span class="sc">+</span> S <span class="sc">+</span> T2007,</span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a>                  <span class="at">data =</span> biolog,</span>
<span id="cb4-10"><a href="#cb4-10" aria-hidden="true" tabindex="-1"></a>                  <span class="at">family =</span> <span class="fu">bernoulli</span>(<span class="at">link =</span> <span class="st">"logit"</span>),</span>
<span id="cb4-11"><a href="#cb4-11" aria-hidden="true" tabindex="-1"></a>                  <span class="at">prior =</span> biolog_prior,</span>
<span id="cb4-12"><a href="#cb4-12" aria-hidden="true" tabindex="-1"></a>                  <span class="at">chains =</span> <span class="dv">4</span>,</span>
<span id="cb4-13"><a href="#cb4-13" aria-hidden="true" tabindex="-1"></a>                  <span class="at">iter =</span> <span class="dv">50000</span>,</span>
<span id="cb4-14"><a href="#cb4-14" aria-hidden="true" tabindex="-1"></a>                  <span class="at">seed =</span> <span class="dv">6789</span>,</span>
<span id="cb4-15"><a href="#cb4-15" aria-hidden="true" tabindex="-1"></a>                  <span class="at">refresh =</span> <span class="dv">0</span>)</span>
<span id="cb4-16"><a href="#cb4-16" aria-hidden="true" tabindex="-1"></a>biolog_draws <span class="ot">&lt;-</span> <span class="fu">as_draws_df</span>(biolog_mod)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p>We test our hypotheses using credible intervals of the parameter estimates and contrasts of the posterior probabilities. In our simulation, the 95% credible intervals for coefficients <span class="math inline">\(\beta_t\)</span>, <span class="math inline">\(\beta_e\)</span>, and <span class="math inline">\(\beta_s\)</span> were greater than 0 (<a href="#fig-h1" class="quarto-xref">Figure&nbsp;2</a>A, <a href="#fig-h2" class="quarto-xref">Figure&nbsp;3</a>A, <a href="#fig-h3" class="quarto-xref">Figure&nbsp;4</a>A). The contrasts quantify the effect sizes in the simulation. <a href="#fig-h1" class="quarto-xref">Figure&nbsp;2</a>B, for example, shows the posterior distribution of the contrast of <span class="math inline">\(p(f)\)</span> from the early to middle periods (above) and from the middle to recent periods (below). The 95% credible interval from early to middle periods was (0.073 - 0.300), indicating the probability of findability increased 7.3% - 30%.</p>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a>h1_dens <span class="ot">&lt;-</span> <span class="fu">density</span>(biolog_draws<span class="sc">$</span>b_T2007)</span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a>h1_dens_dat <span class="ot">&lt;-</span> <span class="fu">tibble</span>(</span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">b_T2007 =</span> h1_dens<span class="sc">$</span>x,</span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">dens =</span> h1_dens<span class="sc">$</span>y,</span>
<span id="cb5-5"><a href="#cb5-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">ci95 =</span> <span class="fu">between</span>(b_T2007, </span>
<span id="cb5-6"><a href="#cb5-6" aria-hidden="true" tabindex="-1"></a>                 <span class="fu">quantile</span>(biolog_draws<span class="sc">$</span>b_T2007, <span class="fl">0.025</span>), </span>
<span id="cb5-7"><a href="#cb5-7" aria-hidden="true" tabindex="-1"></a>                 <span class="fu">quantile</span>(biolog_draws<span class="sc">$</span>b_T2007, <span class="fl">0.975</span>))</span>
<span id="cb5-8"><a href="#cb5-8" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb5-9"><a href="#cb5-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-10"><a href="#cb5-10" aria-hidden="true" tabindex="-1"></a>h1_contrast <span class="ot">&lt;-</span> <span class="fu">map2</span>(</span>
<span id="cb5-11"><a href="#cb5-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">list</span>(<span class="st">"early"</span>, <span class="st">"middle"</span>, <span class="st">"recent"</span>),</span>
<span id="cb5-12"><a href="#cb5-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">list</span>(<span class="dv">2007</span><span class="sc">:</span><span class="dv">2015</span>, <span class="dv">2016</span><span class="sc">:</span><span class="dv">2019</span>, <span class="dv">2020</span><span class="sc">:</span><span class="dv">2023</span>),</span>
<span id="cb5-13"><a href="#cb5-13" aria-hidden="true" tabindex="-1"></a>  <span class="cf">function</span>(era, years) {</span>
<span id="cb5-14"><a href="#cb5-14" aria-hidden="true" tabindex="-1"></a>    dat <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb5-15"><a href="#cb5-15" aria-hidden="true" tabindex="-1"></a>      <span class="at">E =</span> <span class="fu">c</span>(<span class="st">"marine"</span>, <span class="st">"terrestrial"</span>),</span>
<span id="cb5-16"><a href="#cb5-16" aria-hidden="true" tabindex="-1"></a>      <span class="at">S =</span> <span class="fu">c</span>(<span class="st">"aspatial"</span>, <span class="st">"spatial"</span>),</span>
<span id="cb5-17"><a href="#cb5-17" aria-hidden="true" tabindex="-1"></a>      <span class="at">T =</span> years</span>
<span id="cb5-18"><a href="#cb5-18" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">%&gt;%</span> </span>
<span id="cb5-19"><a href="#cb5-19" aria-hidden="true" tabindex="-1"></a>      <span class="fu">mutate</span>(<span class="at">T2007 =</span> T <span class="sc">-</span> <span class="dv">2007</span>)</span>
<span id="cb5-20"><a href="#cb5-20" aria-hidden="true" tabindex="-1"></a>    <span class="fu">linpred_draws</span>(biolog_mod, </span>
<span id="cb5-21"><a href="#cb5-21" aria-hidden="true" tabindex="-1"></a>                  <span class="at">transform =</span> <span class="cn">TRUE</span>, </span>
<span id="cb5-22"><a href="#cb5-22" aria-hidden="true" tabindex="-1"></a>                  <span class="at">newdata =</span> dat) <span class="sc">%&gt;%</span> </span>
<span id="cb5-23"><a href="#cb5-23" aria-hidden="true" tabindex="-1"></a>      <span class="fu">group_by</span>(E, S, .draw) <span class="sc">%&gt;%</span> </span>
<span id="cb5-24"><a href="#cb5-24" aria-hidden="true" tabindex="-1"></a>      <span class="fu">summarize</span>(<span class="at">p_F =</span> <span class="fu">mean</span>(.linpred),</span>
<span id="cb5-25"><a href="#cb5-25" aria-hidden="true" tabindex="-1"></a>                <span class="at">.groups =</span> <span class="st">"drop"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb5-26"><a href="#cb5-26" aria-hidden="true" tabindex="-1"></a>      <span class="fu">mutate</span>(<span class="at">era =</span> era)</span>
<span id="cb5-27"><a href="#cb5-27" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb5-28"><a href="#cb5-28" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span> </span>
<span id="cb5-29"><a href="#cb5-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">list_rbind</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb5-30"><a href="#cb5-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_wider</span>(<span class="at">names_from =</span> <span class="st">"era"</span>, <span class="at">values_from =</span> <span class="st">"p_F"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb5-31"><a href="#cb5-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="st">`</span><span class="at">Early to Middle</span><span class="st">`</span> <span class="ot">=</span> middle <span class="sc">-</span> early,</span>
<span id="cb5-32"><a href="#cb5-32" aria-hidden="true" tabindex="-1"></a>         <span class="st">`</span><span class="at">Middle to Recent</span><span class="st">`</span> <span class="ot">=</span> recent <span class="sc">-</span> middle) <span class="sc">%&gt;%</span> </span>
<span id="cb5-33"><a href="#cb5-33" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="fu">c</span>(<span class="st">`</span><span class="at">Early to Middle</span><span class="st">`</span>, <span class="st">`</span><span class="at">Middle to Recent</span><span class="st">`</span>),</span>
<span id="cb5-34"><a href="#cb5-34" aria-hidden="true" tabindex="-1"></a>               <span class="at">names_to =</span> <span class="st">"Eras"</span>,</span>
<span id="cb5-35"><a href="#cb5-35" aria-hidden="true" tabindex="-1"></a>               <span class="at">values_to =</span> <span class="st">"Contrast"</span>)</span>
<span id="cb5-36"><a href="#cb5-36" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-37"><a href="#cb5-37" aria-hidden="true" tabindex="-1"></a>h1_contrast_dens <span class="ot">&lt;-</span> h1_contrast <span class="sc">%&gt;%</span> </span>
<span id="cb5-38"><a href="#cb5-38" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Eras) <span class="sc">%&gt;%</span> </span>
<span id="cb5-39"><a href="#cb5-39" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_map</span>(\(rows, keys) {</span>
<span id="cb5-40"><a href="#cb5-40" aria-hidden="true" tabindex="-1"></a>    dens <span class="ot">&lt;-</span> <span class="fu">density</span>(rows<span class="sc">$</span>Contrast)</span>
<span id="cb5-41"><a href="#cb5-41" aria-hidden="true" tabindex="-1"></a>    lwr <span class="ot">&lt;-</span> <span class="fu">quantile</span>(rows<span class="sc">$</span>Contrast, <span class="fl">0.025</span>)</span>
<span id="cb5-42"><a href="#cb5-42" aria-hidden="true" tabindex="-1"></a>    upr <span class="ot">&lt;-</span> <span class="fu">quantile</span>(rows<span class="sc">$</span>Contrast, <span class="fl">0.975</span>)</span>
<span id="cb5-43"><a href="#cb5-43" aria-hidden="true" tabindex="-1"></a>    <span class="fu">cross_join</span>(keys,</span>
<span id="cb5-44"><a href="#cb5-44" aria-hidden="true" tabindex="-1"></a>               <span class="fu">tibble</span>(<span class="at">Contrast =</span> dens<span class="sc">$</span>x,</span>
<span id="cb5-45"><a href="#cb5-45" aria-hidden="true" tabindex="-1"></a>                      <span class="at">Density =</span> dens<span class="sc">$</span>y,</span>
<span id="cb5-46"><a href="#cb5-46" aria-hidden="true" tabindex="-1"></a>                      <span class="at">ci_95 =</span> <span class="fu">between</span>(Contrast, lwr, upr)))</span>
<span id="cb5-47"><a href="#cb5-47" aria-hidden="true" tabindex="-1"></a>  }) <span class="sc">%&gt;%</span> </span>
<span id="cb5-48"><a href="#cb5-48" aria-hidden="true" tabindex="-1"></a>  <span class="fu">list_rbind</span>()</span>
<span id="cb5-49"><a href="#cb5-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-50"><a href="#cb5-50" aria-hidden="true" tabindex="-1"></a>beta_t_ci_lwr <span class="ot">&lt;-</span> <span class="fu">quantile</span>(biolog_draws<span class="sc">$</span>b_T2007, <span class="fl">0.025</span>)</span>
<span id="cb5-51"><a href="#cb5-51" aria-hidden="true" tabindex="-1"></a>beta_t_ci_upr <span class="ot">&lt;-</span> <span class="fu">quantile</span>(biolog_draws<span class="sc">$</span>b_T2007, <span class="fl">0.975</span>)</span>
<span id="cb5-52"><a href="#cb5-52" aria-hidden="true" tabindex="-1"></a>early_middle_lwr <span class="ot">&lt;-</span> <span class="fu">with</span>(</span>
<span id="cb5-53"><a href="#cb5-53" aria-hidden="true" tabindex="-1"></a>  h1_contrast, <span class="fu">quantile</span>(Contrast[Eras <span class="sc">==</span> <span class="st">"Early to Middle"</span>], <span class="fl">0.025</span>)</span>
<span id="cb5-54"><a href="#cb5-54" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb5-55"><a href="#cb5-55" aria-hidden="true" tabindex="-1"></a>early_middle_upr <span class="ot">&lt;-</span> <span class="fu">with</span>(</span>
<span id="cb5-56"><a href="#cb5-56" aria-hidden="true" tabindex="-1"></a>  h1_contrast, <span class="fu">quantile</span>(Contrast[Eras <span class="sc">==</span> <span class="st">"Early to Middle"</span>], <span class="fl">0.975</span>)</span>
<span id="cb5-57"><a href="#cb5-57" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb5-58"><a href="#cb5-58" aria-hidden="true" tabindex="-1"></a>middle_recent_lwr <span class="ot">&lt;-</span> <span class="fu">with</span>(</span>
<span id="cb5-59"><a href="#cb5-59" aria-hidden="true" tabindex="-1"></a>  h1_contrast, <span class="fu">quantile</span>(Contrast[Eras <span class="sc">==</span> <span class="st">"Middle to Recent"</span>], <span class="fl">0.025</span>)</span>
<span id="cb5-60"><a href="#cb5-60" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb5-61"><a href="#cb5-61" aria-hidden="true" tabindex="-1"></a>middle_recent_upr <span class="ot">&lt;-</span> <span class="fu">with</span>(</span>
<span id="cb5-62"><a href="#cb5-62" aria-hidden="true" tabindex="-1"></a>  h1_contrast, <span class="fu">quantile</span>(Contrast[Eras <span class="sc">==</span> <span class="st">"Middle to Recent"</span>], <span class="fl">0.975</span>)</span>
<span id="cb5-63"><a href="#cb5-63" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a>p_h1a <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(h1_dens_dat, <span class="fu">aes</span>(b_T2007, dens)) <span class="sc">+</span></span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_area</span>(<span class="at">data =</span> <span class="fu">filter</span>(h1_dens_dat, ci95), <span class="at">fill =</span> <span class="st">"grey50"</span>) <span class="sc">+</span></span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">expand_limits</span>(<span class="at">x =</span> <span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="fu">expression</span>(beta[t]), <span class="at">y =</span> <span class="st">"Density"</span>) <span class="sc">+</span></span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-8"><a href="#cb6-8" aria-hidden="true" tabindex="-1"></a>p_h1b <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(h1_contrast_dens, <span class="fu">aes</span>(Contrast, Density)) <span class="sc">+</span></span>
<span id="cb6-9"><a href="#cb6-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb6-10"><a href="#cb6-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_area</span>(<span class="at">data =</span> <span class="fu">filter</span>(h1_contrast_dens, ci_95), <span class="at">fill =</span> <span class="st">"grey50"</span>) <span class="sc">+</span></span>
<span id="cb6-11"><a href="#cb6-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_grid</span>(<span class="at">rows =</span> <span class="fu">vars</span>(Eras), <span class="at">scales =</span> <span class="st">"free_y"</span>) <span class="sc">+</span></span>
<span id="cb6-12"><a href="#cb6-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Contrast in p(f)"</span>) <span class="sc">+</span></span>
<span id="cb6-13"><a href="#cb6-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">expand_limits</span>(<span class="at">x =</span> <span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb6-14"><a href="#cb6-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span>
<span id="cb6-15"><a href="#cb6-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-16"><a href="#cb6-16" aria-hidden="true" tabindex="-1"></a><span class="fu">plot_grid</span>(p_h1a, p_h1b, <span class="at">nrow =</span> <span class="dv">1</span>, <span class="at">labels =</span> <span class="st">"AUTO"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="fig-h1" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-h1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="registeredreport_files/figure-html/fig-h1-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-h1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2: Evidence supporting H1 in our simulated data. A: The 95% credible interval for <span class="math inline">\(\beta_t\)</span> for the model fit to our simulated data was greater than 0 (0.130 - 0.224). B: The 95% credible interval for the contrasts of <span class="math inline">\(p(f)\)</span> between the early (2007-2015) and middle (2016-2019) periods (0.073 - 0.300), and between the middle and recent (2020-2023) periods (0.085 - 0.205), were also greater than 0.
</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>h2_dens <span class="ot">&lt;-</span> <span class="fu">density</span>(biolog_draws<span class="sc">$</span>b_Eterrestrial)</span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a>h2_dens_dat <span class="ot">&lt;-</span> <span class="fu">tibble</span>(</span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">b_Eterrestrial =</span> h2_dens<span class="sc">$</span>x,</span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">dens =</span> h2_dens<span class="sc">$</span>y,</span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">ci95 =</span> <span class="fu">between</span>(b_Eterrestrial, </span>
<span id="cb7-6"><a href="#cb7-6" aria-hidden="true" tabindex="-1"></a>                 <span class="fu">quantile</span>(biolog_draws<span class="sc">$</span>b_Eterrestrial, <span class="fl">0.025</span>), </span>
<span id="cb7-7"><a href="#cb7-7" aria-hidden="true" tabindex="-1"></a>                 <span class="fu">quantile</span>(biolog_draws<span class="sc">$</span>b_Eterrestrial, <span class="fl">0.975</span>))</span>
<span id="cb7-8"><a href="#cb7-8" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb7-9"><a href="#cb7-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-10"><a href="#cb7-10" aria-hidden="true" tabindex="-1"></a>h2_contrast <span class="ot">&lt;-</span> <span class="fu">map</span>(</span>
<span id="cb7-11"><a href="#cb7-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">"marine"</span>, <span class="st">"terrestrial"</span>),</span>
<span id="cb7-12"><a href="#cb7-12" aria-hidden="true" tabindex="-1"></a>  <span class="cf">function</span>(e) {</span>
<span id="cb7-13"><a href="#cb7-13" aria-hidden="true" tabindex="-1"></a>    dat <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb7-14"><a href="#cb7-14" aria-hidden="true" tabindex="-1"></a>      <span class="at">E =</span> e,</span>
<span id="cb7-15"><a href="#cb7-15" aria-hidden="true" tabindex="-1"></a>      <span class="at">S =</span> <span class="fu">c</span>(<span class="st">"aspatial"</span>, <span class="st">"spatial"</span>),</span>
<span id="cb7-16"><a href="#cb7-16" aria-hidden="true" tabindex="-1"></a>      <span class="at">T =</span> <span class="dv">2007</span><span class="sc">:</span><span class="dv">2023</span></span>
<span id="cb7-17"><a href="#cb7-17" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">%&gt;%</span> </span>
<span id="cb7-18"><a href="#cb7-18" aria-hidden="true" tabindex="-1"></a>      <span class="fu">mutate</span>(<span class="at">T2007 =</span> T <span class="sc">-</span> <span class="dv">2007</span>)</span>
<span id="cb7-19"><a href="#cb7-19" aria-hidden="true" tabindex="-1"></a>    <span class="fu">linpred_draws</span>(biolog_mod, </span>
<span id="cb7-20"><a href="#cb7-20" aria-hidden="true" tabindex="-1"></a>                  <span class="at">transform =</span> <span class="cn">TRUE</span>, </span>
<span id="cb7-21"><a href="#cb7-21" aria-hidden="true" tabindex="-1"></a>                  <span class="at">newdata =</span> dat)</span>
<span id="cb7-22"><a href="#cb7-22" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb7-23"><a href="#cb7-23" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span> </span>
<span id="cb7-24"><a href="#cb7-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">list_rbind</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb7-25"><a href="#cb7-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_wider</span>(<span class="at">names_from =</span> <span class="st">"E"</span>, <span class="at">values_from =</span> <span class="st">".linpred"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb7-26"><a href="#cb7-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Contrast =</span> terrestrial <span class="sc">-</span> marine)</span>
<span id="cb7-27"><a href="#cb7-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-28"><a href="#cb7-28" aria-hidden="true" tabindex="-1"></a>dens <span class="ot">&lt;-</span> <span class="fu">density</span>(h2_contrast<span class="sc">$</span>Contrast)</span>
<span id="cb7-29"><a href="#cb7-29" aria-hidden="true" tabindex="-1"></a>h2_contrast_dens <span class="ot">&lt;-</span> <span class="fu">tibble</span>(</span>
<span id="cb7-30"><a href="#cb7-30" aria-hidden="true" tabindex="-1"></a>  <span class="at">Contrast =</span> dens<span class="sc">$</span>x,</span>
<span id="cb7-31"><a href="#cb7-31" aria-hidden="true" tabindex="-1"></a>  <span class="at">Density =</span> dens<span class="sc">$</span>y,</span>
<span id="cb7-32"><a href="#cb7-32" aria-hidden="true" tabindex="-1"></a>  <span class="at">ci_95 =</span> <span class="fu">between</span>(Contrast, </span>
<span id="cb7-33"><a href="#cb7-33" aria-hidden="true" tabindex="-1"></a>                  <span class="fu">quantile</span>(h2_contrast<span class="sc">$</span>Contrast, <span class="fl">0.025</span>),</span>
<span id="cb7-34"><a href="#cb7-34" aria-hidden="true" tabindex="-1"></a>                  <span class="fu">quantile</span>(h2_contrast<span class="sc">$</span>Contrast, <span class="fl">0.975</span>))</span>
<span id="cb7-35"><a href="#cb7-35" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb7-36"><a href="#cb7-36" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-37"><a href="#cb7-37" aria-hidden="true" tabindex="-1"></a>beta_e_ci_lwr <span class="ot">&lt;-</span> <span class="fu">quantile</span>(biolog_draws<span class="sc">$</span>b_Eterrestrial, <span class="fl">0.025</span>)</span>
<span id="cb7-38"><a href="#cb7-38" aria-hidden="true" tabindex="-1"></a>beta_e_ci_upr <span class="ot">&lt;-</span> <span class="fu">quantile</span>(biolog_draws<span class="sc">$</span>b_Eterrestrial, <span class="fl">0.975</span>)</span>
<span id="cb7-39"><a href="#cb7-39" aria-hidden="true" tabindex="-1"></a>e_contrast_lwr <span class="ot">&lt;-</span> <span class="fu">quantile</span>(h2_contrast<span class="sc">$</span>Contrast, <span class="fl">0.025</span>)</span>
<span id="cb7-40"><a href="#cb7-40" aria-hidden="true" tabindex="-1"></a>e_contrast_upr <span class="ot">&lt;-</span> <span class="fu">quantile</span>(h2_contrast<span class="sc">$</span>Contrast, <span class="fl">0.975</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a>p_h2a <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(h2_dens_dat, <span class="fu">aes</span>(b_Eterrestrial, dens)) <span class="sc">+</span></span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb8-3"><a href="#cb8-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_area</span>(<span class="at">data =</span> <span class="fu">filter</span>(h2_dens_dat, ci95), <span class="at">fill =</span> <span class="st">"grey50"</span>) <span class="sc">+</span></span>
<span id="cb8-4"><a href="#cb8-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">expand_limits</span>(<span class="at">x =</span> <span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb8-5"><a href="#cb8-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="fu">expression</span>(beta[e]), <span class="at">y =</span> <span class="st">"Density"</span>) <span class="sc">+</span></span>
<span id="cb8-6"><a href="#cb8-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span>
<span id="cb8-7"><a href="#cb8-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-8"><a href="#cb8-8" aria-hidden="true" tabindex="-1"></a>p_h2b <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(h2_contrast_dens, <span class="fu">aes</span>(Contrast, Density)) <span class="sc">+</span></span>
<span id="cb8-9"><a href="#cb8-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb8-10"><a href="#cb8-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_area</span>(<span class="at">data =</span> <span class="fu">filter</span>(h2_contrast_dens, ci_95), <span class="at">fill =</span> <span class="st">"grey50"</span>) <span class="sc">+</span></span>
<span id="cb8-11"><a href="#cb8-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">expand_limits</span>(<span class="at">x =</span> <span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb8-12"><a href="#cb8-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Contrast of p(f) [terrestrial - marine]"</span>) <span class="sc">+</span></span>
<span id="cb8-13"><a href="#cb8-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span>
<span id="cb8-14"><a href="#cb8-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-15"><a href="#cb8-15" aria-hidden="true" tabindex="-1"></a><span class="fu">plot_grid</span>(p_h2a, p_h2b, <span class="at">nrow =</span> <span class="dv">1</span>, <span class="at">labels =</span> <span class="st">"AUTO"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="fig-h2" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-h2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="registeredreport_files/figure-html/fig-h2-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-h2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3: Evidence supporting H2 in our simulated data. A: The 95% credible interval for <span class="math inline">\(\beta_e\)</span> for the model fit to our simulated data was greater than 0 (0.416 - 1.176). B: The 95% credible interval for the contrast of <span class="math inline">\(p(f)\)</span> between terrestrial and marine ecosystems was also greater than 0 (0.026 - 0.251).
</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a>h3_dens <span class="ot">&lt;-</span> <span class="fu">density</span>(biolog_draws<span class="sc">$</span>b_Sspatial)</span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a>h3_dens_dat <span class="ot">&lt;-</span> <span class="fu">tibble</span>(</span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">b_Sspatial =</span> h3_dens<span class="sc">$</span>x,</span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">dens =</span> h3_dens<span class="sc">$</span>y,</span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">ci95 =</span> <span class="fu">between</span>(b_Sspatial, </span>
<span id="cb9-6"><a href="#cb9-6" aria-hidden="true" tabindex="-1"></a>                 <span class="fu">quantile</span>(biolog_draws<span class="sc">$</span>b_Sspatial, <span class="fl">0.025</span>), </span>
<span id="cb9-7"><a href="#cb9-7" aria-hidden="true" tabindex="-1"></a>                 <span class="fu">quantile</span>(biolog_draws<span class="sc">$</span>b_Sspatial, <span class="fl">0.975</span>))</span>
<span id="cb9-8"><a href="#cb9-8" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb9-9"><a href="#cb9-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-10"><a href="#cb9-10" aria-hidden="true" tabindex="-1"></a>h3_contrast <span class="ot">&lt;-</span> <span class="fu">map</span>(</span>
<span id="cb9-11"><a href="#cb9-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">c</span>(<span class="st">"aspatial"</span>, <span class="st">"spatial"</span>),</span>
<span id="cb9-12"><a href="#cb9-12" aria-hidden="true" tabindex="-1"></a>  <span class="cf">function</span>(s) {</span>
<span id="cb9-13"><a href="#cb9-13" aria-hidden="true" tabindex="-1"></a>    dat <span class="ot">&lt;-</span> <span class="fu">expand_grid</span>(</span>
<span id="cb9-14"><a href="#cb9-14" aria-hidden="true" tabindex="-1"></a>      <span class="at">E =</span> <span class="fu">c</span>(<span class="st">"marine"</span>, <span class="st">"terrestrial"</span>),</span>
<span id="cb9-15"><a href="#cb9-15" aria-hidden="true" tabindex="-1"></a>      <span class="at">S =</span> s,</span>
<span id="cb9-16"><a href="#cb9-16" aria-hidden="true" tabindex="-1"></a>      <span class="at">T =</span> <span class="dv">2007</span><span class="sc">:</span><span class="dv">2023</span></span>
<span id="cb9-17"><a href="#cb9-17" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">%&gt;%</span> </span>
<span id="cb9-18"><a href="#cb9-18" aria-hidden="true" tabindex="-1"></a>      <span class="fu">mutate</span>(<span class="at">T2007 =</span> T <span class="sc">-</span> <span class="dv">2007</span>)</span>
<span id="cb9-19"><a href="#cb9-19" aria-hidden="true" tabindex="-1"></a>    <span class="fu">linpred_draws</span>(biolog_mod, </span>
<span id="cb9-20"><a href="#cb9-20" aria-hidden="true" tabindex="-1"></a>                  <span class="at">transform =</span> <span class="cn">TRUE</span>, </span>
<span id="cb9-21"><a href="#cb9-21" aria-hidden="true" tabindex="-1"></a>                  <span class="at">newdata =</span> dat)</span>
<span id="cb9-22"><a href="#cb9-22" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb9-23"><a href="#cb9-23" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span> </span>
<span id="cb9-24"><a href="#cb9-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">list_rbind</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb9-25"><a href="#cb9-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_wider</span>(<span class="at">names_from =</span> <span class="st">"S"</span>, <span class="at">values_from =</span> <span class="st">".linpred"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb9-26"><a href="#cb9-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Contrast =</span> spatial <span class="sc">-</span> aspatial)</span>
<span id="cb9-27"><a href="#cb9-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-28"><a href="#cb9-28" aria-hidden="true" tabindex="-1"></a>dens <span class="ot">&lt;-</span> <span class="fu">density</span>(h3_contrast<span class="sc">$</span>Contrast)</span>
<span id="cb9-29"><a href="#cb9-29" aria-hidden="true" tabindex="-1"></a>h3_contrast_dens <span class="ot">&lt;-</span> <span class="fu">tibble</span>(</span>
<span id="cb9-30"><a href="#cb9-30" aria-hidden="true" tabindex="-1"></a>  <span class="at">Contrast =</span> dens<span class="sc">$</span>x,</span>
<span id="cb9-31"><a href="#cb9-31" aria-hidden="true" tabindex="-1"></a>  <span class="at">Density =</span> dens<span class="sc">$</span>y,</span>
<span id="cb9-32"><a href="#cb9-32" aria-hidden="true" tabindex="-1"></a>  <span class="at">ci_95 =</span> <span class="fu">between</span>(Contrast, </span>
<span id="cb9-33"><a href="#cb9-33" aria-hidden="true" tabindex="-1"></a>                  <span class="fu">quantile</span>(h3_contrast<span class="sc">$</span>Contrast, <span class="fl">0.025</span>),</span>
<span id="cb9-34"><a href="#cb9-34" aria-hidden="true" tabindex="-1"></a>                  <span class="fu">quantile</span>(h3_contrast<span class="sc">$</span>Contrast, <span class="fl">0.975</span>))</span>
<span id="cb9-35"><a href="#cb9-35" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb9-36"><a href="#cb9-36" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-37"><a href="#cb9-37" aria-hidden="true" tabindex="-1"></a>beta_s_ci_lwr <span class="ot">&lt;-</span> <span class="fu">quantile</span>(biolog_draws<span class="sc">$</span>b_Sspatial, <span class="fl">0.025</span>)</span>
<span id="cb9-38"><a href="#cb9-38" aria-hidden="true" tabindex="-1"></a>beta_s_ci_upr <span class="ot">&lt;-</span> <span class="fu">quantile</span>(biolog_draws<span class="sc">$</span>b_Sspatial, <span class="fl">0.975</span>)</span>
<span id="cb9-39"><a href="#cb9-39" aria-hidden="true" tabindex="-1"></a>s_contrast_lwr <span class="ot">&lt;-</span> <span class="fu">quantile</span>(h3_contrast<span class="sc">$</span>Contrast, <span class="fl">0.025</span>)</span>
<span id="cb9-40"><a href="#cb9-40" aria-hidden="true" tabindex="-1"></a>s_contrast_upr <span class="ot">&lt;-</span> <span class="fu">quantile</span>(h3_contrast<span class="sc">$</span>Contrast, <span class="fl">0.975</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a>p_h3a <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(h3_dens_dat, <span class="fu">aes</span>(b_Sspatial, dens)) <span class="sc">+</span></span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_area</span>(<span class="at">data =</span> <span class="fu">filter</span>(h3_dens_dat, ci95), <span class="at">fill =</span> <span class="st">"grey50"</span>) <span class="sc">+</span></span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">expand_limits</span>(<span class="at">x =</span> <span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="fu">expression</span>(beta[s]), <span class="at">y =</span> <span class="st">"Density"</span>) <span class="sc">+</span></span>
<span id="cb10-6"><a href="#cb10-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span>
<span id="cb10-7"><a href="#cb10-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-8"><a href="#cb10-8" aria-hidden="true" tabindex="-1"></a>p_h3b <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(h3_contrast_dens, <span class="fu">aes</span>(Contrast, Density)) <span class="sc">+</span></span>
<span id="cb10-9"><a href="#cb10-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb10-10"><a href="#cb10-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_area</span>(<span class="at">data =</span> <span class="fu">filter</span>(h3_contrast_dens, ci_95), <span class="at">fill =</span> <span class="st">"grey50"</span>) <span class="sc">+</span></span>
<span id="cb10-11"><a href="#cb10-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">expand_limits</span>(<span class="at">x =</span> <span class="dv">0</span>) <span class="sc">+</span></span>
<span id="cb10-12"><a href="#cb10-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Contrast of p(f) [spatial - aspatial]"</span>) <span class="sc">+</span></span>
<span id="cb10-13"><a href="#cb10-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_classic</span>()</span>
<span id="cb10-14"><a href="#cb10-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-15"><a href="#cb10-15" aria-hidden="true" tabindex="-1"></a><span class="fu">plot_grid</span>(p_h3a, p_h3b, <span class="at">nrow =</span> <span class="dv">1</span>, <span class="at">labels =</span> <span class="st">"AUTO"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="fig-h3" class="quarto-figure quarto-figure-center quarto-float anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-h3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="registeredreport_files/figure-html/fig-h3-1.png" class="img-fluid figure-img" width="672">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-h3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4: Evidence supporting H3 in our simulated data. A: The 95% credible interval for <span class="math inline">\(\beta_s\)</span> for the model fit to our simulated data was greater than 0 (0.867 - 1.638). B: The 95% credible interval for the contrast of <span class="math inline">\(p(f)\)</span> between aspatial and spatial ecosystems was also greater than 0 (0.056 - 0.354).
</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="planned-tables-and-figures" class="level2">
<h2 class="anchored" data-anchor-id="planned-tables-and-figures">Planned tables and figures</h2>
<p>Table 1: Description of the FAIR principles and their application to biologging data (see <a href="#tbl-fair" class="quarto-xref">Table&nbsp;1</a> above).</p>
<p>Table 2: Results of our hypothesis tests for the three dependent variables. 95% credible intervals for posterior probability distributions of our model coefficients and contrasts. This table summarizes <a href="#fig-h1" class="quarto-xref">Figure&nbsp;2</a>, <a href="#fig-h2" class="quarto-xref">Figure&nbsp;3</a>, and <a href="#fig-h3" class="quarto-xref">Figure&nbsp;4</a> from the section “Analysis of simulated data”, as well as results for the other two dependent variables.</p>
<p>Figure 1: Results of the initial classification. <strong>A</strong> Sankey diagram showing the frequencies of instrumented species by number of studies, with aggregations at higher taxa. <strong>B</strong> Frequencies of location, intrinsic, and environmental sensors used in studies. <strong>C</strong> Prevalence of data availability statements over time. <strong>D</strong> Cumulative biologging studies published over time in the ten journals with the most published biologging studies.</p>
<p>Figure 2: Results of the open data assessment. Mean and standard error of dependent variables (<span class="math inline">\(f\)</span> findability, <span class="math inline">\(a\)</span> accessibility, <span class="math inline">\(b\)</span> reusability) aggregated over: <strong>A</strong> sensor type (aspatial or spatial), <strong>B</strong> ecosystem (marine or terrestrial), and <strong>C</strong> time (year of publication).</p>
<p>Supplemental figures 1-3: Posterior probability distributions of model coefficients and contrasts. Each supplemental figure will show the full posterior probability distributions for three hypotheses and one dependent variable. I.e. supplemental figure 1 would combine <a href="#fig-h1" class="quarto-xref">Figure&nbsp;2</a>, <a href="#fig-h2" class="quarto-xref">Figure&nbsp;3</a>, and <a href="#fig-h3" class="quarto-xref">Figure&nbsp;4</a> from the section “Analysis of simulated data” above. Supplemental figures 2 and 3 would be similar, showing the results for the other dependent variables accessibility ( <span class="math inline">\(a\)</span> ) and reproducibility ( <span class="math inline">\(r\)</span> ).</p>

</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body" data-entry-spacing="0" data-line-spacing="2" role="list">
<div id="ref-rutz2009" class="csl-entry" role="listitem">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Rutz, C. &amp; Hays, G. C. <a href="https://doi.org/10.1098/rsbl.2009.0089">New frontiers in biologging science</a>. <em>Biology Letters</em> <strong>5</strong>, 289–292 (2009).</div>
</div>
<div id="ref-kooyman1966" class="csl-entry" role="listitem">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Kooyman, G. L. <a href="https://doi.org/10.1126/science.151.3717.1553">Maximum Diving Capacities of the Weddell Seal,<span><em>Leptonychotes weddelli</em></span></a>. <em>Science</em> <strong>151</strong>, 1553–1554 (1966).</div>
</div>
<div id="ref-wilmers2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Wilmers, C. C. <em>et al.</em> <a href="https://doi.org/10.1890/14-1401.1">The golden age of bio<span>-</span>logging: how animal<span>-</span>borne sensors are advancing the frontiers of ecology</a>. <em>Ecology</em> <strong>96</strong>, 1741–1753 (2015).</div>
</div>
<div id="ref-weinzierl2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Weinzierl, R. <em>et al.</em> <a href="https://doi.org/10.1002/ece3.2585">Wind estimation based on thermal soaring of birds</a>. <em>Ecology and Evolution</em> <strong>6</strong>, 8706–8718 (2016).</div>
</div>
<div id="ref-yonehara2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Yonehara, Y. <em>et al.</em> <a href="https://doi.org/10.1073/pnas.1523853113">Flight paths of seabirds soaring over the ocean surface enable measurement of fine-scale wind speed and direction</a>. <em>Proceedings of the National Academy of Sciences</em> <strong>113</strong>, 9039–9044 (2016).</div>
</div>
<div id="ref-treasure2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Treasure, A. <em>et al.</em> <a href="https://doi.org/10.5670/oceanog.2017.234">Marine mammals exploring the oceans pole to pole: A review of the MEOP consortium</a>. <em>Oceanography</em> <strong>30</strong>, 132–138 (2017).</div>
</div>
<div id="ref-kays2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Kays, R. <em>et al.</em> <a href="https://doi.org/10.1111/2041-210x.13767">The Movebank system for studying global animal movement and demography</a>. <em>Methods in Ecology and Evolution</em> <strong>13</strong>, 419–431 (2021).</div>
</div>
<div id="ref-denouden2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Ouden, O. F. C. den, Smets, P. S. M., Assink, J. D. &amp; Evers, L. G. <a href="https://doi.org/10.1029/2021gl094555">A Bird<span>’</span>s<span>-</span>Eye View on Ambient Infrasonic Soundscapes</a>. <em>Geophysical Research Letters</em> <strong>48</strong>, (2021).</div>
</div>
<div id="ref-nathan2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Nathan, R. <em>et al.</em> <a href="https://doi.org/10.1126/science.abg1780">Big-data approaches lead to an increased understanding of the ecology of animal movement</a>. <em>Science</em> <strong>375</strong>, (2022).</div>
</div>
<div id="ref-grémillet2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Grémillet, D., Chevallier, D. &amp; Guinet, C. <a href="https://doi.org/10.1093/icesjms/fsac059">Big data approaches to the spatial ecology and conservation of marine megafauna</a>. <em>ICES Journal of Marine Science</em> <strong>79</strong>, 975–986 (2022).</div>
</div>
<div id="ref-beal2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">Beal, M. <em>et al.</em> <a href="https://doi.org/10.1126/sciadv.abd7225">Global political responsibility for the conservation of albatrosses and large petrels</a>. <em>Science Advances</em> <strong>7</strong>, (2021).</div>
</div>
<div id="ref-queiroz2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Queiroz, N. <em>et al.</em> <a href="https://doi.org/10.1038/s41586-019-1444-4">Global spatial risk assessment of sharks under the footprint of fisheries</a>. <em>Nature</em> <strong>572</strong>, 461–466 (2019).</div>
</div>
<div id="ref-hindell2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Hindell, M. A. <em>et al.</em> <a href="https://doi.org/10.1038/s41586-020-2126-y">Tracking of marine predators to protect Southern Ocean ecosystems</a>. <em>Nature</em> <strong>580</strong>, 87–92 (2020).</div>
</div>
<div id="ref-sequeira2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Sequeira, A. M. M. <em>et al.</em> <a href="https://doi.org/10.1111/2041-210x.13593">A standardisation framework for bio<span>-</span>logging data to advance ecological research and conservation</a>. <em>Methods in Ecology and Evolution</em> <strong>12</strong>, 996–1007 (2021).</div>
</div>
<div id="ref-rutz2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Rutz, C. <a href="https://doi.org/10.1038/d41586-022-02821-6">Register animal-tracking tags to boost conservation</a>. <em>Nature</em> <strong>609</strong>, 221–221 (2022).</div>
</div>
<div id="ref-lowndes2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Lowndes, J. S. S. <em>et al.</em> <a href="https://doi.org/10.1038/s41559-017-0160">Our path to better science in less time using open data science tools</a>. <em>Nature Ecology &amp; Evolution</em> <strong>1</strong>, 1–7 (2017).</div>
</div>
<div id="ref-hampton2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">Hampton, S. E. <em>et al.</em> <a href="https://doi.org/10.1093/biosci/bix025">Skills and Knowledge for Data-Intensive Environmental Research</a>. <em>BioScience</em> <strong>67</strong>, 546–557 (2017).</div>
</div>
<div id="ref-peterson2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Peterson, A. T., Asase, A., Canhos, D., Souza, S. de &amp; Wieczorek, J. <a href="https://doi.org/10.3897/bdj.6.e26826">Data leakage and loss in biodiversity informatics</a>. <em>Biodiversity Data Journal</em> <strong>6</strong>, (2018).</div>
</div>
<div id="ref-wilkinson2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Wilkinson, M. D. <em>et al.</em> <a href="https://doi.org/10.1038/sdata.2016.18">The FAIR Guiding Principles for scientific data management and stewardship</a>. <em>Scientific Data</em> <strong>3</strong>, 160018 (2016).</div>
</div>
<div id="ref-halpern2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Halpern, B. S. <em>et al.</em> Ecological Synthesis and Its Role in Advancing Knowledge. <em>BioScience</em> (2020) doi:<a href="https://doi.org/10.1093/biosci/biaa105">10.1093/biosci/biaa105</a>.</div>
</div>
<div id="ref-fredston2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">Fredston, A. L. &amp; Lowndes, J. S. S. <a href="https://doi.org/10.1146/annurev-marine-041723-094741">Welcoming More Participation in Open Data Science for the Oceans</a>. <em>Annual Review of Marine Science</em> <strong>16</strong>, (2023).</div>
</div>
<div id="ref-roche2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Roche, D. G. <em>et al.</em> <a href="https://doi.org/10.1111/cobi.13835">Closing the knowledge<span>-</span>action gap in conservation with open science</a>. <em>Conservation Biology</em> <strong>36</strong>, (2021).</div>
</div>
<div id="ref-buxton2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Buxton, R. T. <em>et al.</em> <a href="https://doi.org/10.1111/csp2.329">Avoiding wasted research resources in conservation science</a>. <em>Conservation Science and Practice</em> <strong>3</strong>, (2021).</div>
</div>
<div id="ref-dettki2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Dettki, H., Arlt, D., Bäckman, J. &amp; Blanchet, M. <a href="https://doi.org/10.3897/biss.7.111438">Key-value pairs and NoSQL databases: A novel concept to manage biologging data in data repositories</a>. <em>Biodiversity Information Science and Standards</em> <strong>7</strong>, (2023).</div>
</div>
<div id="ref-kranstauber2011" class="csl-entry" role="listitem">
<div class="csl-left-margin">25. </div><div class="csl-right-inline">Kranstauber, B. <em>et al.</em> <a href="https://doi.org/10.1016/j.envsoft.2010.12.005">The Movebank data model for animal tracking</a>. <em>Environmental Modelling &amp; Software</em> <strong>26</strong>, 834–835 (2011).</div>
</div>
<div id="ref-urbano2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Urbano, F. &amp; Cagnacci, F. <a href="https://doi.org/10.3389/fevo.2021.727023">Data management and sharing for collaborative science: Lessons learnt from the euromammals initiative</a>. <em>Frontiers in Ecology and Evolution</em> <strong>9</strong>, (2021).</div>
</div>
<div id="ref-campbell2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Campbell, H. A. <em>et al.</em> <a href="https://doi.org/10.1016/j.scitotenv.2015.01.089">Finding our way: On the sharing and reuse of animal telemetry data in Australasia</a>. <em>Science of The Total Environment</em> <strong>534</strong>, 79–84 (2015).</div>
</div>
<div id="ref-scarpignato2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Scarpignato, A. L. <em>et al.</em> <a href="https://doi.org/10.1016/j.biocon.2023.110224">Shortfalls in tracking data available to inform North American migratory bird conservation</a>. <em>Biological Conservation</em> <strong>286</strong>, 110224 (2023).</div>
</div>
<div id="ref-kranstauber2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">Kranstauber, B., Safi, K. &amp; Scharf, A. K. <em><a href="https://CRAN.R-project.org/package=move2">move2: Processing and analysing animal trajectories</a></em>. (2023).</div>
</div>
<div id="ref-wieczorek2012" class="csl-entry" role="listitem">
<div class="csl-left-margin">30. </div><div class="csl-right-inline">Wieczorek, J. <em>et al.</em> <a href="https://doi.org/10.1371/journal.pone.0029715">Darwin Core: an evolving community-developed biodiversity data standard</a>. <em>PLoS ONE</em> <strong>7</strong>, e29715 (2012).</div>
</div>
<div id="ref-MPIAB2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">31. </div><div class="csl-right-inline">Animal Behavior, M. P. I. of. <a href="http://dx.doi.org/10.1038/s41597-019-0042-5">Movebank Attribute Dictionary</a>.</div>
</div>
<div id="ref-roche2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">32. </div><div class="csl-right-inline">Roche, D. G., Kruuk, L. E. B., Lanfear, R. &amp; Binning, S. A. <a href="https://doi.org/10.1371/journal.pbio.1002295">Public Data Archiving in Ecology and Evolution: How Well Are We Doing?</a> <em>PLOS Biology</em> <strong>13</strong>, e1002295 (2015).</div>
</div>
<div id="ref-taylor2007" class="csl-entry" role="listitem">
<div class="csl-left-margin">33. </div><div class="csl-right-inline">Taylor, C. F. <em>et al.</em> <a href="https://doi.org/10.1038/nbt1329">The minimum information about a proteomics experiment (MIAPE)</a>. <em>Nature Biotechnology</em> <strong>25</strong>, 887–893 (2007).</div>
</div>
<div id="ref-rund2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">34. </div><div class="csl-right-inline">Rund, S. S. C. <em>et al.</em> <a href="https://doi.org/10.1038/s41597-019-0042-5">MIReAD, a minimum information standard for reporting arthropod abundance data</a>. <em>Scientific Data</em> <strong>6</strong>, (2019).</div>
</div>
<div id="ref-cooke2004" class="csl-entry" role="listitem">
<div class="csl-left-margin">35. </div><div class="csl-right-inline">Cooke, S. J. <em>et al.</em> <a href="https://doi.org/10.1016/j.tree.2004.04.003">Biotelemetry: a mechanistic approach to ecology</a>. <em>Trends in Ecology &amp; Evolution</em> <strong>19</strong>, 334–343 (2004).</div>
</div>
<div id="ref-williams2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">36. </div><div class="csl-right-inline">Williams, H. J. <em>et al.</em> <a href="https://doi.org/10.1111/1365-2656.13094">Optimizing the use of biologgers for movement ecology research</a>. <em>Journal of Animal Ecology</em> <strong>89</strong>, 186–206 (2019).</div>
</div>
<div id="ref-Stan" class="csl-entry" role="listitem">
<div class="csl-left-margin">37. </div><div class="csl-right-inline">Stan Development Team. <a href="https://mc-stan.org">Stan modeling language users guide and reference manual, version 2.34</a>. (2024).</div>
</div>
<div id="ref-bürkner2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">38. </div><div class="csl-right-inline">Bürkner, P.-C. <a href="https://doi.org/10.32614/rj-2018-017">Advanced Bayesian Multilevel Modeling with the R Package brms</a>. <em>The R Journal</em> <strong>10</strong>, 395 (2018).</div>
</div>
<div id="ref-bürkner2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">39. </div><div class="csl-right-inline">Bürkner, P.-C. <a href="https://doi.org/10.18637/jss.v100.i05">Bayesian Item Response Modeling in <span><em>R</em></span> with <span><strong>brms</strong></span> and <span><em>Stan</em></span></a>. <em>Journal of Statistical Software</em> <strong>100</strong>, (2021).</div>
</div>
<div id="ref-bürkner2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">40. </div><div class="csl-right-inline">Bürkner, P.-C. &amp; Charpentier, E. <a href="https://doi.org/10.1111/bmsp.12195">Modelling monotonic effects of ordinal predictors in Bayesian regression models</a>. <em>British Journal of Mathematical and Statistical Psychology</em> <strong>73</strong>, 420–451 (2020).</div>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>